
<!DOCTYPE html>
<html class="client-nojs" lang="en" dir="ltr">
<head>
<meta charset="UTF-8"/>
<title>DNA sequencing - Wikipedia</title>
<script>document.documentElement.className="client-js";RLCONF={"wgBreakFrames":!1,"wgSeparatorTransformTable":["",""],"wgDigitTransformTable":["",""],"wgDefaultDateFormat":"dmy","wgMonthNames":["","January","February","March","April","May","June","July","August","September","October","November","December"],"wgRequestId":"Xn9CgwpAEJcAA4CJL@oAAABI","wgCSPNonce":!1,"wgCanonicalNamespace":"","wgCanonicalSpecialPageName":!1,"wgNamespaceNumber":0,"wgPageName":"DNA_sequencing","wgTitle":"DNA sequencing","wgCurRevisionId":945603030,"wgRevisionId":945603030,"wgArticleId":1158125,"wgIsArticle":!0,"wgIsRedirect":!1,"wgAction":"view","wgUserName":null,"wgUserGroups":["*"],"wgCategories":["CS1 errors: missing periodical","Pages containing links to subscription-only content","Wikipedia articles needing page number citations from June 2013","Pages with login required references or sources","CS1 maint: archived copy as title","CS1: long volume value","Wikipedia indefinitely move-protected pages"
,"Articles with short description","Use dmy dates from April 2011","Articles containing potentially dated statements from 2019","All articles containing potentially dated statements","All articles with unsourced statements","Articles with unsourced statements from July 2018","Articles to be expanded from May 2015","All articles to be expanded","Articles using small message boxes","DNA sequencing","Biotechnology","DNA","Genetic mapping","Molecular biology","Molecular biology techniques","1970 introductions","1970 in biology","1970 in biotechnology","1970 in science","1998 in technology"],"wgPageContentLanguage":"en","wgPageContentModel":"wikitext","wgRelevantPageName":"DNA_sequencing","wgRelevantArticleId":1158125,"wgIsProbablyEditable":!0,"wgRelevantPageIsProbablyEditable":!0,"wgRestrictionEdit":[],"wgRestrictionMove":["sysop"],"wgRedirectedFrom":"DNA_sequence","wgMediaViewerOnClick":!0,"wgMediaViewerEnabledByDefault":!0,"wgPopupsReferencePreviews":!1,
"wgPopupsConflictsWithNavPopupGadget":!1,"wgVisualEditor":{"pageLanguageCode":"en","pageLanguageDir":"ltr","pageVariantFallbacks":"en"},"wgMFDisplayWikibaseDescriptions":{"search":!0,"nearby":!0,"watchlist":!0,"tagline":!1},"wgWMESchemaEditAttemptStepOversample":!1,"wgULSCurrentAutonym":"English","wgNoticeProject":"wikipedia","wgInternalRedirectTargetUrl":"/wiki/DNA_sequencing","wgWikibaseItemId":"Q380546","wgCentralAuthMobileDomain":!1,"wgEditSubmitButtonLabelPublish":!0};RLSTATE={"ext.globalCssJs.user.styles":"ready","site.styles":"ready","noscript":"ready","user.styles":"ready","ext.globalCssJs.user":"ready","user":"ready","user.options":"loading","ext.cite.styles":"ready","mediawiki.toc.styles":"ready","skins.vector.styles.legacy":"ready","wikibase.client.init":"ready","ext.visualEditor.desktopArticleTarget.noscript":"ready","ext.uls.interlanguage":"ready","ext.wikimediaBadges":"ready"};RLPAGEMODULES=["mediawiki.action.view.redirect","ext.cite.ux-enhancements",
"site","mediawiki.page.startup","skins.vector.js","mediawiki.page.ready","mediawiki.toc","ext.gadget.ReferenceTooltips","ext.gadget.charinsert","ext.gadget.refToolbar","ext.gadget.extra-toolbar-buttons","ext.gadget.switcher","ext.centralauth.centralautologin","mmv.head","mmv.bootstrap.autostart","ext.popups","ext.visualEditor.desktopArticleTarget.init","ext.visualEditor.targetLoader","ext.eventLogging","ext.wikimediaEvents","ext.navigationTiming","ext.uls.compactlinks","ext.uls.interface","ext.cx.eventlogging.campaigns","ext.quicksurveys.init","ext.centralNotice.geoIP","ext.centralNotice.startUp"];</script>
<script>(RLQ=window.RLQ||[]).push(function(){mw.loader.implement("user.options@1hzgi",function($,jQuery,require,module){/*@nomin*/mw.user.tokens.set({"patrolToken":"+\\","watchToken":"+\\","csrfToken":"+\\"});
});});</script>
<link rel="stylesheet" href="/w/load.php?lang=en&amp;modules=ext.cite.styles%7Cext.uls.interlanguage%7Cext.visualEditor.desktopArticleTarget.noscript%7Cext.wikimediaBadges%7Cmediawiki.toc.styles%7Cskins.vector.styles.legacy%7Cwikibase.client.init&amp;only=styles&amp;skin=vector"/>
<script async="" src="/w/load.php?lang=en&amp;modules=startup&amp;only=scripts&amp;raw=1&amp;skin=vector"></script>
<meta name="ResourceLoaderDynamicStyles" content=""/>
<link rel="stylesheet" href="/w/load.php?lang=en&amp;modules=site.styles&amp;only=styles&amp;skin=vector"/>
<meta name="generator" content="MediaWiki 1.35.0-wmf.25"/>
<meta name="referrer" content="origin"/>
<meta name="referrer" content="origin-when-crossorigin"/>
<meta name="referrer" content="origin-when-cross-origin"/>
<meta property="og:image" content="https://upload.wikimedia.org/wikipedia/commons/thumb/f/f0/Hybridogenesis_in_water_frogs_gametes.svg/1200px-Hybridogenesis_in_water_frogs_gametes.svg.png"/>
<link rel="alternate" type="application/x-wiki" title="Edit this page" href="/w/index.php?title=DNA_sequencing&amp;action=edit"/>
<link rel="edit" title="Edit this page" href="/w/index.php?title=DNA_sequencing&amp;action=edit"/>
<link rel="apple-touch-icon" href="/static/apple-touch/wikipedia.png"/>
<link rel="shortcut icon" href="/static/favicon/wikipedia.ico"/>
<link rel="search" type="application/opensearchdescription+xml" href="/w/opensearch_desc.php" title="Wikipedia (en)"/>
<link rel="EditURI" type="application/rsd+xml" href="//en.wikipedia.org/w/api.php?action=rsd"/>
<link rel="license" href="//creativecommons.org/licenses/by-sa/3.0/"/>
<link rel="alternate" type="application/atom+xml" title="Wikipedia Atom feed" href="/w/index.php?title=Special:RecentChanges&amp;feed=atom"/>
<link rel="canonical" href="https://en.wikipedia.org/wiki/DNA_sequencing"/>
<link rel="dns-prefetch" href="//login.wikimedia.org"/>
<link rel="dns-prefetch" href="//meta.wikimedia.org" />
<!--[if lt IE 9]><script src="/w/resources/lib/html5shiv/html5shiv.js"></script><![endif]-->
</head>
<body class="mediawiki ltr sitedir-ltr mw-hide-empty-elt ns-0 ns-subject mw-editable page-DNA_sequencing rootpage-DNA_sequencing skin-vector action-view">
<div id="mw-page-base" class="noprint"></div>
<div id="mw-head-base" class="noprint"></div>
<div id="content" class="mw-body" role="main">
	<a id="top"></a>
	<div id="siteNotice" class="mw-body-content"><!-- CentralNotice --></div>
	<div class="mw-indicators mw-body-content">
</div>

	<h1 id="firstHeading" class="firstHeading" lang="en">DNA sequencing</h1>
	
	<div id="bodyContent" class="mw-body-content">
		<div id="siteSub" class="noprint">From Wikipedia, the free encyclopedia</div>
		<div id="contentSub"><span class="mw-redirectedfrom">&#160;&#160;(Redirected from <a href="/w/index.php?title=DNA_sequence&amp;redirect=no" class="mw-redirect" title="DNA sequence">DNA sequence</a>)</span></div>
		
		
		<div id="jump-to-nav"></div>
		<a class="mw-jump-link" href="#mw-head">Jump to navigation</a>
		<a class="mw-jump-link" href="#p-search">Jump to search</a>
		<div id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><div class="mw-parser-output"><div class="shortdescription nomobile noexcerpt noprint searchaux" style="display:none">process of determining the nucleic acid sequence – the order of nucleotides in DNA</div>
<p class="mw-empty-elt">
</p>
<table class="vertical-navbox nowraplinks plainlist" style="float:right;clear:right;width:22.0em;margin:0 0 1.0em 1.0em;background:#f9f9f9;border:1px solid #aaa;padding:0.2em;border-spacing:0.4em 0;text-align:center;line-height:1.4em;font-size:88%;border:2px solid #90db90;"><tbody><tr><td style="padding-top:0.4em;line-height:1.2em;padding-bottom:0.15em;">Part of <a href="/wiki/Category:Genetics" title="Category:Genetics">a series</a> on</td></tr><tr><th style="padding:0.2em 0.4em 0.2em;padding-top:0;font-size:145%;line-height:1.2em;background:#90db90;font-size:175%;padding-top:0.2em;font-weight:normal;"><b><a href="/wiki/Genetics" title="Genetics">Genetics</a></b></th></tr><tr><td style="padding:0.2em 0 0.4em"><a href="/wiki/File:Hybridogenesis_in_water_frogs_gametes.svg" class="image"><img alt="Hybridogenesis in water frogs gametes.svg" src="//upload.wikimedia.org/wikipedia/commons/thumb/f/f0/Hybridogenesis_in_water_frogs_gametes.svg/150px-Hybridogenesis_in_water_frogs_gametes.svg.png" decoding="async" width="150" height="74" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/f/f0/Hybridogenesis_in_water_frogs_gametes.svg/225px-Hybridogenesis_in_water_frogs_gametes.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/f/f0/Hybridogenesis_in_water_frogs_gametes.svg/300px-Hybridogenesis_in_water_frogs_gametes.svg.png 2x" data-file-width="225" data-file-height="111" /></a></td></tr><tr><td style="padding:0 0.1em 0.4em;background:#e7f4e7;padding:0.2em 0.4em 0.4em;;background:none;font-size:40%;line-height:0.4em;">
&#160;</td>
</tr><tr><th style="padding:0.1em;background:#90db90;">
Key components</th></tr><tr><td style="padding:0 0.1em 0.4em;background:#e7f4e7;padding:0.2em 0.4em 0.4em;">
<div class="hlist">
<ul><li><a href="/wiki/Chromosome" title="Chromosome">Chromosome</a></li>
<li><a href="/wiki/DNA" title="DNA">DNA</a></li>
<li><a href="/wiki/RNA" title="RNA">RNA</a></li></ul>
</div><div class="hlist">
<ul><li><a href="/wiki/Genome" title="Genome">Genome</a></li>
<li><a href="/wiki/Heredity" title="Heredity">Heredity</a></li>
<li><a href="/wiki/Mutation" title="Mutation">Mutation</a></li></ul>
</div><div class="hlist">
<ul><li><a href="/wiki/Nucleotide" title="Nucleotide">Nucleotide</a></li>
<li><a href="/wiki/Genetic_variation" title="Genetic variation">Variation</a></li></ul>
</div></td>
</tr><tr><td style="padding:0 0.1em 0.4em;background:#e7f4e7;padding:0.2em 0.4em 0.4em;;border-top:1px solid #90db90;padding-top:0.3em;">
<div class="hlist">
<ul><li><a href="/wiki/Outline_of_genetics" title="Outline of genetics">Outline</a></li>
<li><a href="/wiki/Index_of_genetics_articles" title="Index of genetics articles">Index</a></li></ul>
</div></td>
</tr><tr><th style="padding:0.1em;background:#90db90;">
History and topics</th></tr><tr><td style="padding:0 0.1em 0.4em;background:#e7f4e7;padding:0.2em 0.4em 0.4em;;padding-bottom:0.2em;">
<div class="hlist">
<ul><li><a href="/wiki/Introduction_to_genetics" title="Introduction to genetics">Introduction</a></li>
<li><a href="/wiki/History_of_genetics" title="History of genetics">History</a></li></ul>
</div>
<ul><li><a href="/wiki/Evolution" title="Evolution">Evolution</a> (<a href="/wiki/Molecular_evolution" title="Molecular evolution">molecular</a>)</li>
<li><a href="/wiki/Population_genetics" title="Population genetics">Population genetics</a></li>
<li><a href="/wiki/Mendelian_inheritance" title="Mendelian inheritance">Mendelian inheritance</a></li>
<li><a href="/wiki/Quantitative_genetics" title="Quantitative genetics">Quantitative genetics</a></li>
<li><a href="/wiki/Molecular_genetics" title="Molecular genetics">Molecular genetics</a></li></ul></td>
</tr><tr><th style="padding:0.1em;background:#90db90;">
Research</th></tr><tr><td style="padding:0 0.1em 0.4em;background:#e7f4e7;padding:0.2em 0.4em 0.4em;;border-top:1px solid #90db90;padding-top:0.3em;">
<ul><li><a class="mw-selflink selflink">DNA sequencing</a></li>
<li><a href="/wiki/Genetic_engineering" title="Genetic engineering">Genetic engineering</a></li>
<li><span class="nowrap"><a href="/wiki/Genomics" title="Genomics">Genomics</a> (<img alt="Template" src="//upload.wikimedia.org/wikipedia/en/thumb/5/5c/Symbol_template_class.svg/12px-Symbol_template_class.svg.png" decoding="async" title="Template" width="12" height="12" srcset="//upload.wikimedia.org/wikipedia/en/thumb/5/5c/Symbol_template_class.svg/18px-Symbol_template_class.svg.png 1.5x, //upload.wikimedia.org/wikipedia/en/thumb/5/5c/Symbol_template_class.svg/24px-Symbol_template_class.svg.png 2x" data-file-width="180" data-file-height="185" /> <a href="/wiki/Template:Genomics" title="Template:Genomics">template</a>)</span></li>
<li><a href="/wiki/Medical_genetics" title="Medical genetics">Medical genetics</a></li></ul></td>
</tr><tr><td style="padding:0 0.1em 0.4em;background:#e7f4e7;padding:0.2em 0.4em 0.4em;;border-top:1px solid #90db90;padding-top:0.15em;">
<ul><li><a href="/wiki/Outline_of_genetics#Branches_of_genetics" title="Outline of genetics">Branches of genetics</a></li></ul></td>
</tr><tr><th style="padding:0.1em;background:#90db90;">
Personalized medicine</th></tr><tr><td style="padding:0 0.1em 0.4em;background:#e7f4e7;padding:0.2em 0.4em 0.4em;">
<a href="/wiki/Personalized_medicine" title="Personalized medicine">Personalized medicine</a></td>
</tr><tr><td style="text-align:right;font-size:115%"><div class="plainlinks hlist navbar mini"><ul><li class="nv-view"><a href="/wiki/Template:Genetics_sidebar" title="Template:Genetics sidebar"><abbr title="View this template">v</abbr></a></li><li class="nv-talk"><a href="/wiki/Template_talk:Genetics_sidebar" title="Template talk:Genetics sidebar"><abbr title="Discuss this template">t</abbr></a></li><li class="nv-edit"><a class="external text" href="https://en.wikipedia.org/w/index.php?title=Template:Genetics_sidebar&amp;action=edit"><abbr title="Edit this template">e</abbr></a></li></ul></div></td></tr></tbody></table>
<p><b>DNA sequencing</b> is the process of determining the <a href="/wiki/Nucleic_acid_sequence" title="Nucleic acid sequence">nucleic acid sequence</a> – the order of <a href="/wiki/Nucleotides" class="mw-redirect" title="Nucleotides">nucleotides</a> in <a href="/wiki/DNA" title="DNA">DNA</a>. It includes any method or technology that is used to determine the order of the four bases: <a href="/wiki/Adenine" title="Adenine">adenine</a>, <a href="/wiki/Guanine" title="Guanine">guanine</a>, <a href="/wiki/Cytosine" title="Cytosine">cytosine</a>, and <a href="/wiki/Thymine" title="Thymine">thymine</a>. The advent of rapid DNA sequencing methods has greatly accelerated biological and medical research and discovery.<sup id="cite_ref-1" class="reference"><a href="#cite_note-1">&#91;1&#93;</a></sup><sup id="cite_ref-2" class="reference"><a href="#cite_note-2">&#91;2&#93;</a></sup>
</p><p>Knowledge of <b>DNA sequences</b> has become indispensable for basic biological research, and in numerous applied fields such as <a href="/wiki/Medical_diagnosis" title="Medical diagnosis">medical diagnosis</a>, <a href="/wiki/Biotechnology" title="Biotechnology">biotechnology</a>, <a href="/wiki/Forensic_biology" title="Forensic biology">forensic biology</a>, <a href="/wiki/Virology" title="Virology">virology</a> and biological <a href="/wiki/Systematics" title="Systematics">systematics</a>. Comparing healthy and mutated DNA sequences can diagnose different diseases including various cancers,<sup id="cite_ref-3" class="reference"><a href="#cite_note-3">&#91;3&#93;</a></sup> characterize antibody repertoire,<sup id="cite_ref-:3_4-0" class="reference"><a href="#cite_note-:3-4">&#91;4&#93;</a></sup> and can be used to guide patient treatment.<sup id="cite_ref-5" class="reference"><a href="#cite_note-5">&#91;5&#93;</a></sup> Having a quick way to sequence DNA allows for faster and more individualized medical care to be administered, and for more organisms to be identified and cataloged.<sup id="cite_ref-:3_4-1" class="reference"><a href="#cite_note-:3-4">&#91;4&#93;</a></sup>
</p><p>
The rapid speed of sequencing attained with modern DNA sequencing technology has been instrumental in the sequencing of complete DNA sequences, or <a href="/wiki/Genomes" class="mw-redirect" title="Genomes">genomes</a>, of numerous types and species of life, including the <a href="/wiki/Human_genome" title="Human genome">human genome</a> and other complete DNA sequences of many animal, plant, and microbial species.</p><div class="thumb tright"><div class="thumbinner" style="width:222px;"><a href="/wiki/File:Radioactive_Fluorescent_Seq.jpg" class="image"><img alt="" src="//upload.wikimedia.org/wikipedia/commons/thumb/3/3d/Radioactive_Fluorescent_Seq.jpg/220px-Radioactive_Fluorescent_Seq.jpg" decoding="async" width="220" height="313" class="thumbimage" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/3/3d/Radioactive_Fluorescent_Seq.jpg/330px-Radioactive_Fluorescent_Seq.jpg 1.5x, //upload.wikimedia.org/wikipedia/commons/3/3d/Radioactive_Fluorescent_Seq.jpg 2x" data-file-width="417" data-file-height="594" /></a>  <div class="thumbcaption"><div class="magnify"><a href="/wiki/File:Radioactive_Fluorescent_Seq.jpg" class="internal" title="Enlarge"></a></div>An example of the results of automated chain-termination DNA sequencing.</div></div></div>
<p>The first DNA sequences were obtained in the early 1970s by academic researchers using laborious methods based on <a href="/wiki/Two-dimensional_chromatography" title="Two-dimensional chromatography">two-dimensional chromatography</a>. Following the development of <a href="/wiki/Fluorescence" title="Fluorescence">fluorescence</a>-based sequencing methods with a <a href="/wiki/DNA_sequencer" title="DNA sequencer">DNA sequencer</a>,<sup id="cite_ref-olsvik1993_6-0" class="reference"><a href="#cite_note-olsvik1993-6">&#91;6&#93;</a></sup> DNA sequencing has become easier and orders of magnitude faster.<sup id="cite_ref-pmid18992322_7-0" class="reference"><a href="#cite_note-pmid18992322-7">&#91;7&#93;</a></sup>
</p>
<div id="toc" class="toc" role="navigation" aria-labelledby="mw-toc-heading"><input type="checkbox" role="button" id="toctogglecheckbox" class="toctogglecheckbox" style="display:none" /><div class="toctitle" lang="en" dir="ltr"><h2 id="mw-toc-heading">Contents</h2><span class="toctogglespan"><label class="toctogglelabel" for="toctogglecheckbox"></label></span></div>
<ul>
<li class="toclevel-1 tocsection-1"><a href="#Applications"><span class="tocnumber">1</span> <span class="toctext">Applications</span></a>
<ul>
<li class="toclevel-2 tocsection-2"><a href="#Molecular_biology"><span class="tocnumber">1.1</span> <span class="toctext">Molecular biology</span></a></li>
<li class="toclevel-2 tocsection-3"><a href="#Evolutionary_biology"><span class="tocnumber">1.2</span> <span class="toctext">Evolutionary biology</span></a></li>
<li class="toclevel-2 tocsection-4"><a href="#Metagenomics"><span class="tocnumber">1.3</span> <span class="toctext">Metagenomics</span></a></li>
<li class="toclevel-2 tocsection-5"><a href="#Virology"><span class="tocnumber">1.4</span> <span class="toctext">Virology</span></a></li>
<li class="toclevel-2 tocsection-6"><a href="#Medicine"><span class="tocnumber">1.5</span> <span class="toctext">Medicine</span></a></li>
<li class="toclevel-2 tocsection-7"><a href="#Forensics"><span class="tocnumber">1.6</span> <span class="toctext">Forensics</span></a></li>
</ul>
</li>
<li class="toclevel-1 tocsection-8"><a href="#The_four_canonical_bases"><span class="tocnumber">2</span> <span class="toctext">The four canonical bases</span></a></li>
<li class="toclevel-1 tocsection-9"><a href="#History"><span class="tocnumber">3</span> <span class="toctext">History</span></a>
<ul>
<li class="toclevel-2 tocsection-10"><a href="#Discovery_of_DNA_structure_and_function"><span class="tocnumber">3.1</span> <span class="toctext">Discovery of DNA structure and function</span></a></li>
<li class="toclevel-2 tocsection-11"><a href="#RNA_sequencing"><span class="tocnumber">3.2</span> <span class="toctext">RNA sequencing</span></a></li>
<li class="toclevel-2 tocsection-12"><a href="#Early_DNA_sequencing_methods"><span class="tocnumber">3.3</span> <span class="toctext">Early DNA sequencing methods</span></a></li>
<li class="toclevel-2 tocsection-13"><a href="#Sequencing_of_full_genomes"><span class="tocnumber">3.4</span> <span class="toctext">Sequencing of full genomes</span></a></li>
<li class="toclevel-2 tocsection-14"><a href="#High-throughput_sequencing_(HTS)_methods"><span class="tocnumber">3.5</span> <span class="toctext">High-throughput sequencing (HTS) methods</span></a></li>
</ul>
</li>
<li class="toclevel-1 tocsection-15"><a href="#Basic_methods"><span class="tocnumber">4</span> <span class="toctext">Basic methods</span></a>
<ul>
<li class="toclevel-2 tocsection-16"><a href="#Maxam-Gilbert_sequencing"><span class="tocnumber">4.1</span> <span class="toctext">Maxam-Gilbert sequencing</span></a></li>
<li class="toclevel-2 tocsection-17"><a href="#Chain-termination_methods"><span class="tocnumber">4.2</span> <span class="toctext">Chain-termination methods</span></a></li>
</ul>
</li>
<li class="toclevel-1 tocsection-18"><a href="#Large-scale_sequencing_and_de_novo_sequencing"><span class="tocnumber">5</span> <span class="toctext">Large-scale sequencing and <i>de novo</i> sequencing</span></a>
<ul>
<li class="toclevel-2 tocsection-19"><a href="#Shotgun_sequencing"><span class="tocnumber">5.1</span> <span class="toctext">Shotgun sequencing</span></a></li>
</ul>
</li>
<li class="toclevel-1 tocsection-20"><a href="#High-throughput_methods"><span class="tocnumber">6</span> <span class="toctext">High-throughput methods</span></a>
<ul>
<li class="toclevel-2 tocsection-21"><a href="#Long-read_sequencing_methods"><span class="tocnumber">6.1</span> <span class="toctext">Long-read sequencing methods</span></a>
<ul>
<li class="toclevel-3 tocsection-22"><a href="#Single_molecule_real_time_(SMRT)_sequencing"><span class="tocnumber">6.1.1</span> <span class="toctext">Single molecule real time (SMRT) sequencing</span></a></li>
<li class="toclevel-3 tocsection-23"><a href="#Nanopore_DNA_sequencing"><span class="tocnumber">6.1.2</span> <span class="toctext">Nanopore DNA sequencing</span></a></li>
</ul>
</li>
<li class="toclevel-2 tocsection-24"><a href="#Short-read_sequencing_methods"><span class="tocnumber">6.2</span> <span class="toctext">Short-read sequencing methods</span></a>
<ul>
<li class="toclevel-3 tocsection-25"><a href="#Massively_parallel_signature_sequencing_(MPSS)"><span class="tocnumber">6.2.1</span> <span class="toctext">Massively parallel signature sequencing (MPSS)</span></a></li>
<li class="toclevel-3 tocsection-26"><a href="#Polony_sequencing"><span class="tocnumber">6.2.2</span> <span class="toctext">Polony sequencing</span></a></li>
<li class="toclevel-3 tocsection-27"><a href="#454_pyrosequencing"><span class="tocnumber">6.2.3</span> <span class="toctext">454 pyrosequencing</span></a></li>
<li class="toclevel-3 tocsection-28"><a href="#Illumina_(Solexa)_sequencing"><span class="tocnumber">6.2.4</span> <span class="toctext">Illumina (Solexa) sequencing</span></a></li>
<li class="toclevel-3 tocsection-29"><a href="#Combinatorial_probe_anchor_synthesis_(cPAS)"><span class="tocnumber">6.2.5</span> <span class="toctext">Combinatorial probe anchor synthesis (cPAS)</span></a></li>
<li class="toclevel-3 tocsection-30"><a href="#SOLiD_sequencing"><span class="tocnumber">6.2.6</span> <span class="toctext">SOLiD sequencing</span></a></li>
<li class="toclevel-3 tocsection-31"><a href="#Ion_Torrent_semiconductor_sequencing"><span class="tocnumber">6.2.7</span> <span class="toctext">Ion Torrent semiconductor sequencing</span></a></li>
<li class="toclevel-3 tocsection-32"><a href="#DNA_nanoball_sequencing"><span class="tocnumber">6.2.8</span> <span class="toctext">DNA nanoball sequencing</span></a></li>
<li class="toclevel-3 tocsection-33"><a href="#Heliscope_single_molecule_sequencing"><span class="tocnumber">6.2.9</span> <span class="toctext">Heliscope single molecule sequencing</span></a></li>
<li class="toclevel-3 tocsection-34"><a href="#Microfluidic_Systems"><span class="tocnumber">6.2.10</span> <span class="toctext">Microfluidic Systems</span></a></li>
</ul>
</li>
</ul>
</li>
<li class="toclevel-1 tocsection-35"><a href="#Methods_in_development"><span class="tocnumber">7</span> <span class="toctext">Methods in development</span></a>
<ul>
<li class="toclevel-2 tocsection-36"><a href="#Tunnelling_currents_DNA_sequencing"><span class="tocnumber">7.1</span> <span class="toctext">Tunnelling currents DNA sequencing</span></a></li>
<li class="toclevel-2 tocsection-37"><a href="#Sequencing_by_hybridization"><span class="tocnumber">7.2</span> <span class="toctext">Sequencing by hybridization</span></a></li>
<li class="toclevel-2 tocsection-38"><a href="#Sequencing_with_mass_spectrometry"><span class="tocnumber">7.3</span> <span class="toctext">Sequencing with mass spectrometry</span></a></li>
<li class="toclevel-2 tocsection-39"><a href="#Microfluidic_Sanger_sequencing"><span class="tocnumber">7.4</span> <span class="toctext">Microfluidic Sanger sequencing</span></a></li>
<li class="toclevel-2 tocsection-40"><a href="#Microscopy-based_techniques"><span class="tocnumber">7.5</span> <span class="toctext">Microscopy-based techniques</span></a></li>
<li class="toclevel-2 tocsection-41"><a href="#RNAP_sequencing"><span class="tocnumber">7.6</span> <span class="toctext">RNAP sequencing</span></a></li>
<li class="toclevel-2 tocsection-42"><a href="#In_vitro_virus_high-throughput_sequencing"><span class="tocnumber">7.7</span> <span class="toctext"><i>In vitro</i> virus high-throughput sequencing</span></a></li>
</ul>
</li>
<li class="toclevel-1 tocsection-43"><a href="#Sample_preparation"><span class="tocnumber">8</span> <span class="toctext">Sample preparation</span></a></li>
<li class="toclevel-1 tocsection-44"><a href="#Development_initiatives"><span class="tocnumber">9</span> <span class="toctext">Development initiatives</span></a></li>
<li class="toclevel-1 tocsection-45"><a href="#Computational_challenges"><span class="tocnumber">10</span> <span class="toctext">Computational challenges</span></a>
<ul>
<li class="toclevel-2 tocsection-46"><a href="#Read_trimming"><span class="tocnumber">10.1</span> <span class="toctext">Read trimming</span></a></li>
</ul>
</li>
<li class="toclevel-1 tocsection-47"><a href="#Ethical_issues"><span class="tocnumber">11</span> <span class="toctext">Ethical issues</span></a></li>
<li class="toclevel-1 tocsection-48"><a href="#See_also"><span class="tocnumber">12</span> <span class="toctext">See also</span></a></li>
<li class="toclevel-1 tocsection-49"><a href="#Notes"><span class="tocnumber">13</span> <span class="toctext">Notes</span></a></li>
<li class="toclevel-1 tocsection-50"><a href="#References"><span class="tocnumber">14</span> <span class="toctext">References</span></a></li>
<li class="toclevel-1 tocsection-51"><a href="#External_links"><span class="tocnumber">15</span> <span class="toctext">External links</span></a></li>
</ul>
</div>

<h2><span class="mw-headline" id="Applications">Applications</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=DNA_sequencing&amp;action=edit&amp;section=1" title="Edit section: Applications">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<p>DNA sequencing may be used to determine the sequence of individual <a href="/wiki/Gene" title="Gene">genes</a>, larger genetic regions (i.e. clusters of genes or <a href="/wiki/Operons" class="mw-redirect" title="Operons">operons</a>), full chromosomes, or <a href="/wiki/Whole_genome_sequencing" title="Whole genome sequencing">entire genomes</a> of any organism.  DNA sequencing is also the most efficient way to indirectly sequence <a href="/wiki/RNA" title="RNA">RNA</a> or <a href="/wiki/Protein" title="Protein">proteins</a> (via their <a href="/wiki/Open_reading_frame" title="Open reading frame">open reading frames</a>). In fact, DNA sequencing has become a key technology in many areas of biology and other sciences such as medicine, <a href="/wiki/Forensics" class="mw-redirect" title="Forensics">forensics</a>, and <a href="/wiki/Anthropology" title="Anthropology">anthropology</a>.
</p>
<h3><span class="mw-headline" id="Molecular_biology">Molecular biology</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=DNA_sequencing&amp;action=edit&amp;section=2" title="Edit section: Molecular biology">edit</a><span class="mw-editsection-bracket">]</span></span></h3>
<p>Sequencing is used in <a href="/wiki/Molecular_biology" title="Molecular biology">molecular biology</a> to study genomes and the proteins they encode. Information obtained using sequencing allows researchers to identify changes in genes, associations with diseases and phenotypes, and identify potential drug targets.
</p>
<h3><span class="mw-headline" id="Evolutionary_biology">Evolutionary biology</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=DNA_sequencing&amp;action=edit&amp;section=3" title="Edit section: Evolutionary biology">edit</a><span class="mw-editsection-bracket">]</span></span></h3>
<p>Since DNA is an informative macromolecule in terms of transmission from one generation to another, DNA sequencing is used in <a href="/wiki/Evolutionary_biology" title="Evolutionary biology">evolutionary biology</a> to study how different organisms are related and how they evolved.
</p>
<h3><span class="mw-headline" id="Metagenomics">Metagenomics</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=DNA_sequencing&amp;action=edit&amp;section=4" title="Edit section: Metagenomics">edit</a><span class="mw-editsection-bracket">]</span></span></h3>
<div role="note" class="hatnote navigation-not-searchable">Main article: <a href="/wiki/Metagenomics" title="Metagenomics">Metagenomics</a></div>
<p>The field of <a href="/wiki/Metagenomics" title="Metagenomics">metagenomics</a> involves identification of organisms present in a body of water, <a href="/wiki/Sewage" title="Sewage">sewage</a>, dirt, debris filtered from the air, or swab samples from organisms. Knowing which organisms are present in a particular environment is critical to research in <a href="/wiki/Ecology" title="Ecology">ecology</a>, <a href="/wiki/Epidemiology" title="Epidemiology">epidemiology</a>, <a href="/wiki/Microbiology" title="Microbiology">microbiology</a>, and other fields. Sequencing enables researchers to determine which types of microbes may be present in a <a href="/wiki/Microbiome" class="mw-redirect" title="Microbiome">microbiome</a>, for example.
</p>
<h3><span class="mw-headline" id="Virology">Virology</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=DNA_sequencing&amp;action=edit&amp;section=5" title="Edit section: Virology">edit</a><span class="mw-editsection-bracket">]</span></span></h3>
<div role="note" class="hatnote navigation-not-searchable">Main article: <a href="/wiki/Virology" title="Virology">Virology</a></div>
<p>As most viruses are too small to be seen by a light microscope, sequencing is one of the main tools in virology to identify and study the virus. Traditional Sanger sequencing and next-generation sequencing are used to sequence viruses in basic and clinical research, as wells as for the diagnosis of emerging viral infections, <a href="/wiki/Molecular_epidemiology" title="Molecular epidemiology">molecular epidemiology</a> of viral pathogens, and drug-resistance testing. There are more than 2.3 million unique viral sequences in <a href="/wiki/GenBank" title="GenBank">GenBank</a> <sup id="cite_ref-:0c_8-0" class="reference"><a href="#cite_note-:0c-8">&#91;8&#93;</a></sup>. Recently, NGS has surpassed traditional Sanger as the most popular approach for generating viral genomes <sup id="cite_ref-:0c_8-1" class="reference"><a href="#cite_note-:0c-8">&#91;8&#93;</a></sup>.
</p>
<h3><span class="mw-headline" id="Medicine">Medicine</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=DNA_sequencing&amp;action=edit&amp;section=6" title="Edit section: Medicine">edit</a><span class="mw-editsection-bracket">]</span></span></h3>
<p>Medical technicians may sequence genes (or, theoretically, full genomes) from patients to determine if there is risk of genetic diseases. This is a form of <a href="/wiki/Genetic_testing" title="Genetic testing">genetic testing</a>, though some genetic tests may not involve DNA sequencing.
Also, DNA sequencing may be useful for determining a specific bacteria, to allow for more <a href="/wiki/Antimicrobial_spectrum" title="Antimicrobial spectrum">precise antibiotics treatments</a>, hereby reducing the risk of creating <a href="/wiki/Antimicrobial_resistance" title="Antimicrobial resistance">antimicrobial resistance</a> in bacteria populations.<sup id="cite_ref-9" class="reference"><a href="#cite_note-9">&#91;9&#93;</a></sup><sup id="cite_ref-10" class="reference"><a href="#cite_note-10">&#91;10&#93;</a></sup><sup id="cite_ref-11" class="reference"><a href="#cite_note-11">&#91;11&#93;</a></sup><sup id="cite_ref-12" class="reference"><a href="#cite_note-12">&#91;12&#93;</a></sup><sup id="cite_ref-13" class="reference"><a href="#cite_note-13">&#91;13&#93;</a></sup><sup id="cite_ref-14" class="reference"><a href="#cite_note-14">&#91;14&#93;</a></sup>
</p>
<h3><span class="mw-headline" id="Forensics">Forensics</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=DNA_sequencing&amp;action=edit&amp;section=7" title="Edit section: Forensics">edit</a><span class="mw-editsection-bracket">]</span></span></h3>
<p>DNA sequencing may be used along with <a href="/wiki/DNA_profiling" title="DNA profiling">DNA profiling</a> methods for <a href="/wiki/Forensic_identification" title="Forensic identification">forensic identification</a><sup id="cite_ref-15" class="reference"><a href="#cite_note-15">&#91;15&#93;</a></sup> and <a href="/wiki/DNA_paternity_testing" title="DNA paternity testing">paternity testing</a>. DNA testing has evolved tremendously in the last few decades to ultimately link a DNA print to what is under investigation. The DNA patterns in fingerprint, saliva, hair follicles, etc. uniquely separate each living organism from another. Testing DNA is a technique which can detect specific genomes in a DNA strand to produce a unique and individualized pattern.
</p>
<h2><span class="mw-headline" id="The_four_canonical_bases">The four canonical bases</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=DNA_sequencing&amp;action=edit&amp;section=8" title="Edit section: The four canonical bases">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<div role="note" class="hatnote navigation-not-searchable">Main article: <a href="/wiki/Nucleotide" title="Nucleotide">Nucleotide</a></div>
<p>The canonical structure of DNA has four bases: <a href="/wiki/Thymine" title="Thymine">thymine</a> (T), <a href="/wiki/Adenine" title="Adenine">adenine</a> (A), <a href="/wiki/Cytosine" title="Cytosine">cytosine</a> (C), and <a href="/wiki/Guanine" title="Guanine">guanine</a> (G). DNA sequencing is the determination of the physical order of these bases in a molecule of DNA. However, there are many other bases that may be present in a molecule. In some viruses (specifically, <a href="/wiki/Bacteriophage" title="Bacteriophage">bacteriophage</a>), cytosine may be replaced by hydroxy methyl or hydroxy methyl glucose cytosine.<sup id="cite_ref-16" class="reference"><a href="#cite_note-16">&#91;16&#93;</a></sup> In mammalian DNA, variant bases with <a href="/wiki/Methyl" class="mw-redirect" title="Methyl">methyl</a> groups or phosphosulfate may be found.<sup id="cite_ref-17" class="reference"><a href="#cite_note-17">&#91;17&#93;</a></sup><sup id="cite_ref-18" class="reference"><a href="#cite_note-18">&#91;18&#93;</a></sup> Depending on the sequencing technique, a particular modification, e.g., the 5mC (<a href="/wiki/5_methyl_cytosine" class="mw-redirect" title="5 methyl cytosine">5 methyl cytosine</a>) common in humans, may or may not be detected.<sup id="cite_ref-19" class="reference"><a href="#cite_note-19">&#91;19&#93;</a></sup>
</p>
<h2><span class="mw-headline" id="History">History</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=DNA_sequencing&amp;action=edit&amp;section=9" title="Edit section: History">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<h3><span class="mw-headline" id="Discovery_of_DNA_structure_and_function">Discovery of DNA structure and function</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=DNA_sequencing&amp;action=edit&amp;section=10" title="Edit section: Discovery of DNA structure and function">edit</a><span class="mw-editsection-bracket">]</span></span></h3>
<p>Deoxyribonucleic acid (<a href="/wiki/DNA" title="DNA">DNA</a>) was first discovered and isolated by <a href="/wiki/Friedrich_Miescher" title="Friedrich Miescher">Friedrich Miescher</a> in 1869, but it remained under-studied for many decades because proteins, rather than DNA, were thought to hold the genetic blueprint to life. This situation changed after 1944 as a result of some experiments by <a href="/wiki/Oswald_Avery" title="Oswald Avery">Oswald Avery</a>, <a href="/wiki/Colin_Munro_MacLeod" title="Colin Munro MacLeod">Colin MacLeod</a>, and <a href="/wiki/Maclyn_McCarty" title="Maclyn McCarty">Maclyn McCarty</a> demonstrating that purified DNA could change one strain of bacteria into another. This was the first time that DNA was shown capable of transforming the properties of cells.
</p><p>In 1953, <a href="/wiki/James_Watson" title="James Watson">James Watson</a> and <a href="/wiki/Francis_Crick" title="Francis Crick">Francis Crick</a> put forward their <a href="/wiki/Double-helix" class="mw-redirect" title="Double-helix">double-helix</a> model of DNA, based on <a href="/wiki/X-ray_crystallography" title="X-ray crystallography">crystallized X-ray</a> structures being studied by <a href="/wiki/Rosalind_Franklin" title="Rosalind Franklin">Rosalind Franklin</a>. According to the model, DNA is composed of two strands of nucleotides coiled around each other, linked together by hydrogen bonds and running in opposite directions. Each strand is composed of four complementary nucleotides – adenine (A), cytosine (C), guanine (G) and thymine (T) – with an A on one strand always paired with T on the other, and C always paired with G. They proposed that such a structure allowed each strand to be used to reconstruct the other, an idea central to the passing on of hereditary information between generations.<sup id="cite_ref-pmid13168976_20-0" class="reference"><a href="#cite_note-pmid13168976-20">&#91;20&#93;</a></sup>
</p>
<div class="thumb tright"><div class="thumbinner" style="width:222px;"><a href="/wiki/File:Frederick_Sanger2.jpg" class="image"><img alt="" src="//upload.wikimedia.org/wikipedia/commons/thumb/f/f8/Frederick_Sanger2.jpg/220px-Frederick_Sanger2.jpg" decoding="async" width="220" height="293" class="thumbimage" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/f/f8/Frederick_Sanger2.jpg/330px-Frederick_Sanger2.jpg 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/f/f8/Frederick_Sanger2.jpg/440px-Frederick_Sanger2.jpg 2x" data-file-width="897" data-file-height="1196" /></a>  <div class="thumbcaption"><div class="magnify"><a href="/wiki/File:Frederick_Sanger2.jpg" class="internal" title="Enlarge"></a></div><a href="/wiki/Frederick_Sanger" title="Frederick Sanger">Frederick Sanger</a>, a pioneer of sequencing. Sanger is one of the few scientists who was awarded two Nobel prizes, one for the <a href="/wiki/Protein_sequencing" title="Protein sequencing">sequencing of proteins</a>, and the other for the sequencing of DNA.</div></div></div>
<p>The foundation for sequencing proteins was first laid by the work of <a href="/wiki/Frederick_Sanger" title="Frederick Sanger">Frederick Sanger</a> who by 1955 had completed the sequence of all the amino acids in <a href="/wiki/Insulin" title="Insulin">insulin</a>, a small protein secreted by the pancreas. This provided the first conclusive evidence that proteins were chemical entities with a specific molecular pattern rather than a random mixture of material suspended in fluid. Sanger's success in sequencing insulin spurred on x-ray crystallographers, including Watson and Crick, who by now were trying to understand how DNA directed the formation of proteins within a cell. Soon after attending a series of lectures given by Frederick Sanger in October 1954, Crick began developing a theory which argued that the arrangement of nucleotides in DNA determined the sequence of amino acids in proteins, which in turn helped determine the function of a protein. He published this theory in 1958.<sup id="cite_ref-21" class="reference"><a href="#cite_note-21">&#91;21&#93;</a></sup>
</p>
<h3><span class="mw-headline" id="RNA_sequencing">RNA sequencing</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=DNA_sequencing&amp;action=edit&amp;section=11" title="Edit section: RNA sequencing">edit</a><span class="mw-editsection-bracket">]</span></span></h3>
<p><a href="/wiki/RNA_sequencing" class="mw-redirect" title="RNA sequencing">RNA sequencing</a> was one of the earliest forms of nucleotide sequencing. The major landmark of RNA sequencing is the sequence of the first complete gene and the complete genome of <a href="/wiki/Bacteriophage_MS2" title="Bacteriophage MS2">Bacteriophage MS2</a>, identified and published by <a href="/wiki/Walter_Fiers" title="Walter Fiers">Walter Fiers</a> and his coworkers at the <a href="/wiki/University_of_Ghent" class="mw-redirect" title="University of Ghent">University of Ghent</a> (<a href="/wiki/Ghent" title="Ghent">Ghent</a>, <a href="/wiki/Belgium" title="Belgium">Belgium</a>), in 1972<sup id="cite_ref-22" class="reference"><a href="#cite_note-22">&#91;22&#93;</a></sup> and 1976.<sup id="cite_ref-23" class="reference"><a href="#cite_note-23">&#91;23&#93;</a></sup> Traditional RNA sequencing methods require the creation of a <a href="/wiki/Complementary_DNA" title="Complementary DNA">cDNA</a> molecule which must be sequenced.<sup id="cite_ref-24" class="reference"><a href="#cite_note-24">&#91;24&#93;</a></sup>
</p>
<h3><span class="mw-headline" id="Early_DNA_sequencing_methods">Early DNA sequencing methods</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=DNA_sequencing&amp;action=edit&amp;section=12" title="Edit section: Early DNA sequencing methods">edit</a><span class="mw-editsection-bracket">]</span></span></h3>
<p>The first method for determining DNA sequences involved a location-specific primer extension strategy established by <a href="/wiki/Ray_Wu" title="Ray Wu">Ray Wu</a> at <a href="/wiki/Cornell_University" title="Cornell University">Cornell University</a> in 1970.<sup id="cite_ref-25" class="reference"><a href="#cite_note-25">&#91;25&#93;</a></sup> DNA polymerase catalysis and specific nucleotide labeling, both of which figure prominently in current sequencing schemes, were used to sequence the cohesive ends of lambda phage DNA.<sup id="cite_ref-26" class="reference"><a href="#cite_note-26">&#91;26&#93;</a></sup><sup id="cite_ref-27" class="reference"><a href="#cite_note-27">&#91;27&#93;</a></sup><sup id="cite_ref-pmid4553110_28-0" class="reference"><a href="#cite_note-pmid4553110-28">&#91;28&#93;</a></sup>  Between 1970 and 1973, Wu, R Padmanabhan and colleagues demonstrated that this method can be employed to determine any DNA sequence using synthetic location-specific primers.<sup id="cite_ref-pmid4560009_29-0" class="reference"><a href="#cite_note-pmid4560009-29">&#91;29&#93;</a></sup><sup id="cite_ref-pmid4358929_30-0" class="reference"><a href="#cite_note-pmid4358929-30">&#91;30&#93;</a></sup><sup id="cite_ref-31" class="reference"><a href="#cite_note-31">&#91;31&#93;</a></sup> <a href="/wiki/Frederick_Sanger" title="Frederick Sanger">Frederick Sanger</a> then adopted this primer-extension strategy to develop more rapid DNA sequencing methods at the <a href="/wiki/Medical_Research_Council_(United_Kingdom)" title="Medical Research Council (United Kingdom)">MRC Centre</a>, <a href="/wiki/Cambridge" title="Cambridge">Cambridge</a>, UK and published a method for "DNA sequencing with chain-terminating inhibitors" in 1977.<sup id="cite_ref-Sanger1977_32-0" class="reference"><a href="#cite_note-Sanger1977-32">&#91;32&#93;</a></sup> <a href="/wiki/Walter_Gilbert" title="Walter Gilbert">Walter Gilbert</a> and <a href="/wiki/Allan_Maxam" title="Allan Maxam">Allan Maxam</a> at <a href="/wiki/Harvard_University" title="Harvard University">Harvard</a> also developed sequencing methods, including one for "DNA sequencing by chemical degradation".<sup id="cite_ref-Maxam77_33-0" class="reference"><a href="#cite_note-Maxam77-33">&#91;33&#93;</a></sup><sup id="cite_ref-34" class="reference"><a href="#cite_note-34">&#91;34&#93;</a></sup> In 1973, Gilbert and Maxam reported the sequence of 24 basepairs using a method known as wandering-spot analysis.<sup id="cite_ref-35" class="reference"><a href="#cite_note-35">&#91;35&#93;</a></sup> Advancements in sequencing were aided by the concurrent development of <a href="/wiki/Recombinant_DNA" title="Recombinant DNA">recombinant DNA</a> technology, allowing DNA samples to be isolated from sources other than viruses.
</p>
<h3><span class="mw-headline" id="Sequencing_of_full_genomes">Sequencing of full genomes</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=DNA_sequencing&amp;action=edit&amp;section=13" title="Edit section: Sequencing of full genomes">edit</a><span class="mw-editsection-bracket">]</span></span></h3>
<div class="thumb tright"><div class="thumbinner" style="width:302px;"><a href="/wiki/File:Genome_map_of_the_bacteriophage_%CE%A6X174_showing_overlapping_genes.svg" class="image"><img alt="" src="//upload.wikimedia.org/wikipedia/commons/thumb/7/7e/Genome_map_of_the_bacteriophage_%CE%A6X174_showing_overlapping_genes.svg/300px-Genome_map_of_the_bacteriophage_%CE%A6X174_showing_overlapping_genes.svg.png" decoding="async" width="300" height="322" class="thumbimage" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/7/7e/Genome_map_of_the_bacteriophage_%CE%A6X174_showing_overlapping_genes.svg/450px-Genome_map_of_the_bacteriophage_%CE%A6X174_showing_overlapping_genes.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/7/7e/Genome_map_of_the_bacteriophage_%CE%A6X174_showing_overlapping_genes.svg/600px-Genome_map_of_the_bacteriophage_%CE%A6X174_showing_overlapping_genes.svg.png 2x" data-file-width="1040" data-file-height="1117" /></a>  <div class="thumbcaption"><div class="magnify"><a href="/wiki/File:Genome_map_of_the_bacteriophage_%CE%A6X174_showing_overlapping_genes.svg" class="internal" title="Enlarge"></a></div>The 5,386 bp genome of <a href="/wiki/Bacteriophage_%CF%86X174" class="mw-redirect" title="Bacteriophage φX174">bacteriophage φX174</a>. Each coloured block represents a gene.</div></div></div>
<p>The first full DNA genome to be sequenced was that of <a href="/wiki/Bacteriophage_%CF%86X174" class="mw-redirect" title="Bacteriophage φX174">bacteriophage φX174</a> in 1977.<sup id="cite_ref-36" class="reference"><a href="#cite_note-36">&#91;36&#93;</a></sup> <a href="/wiki/Medical_Research_Council_(UK)" class="mw-redirect" title="Medical Research Council (UK)">Medical Research Council</a> scientists deciphered the complete DNA sequence of the <a href="/wiki/Epstein-Barr_virus" class="mw-redirect" title="Epstein-Barr virus">Epstein-Barr virus</a> in 1984, finding it contained 172,282 nucleotides. Completion of the sequence marked a significant turning point in DNA sequencing because it was achieved with no prior genetic profile knowledge of the virus.<sup id="cite_ref-37" class="reference"><a href="#cite_note-37">&#91;37&#93;</a></sup>
</p><p>A non-radioactive method for transferring the DNA molecules of sequencing reaction mixtures onto an immobilizing matrix during electrophoresis was developed by Pohl and co-workers in the early 1980s.<sup id="cite_ref-38" class="reference"><a href="#cite_note-38">&#91;38&#93;</a></sup><sup id="cite_ref-39" class="reference"><a href="#cite_note-39">&#91;39&#93;</a></sup> Followed by the commercialization of the DNA sequencer "Direct-Blotting-Electrophoresis-System GATC 1500" by <a href="/wiki/GATC_Biotech" title="GATC Biotech">GATC Biotech</a>, which was intensively used in the framework of the EU genome-sequencing programme, the complete DNA sequence of the yeast <i><a href="/wiki/Saccharomyces_cerevisiae" title="Saccharomyces cerevisiae">Saccharomyces cerevisiae</a></i> chromosome II.<sup id="cite_ref-Feldmann_1994_40-0" class="reference"><a href="#cite_note-Feldmann_1994-40">&#91;40&#93;</a></sup> <a href="/wiki/Leroy_E._Hood" class="mw-redirect" title="Leroy E. Hood">Leroy E. Hood</a>'s laboratory at the <a href="/wiki/California_Institute_of_Technology" title="California Institute of Technology">California Institute of Technology</a> announced the first semi-automated DNA sequencing machine in 1986.<sup id="cite_ref-41" class="reference"><a href="#cite_note-41">&#91;41&#93;</a></sup> This was followed by <a href="/wiki/Applied_Biosystems" title="Applied Biosystems">Applied Biosystems</a>' marketing of the first fully automated sequencing machine, the ABI 370, in 1987 and by Dupont's Genesis 2000<sup id="cite_ref-42" class="reference"><a href="#cite_note-42">&#91;42&#93;</a></sup>  which used a novel fluorescent labeling technique enabling all four dideoxynucleotides to be identified in a single lane. By 1990, the U.S. <a href="/wiki/National_Institutes_of_Health" title="National Institutes of Health">National Institutes of Health</a> (NIH) had begun large-scale sequencing trials on <i><a href="/wiki/Mycoplasma_capricolum" title="Mycoplasma capricolum">Mycoplasma capricolum</a></i>, <i><a href="/wiki/Escherichia_coli" title="Escherichia coli">Escherichia coli</a></i>, <i><a href="/wiki/Caenorhabditis_elegans" title="Caenorhabditis elegans">Caenorhabditis elegans</a></i>, and <i><a href="/wiki/Saccharomyces_cerevisiae" title="Saccharomyces cerevisiae">Saccharomyces cerevisiae</a></i> at a cost of US$0.75 per base. Meanwhile, sequencing of human <a href="/wiki/CDNA" class="mw-redirect" title="CDNA">cDNA</a> sequences called <a href="/wiki/Expressed_sequence_tag" title="Expressed sequence tag">expressed sequence tags</a> began in <a href="/wiki/Craig_Venter" title="Craig Venter">Craig Venter</a>'s lab, an attempt to capture the coding fraction of the <a href="/wiki/Human_genome" title="Human genome">human genome</a>.<sup id="cite_ref-pmid2047873_43-0" class="reference"><a href="#cite_note-pmid2047873-43">&#91;43&#93;</a></sup> In 1995, Venter, <a href="/wiki/Hamilton_O._Smith" title="Hamilton O. Smith">Hamilton Smith</a>, and colleagues at <a href="/wiki/The_Institute_for_Genomic_Research" class="mw-redirect" title="The Institute for Genomic Research">The Institute for Genomic Research</a> (TIGR) published the first complete genome of a free-living organism, the bacterium <i><a href="/wiki/Haemophilus_influenzae" title="Haemophilus influenzae">Haemophilus influenzae</a></i>.  The circular chromosome contains 1,830,137 bases and its publication in the journal Science<sup id="cite_ref-44" class="reference"><a href="#cite_note-44">&#91;44&#93;</a></sup> marked the first published use of whole-genome shotgun sequencing, eliminating the need for initial mapping efforts.
</p><p>By 2001, shotgun sequencing methods had been used to produce a draft sequence of the human genome.<sup id="cite_ref-Lander_2001_45-0" class="reference"><a href="#cite_note-Lander_2001-45">&#91;45&#93;</a></sup><sup id="cite_ref-Venter_2001_46-0" class="reference"><a href="#cite_note-Venter_2001-46">&#91;46&#93;</a></sup>
</p>
<h3><span id="High-throughput_sequencing_.28HTS.29_methods"></span><span class="mw-headline" id="High-throughput_sequencing_(HTS)_methods">High-throughput sequencing (HTS) methods</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=DNA_sequencing&amp;action=edit&amp;section=14" title="Edit section: High-throughput sequencing (HTS) methods">edit</a><span class="mw-editsection-bracket">]</span></span></h3>
<p>Several new methods for DNA sequencing were developed in the mid to late 1990s and were implemented in commercial <a href="/wiki/DNA_sequencers" class="mw-redirect" title="DNA sequencers">DNA sequencers</a> by the year 2000. Together these were called the "next-generation" or "second-generation" sequencing (NGS) methods, in order to distinguish them from the aforementioned earlier methods, like Sanger Sequencing. In contrast to the first generation of sequencing, NGS technology is typically characterized by being highly scalable, allowing the entire genome to be sequenced at once. Usually, this is accomplished by fragmenting the genome into small pieces, randomly sampling for a fragment, and sequencing it using one of a variety of technologies, such as those described below. An entire genome is possible because multiple fragments are sequenced at once (giving it the name "massively parallel" sequencing) in an automated process.
</p><p>NGS technology has tremendously empowered researchers to look for insights into health, anthropologists to investigate human origins, and is catalyzing the "<a href="/wiki/Personalized_medicine" title="Personalized medicine">Personalized Medicine</a>" movement. However, it has also opened the door to more room for error. There are many software tools to carry out the computational analysis of NGS data, each with its own algorithm. Even the parameters within one software package can change the outcome of the analysis. In addition, the large quantities of data produced by DNA sequencing have also required development of new methods and programs for sequence analysis. Several efforts to develop standards in the NGS field have been attempted to address these challenges, most of which have been small-scale efforts arising from individual labs. Most recently, a large, organized, FDA-funded effort has culminated in the <a href="/wiki/BioCompute_Object" title="BioCompute Object">BioCompute</a> standard.
</p><p>On 26 October 1990, <a href="/wiki/Roger_Tsien" class="mw-redirect" title="Roger Tsien">Roger Tsien</a>, Pepi Ross, Margaret Fahnestock and Allan J Johnston filed a patent describing stepwise ("base-by-base") sequencing with removable 3' blockers on DNA arrays (blots and single DNA molecules).<sup id="cite_ref-TsienPatent_47-0" class="reference"><a href="#cite_note-TsienPatent-47">&#91;47&#93;</a></sup>
In 1996, <a href="/wiki/P%C3%A5l_Nyr%C3%A9n" title="Pål Nyrén">Pål Nyrén</a> and his student <a href="/wiki/Mostafa_Ronaghi" title="Mostafa Ronaghi">Mostafa Ronaghi</a> at the Royal Institute of Technology in <a href="/wiki/Stockholm" title="Stockholm">Stockholm</a> published their method of <a href="/wiki/Pyrosequencing" title="Pyrosequencing">pyrosequencing</a>.<sup id="cite_ref-Ronaghi_48-0" class="reference"><a href="#cite_note-Ronaghi-48">&#91;48&#93;</a></sup>
</p><p>On 1 April 1997, Pascal Mayer and Laurent Farinelli submitted patents to the World Intellectual Property Organization describing DNA colony sequencing.<sup id="cite_ref-DNA_colony_patents_49-0" class="reference"><a href="#cite_note-DNA_colony_patents-49">&#91;49&#93;</a></sup> The DNA sample preparation and random surface-<a href="/wiki/Polymerase_chain_reaction" title="Polymerase chain reaction">polymerase chain reaction</a> (PCR) arraying methods described in this patent, coupled to Roger Tsien et al.'s "base-by-base" sequencing method, is now implemented in <a href="/wiki/Illumina_(company)" class="mw-redirect" title="Illumina (company)">Illumina</a>'s Hi-Seq genome sequencers.
</p><p>In 1998, Phil Green and Brent Ewing of the University of Washington described their <a href="/wiki/Phred_quality_score" title="Phred quality score">phred quality score</a> for sequencer data analysis,<sup id="cite_ref-50" class="reference"><a href="#cite_note-50">&#91;50&#93;</a></sup> a landmark analysis technique that gained widespread adoption, and which is still the most common metric for assessing the accuracy of a sequencing platform.<sup id="cite_ref-51" class="reference"><a href="#cite_note-51">&#91;51&#93;</a></sup>
</p><p>Lynx Therapeutics published and marketed <a href="/wiki/Massively_parallel_signature_sequencing" title="Massively parallel signature sequencing">massively parallel signature sequencing</a> (MPSS), in 2000. This method incorporated a parallelized, adapter/ligation-mediated, bead-based sequencing technology and served as the first commercially available "next-generation" sequencing method, though no <a href="/wiki/DNA_sequencers" class="mw-redirect" title="DNA sequencers">DNA sequencers</a> were sold to independent laboratories.<sup id="cite_ref-Brenner_2000_52-0" class="reference"><a href="#cite_note-Brenner_2000-52">&#91;52&#93;</a></sup>
</p>
<h2><span class="mw-headline" id="Basic_methods">Basic methods</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=DNA_sequencing&amp;action=edit&amp;section=15" title="Edit section: Basic methods">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<h3><span class="mw-headline" id="Maxam-Gilbert_sequencing">Maxam-Gilbert sequencing</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=DNA_sequencing&amp;action=edit&amp;section=16" title="Edit section: Maxam-Gilbert sequencing">edit</a><span class="mw-editsection-bracket">]</span></span></h3>
<div role="note" class="hatnote navigation-not-searchable">Main article: <a href="/wiki/Maxam-Gilbert_sequencing" class="mw-redirect" title="Maxam-Gilbert sequencing">Maxam-Gilbert sequencing</a></div>
<p><a href="/wiki/Allan_Maxam" title="Allan Maxam">Allan Maxam</a> and <a href="/wiki/Walter_Gilbert" title="Walter Gilbert">Walter Gilbert</a> published a DNA sequencing method in 1977 based on chemical modification of DNA and subsequent cleavage at specific bases.<sup id="cite_ref-Maxam77_33-1" class="reference"><a href="#cite_note-Maxam77-33">&#91;33&#93;</a></sup>  Also known as chemical sequencing, this method allowed purified samples of double-stranded DNA to be used without further cloning.  This method's use of radioactive labeling and its technical complexity discouraged extensive use after refinements in the Sanger methods had been made.
</p><p>Maxam-Gilbert sequencing requires radioactive labeling at one 5' end of the DNA and purification of the DNA fragment to be sequenced. Chemical treatment then generates breaks at a small proportion of one or two of the four nucleotide bases in each of four reactions (G, A+G, C, C+T). The concentration of the modifying chemicals is controlled to introduce on average one modification per DNA molecule. Thus a series of labeled fragments is generated, from the radiolabeled end to the first "cut" site in each molecule. The fragments in the four reactions are electrophoresed side by side in denaturing acrylamide gels for size separation. To visualize the fragments, the gel is exposed to X-ray film for autoradiography, yielding a series of dark bands each corresponding to a radiolabeled DNA fragment, from which the sequence may be inferred.<sup id="cite_ref-Maxam77_33-2" class="reference"><a href="#cite_note-Maxam77-33">&#91;33&#93;</a></sup>
</p>
<h3><span class="mw-headline" id="Chain-termination_methods">Chain-termination methods</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=DNA_sequencing&amp;action=edit&amp;section=17" title="Edit section: Chain-termination methods">edit</a><span class="mw-editsection-bracket">]</span></span></h3>
<div role="note" class="hatnote navigation-not-searchable">Main article: <a href="/wiki/Sanger_sequencing" title="Sanger sequencing">Sanger sequencing</a></div>
<p>The <a href="/wiki/Sanger_sequencing" title="Sanger sequencing">chain-termination method</a> developed by <a href="/wiki/Frederick_Sanger" title="Frederick Sanger">Frederick Sanger</a> and coworkers in 1977 soon became the method of choice, owing to its relative ease and reliability.<sup id="cite_ref-Sanger1977_32-1" class="reference"><a href="#cite_note-Sanger1977-32">&#91;32&#93;</a></sup><sup id="cite_ref-Sanger75_53-0" class="reference"><a href="#cite_note-Sanger75-53">&#91;53&#93;</a></sup> When invented, the chain-terminator method used fewer toxic chemicals and lower amounts of radioactivity than the Maxam and Gilbert method.  Because of its comparative ease, the Sanger method was soon automated and was the method used in the first generation of <a href="/wiki/DNA_sequencer" title="DNA sequencer">DNA sequencers</a>.
</p><p>Sanger sequencing is the method which prevailed from the 1980s until the mid-2000s. Over that period, great advances were made in the technique, such as fluorescent labelling, capillary electrophoresis, and general automation. These developments allowed much more efficient sequencing, leading to lower costs. The Sanger method, in mass production form, is the technology which produced the <a href="/wiki/Human_Genome_Project" title="Human Genome Project">first human genome</a> in 2001, ushering in the age of <a href="/wiki/Genomics" title="Genomics">genomics</a>. However, later in the decade, radically different approaches reached the market, bringing the cost per genome down from $100 million in 2001 to $10,000 in 2011.<sup id="cite_ref-54" class="reference"><a href="#cite_note-54">&#91;54&#93;</a></sup>
</p>
<h2><span class="mw-headline" id="Large-scale_sequencing_and_de_novo_sequencing">Large-scale sequencing and <i>de novo</i> sequencing</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=DNA_sequencing&amp;action=edit&amp;section=18" title="Edit section: Large-scale sequencing and de novo sequencing">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<div class="thumb tright"><div class="thumbinner" style="width:222px;"><a href="/wiki/File:DNA_Sequencing_gDNA_libraries.jpg" class="image"><img alt="" src="//upload.wikimedia.org/wikipedia/commons/thumb/6/60/DNA_Sequencing_gDNA_libraries.jpg/220px-DNA_Sequencing_gDNA_libraries.jpg" decoding="async" width="220" height="293" class="thumbimage" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/6/60/DNA_Sequencing_gDNA_libraries.jpg/330px-DNA_Sequencing_gDNA_libraries.jpg 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/6/60/DNA_Sequencing_gDNA_libraries.jpg/440px-DNA_Sequencing_gDNA_libraries.jpg 2x" data-file-width="540" data-file-height="720" /></a>  <div class="thumbcaption"><div class="magnify"><a href="/wiki/File:DNA_Sequencing_gDNA_libraries.jpg" class="internal" title="Enlarge"></a></div>Genomic DNA is fragmented into random pieces and cloned as a bacterial library. DNA from individual bacterial clones is sequenced and the sequence is assembled by using overlapping DNA regions.(click to expand)</div></div></div><p> Large-scale sequencing often aims at sequencing very long DNA pieces, such as whole <a href="/wiki/Chromosome" title="Chromosome">chromosomes</a>, although large-scale sequencing can also be used to generate very large numbers of short sequences, such as found in <a href="/wiki/Phage_display" title="Phage display">phage display</a>. For longer targets such as chromosomes, common approaches consist of cutting (with <a href="/wiki/Restriction_enzyme" title="Restriction enzyme">restriction enzymes</a>) or shearing (with mechanical forces) large DNA fragments into shorter DNA fragments. The fragmented DNA may then be <a href="/wiki/Clone_(genetics)" class="mw-redirect" title="Clone (genetics)">cloned</a> into a <a href="/wiki/Vector_DNA" class="mw-redirect" title="Vector DNA">DNA vector</a> and amplified in a bacterial host such as <i><a href="/wiki/Escherichia_coli" title="Escherichia coli">Escherichia coli</a></i>. Short DNA fragments purified from individual bacterial colonies are individually sequenced and <a href="/wiki/Sequence_assembly" title="Sequence assembly">assembled electronically</a> into one long, contiguous sequence. Studies have shown that adding a size selection step to collect DNA fragments of uniform size can improve sequencing efficiency and accuracy of the genome assembly. In these studies, automated sizing has proven to be more reproducible and precise than manual gel sizing.<sup id="cite_ref-pmid23147856_55-0" class="reference"><a href="#cite_note-pmid23147856-55">&#91;55&#93;</a></sup><sup id="cite_ref-pmid22713159_56-0" class="reference"><a href="#cite_note-pmid22713159-56">&#91;56&#93;</a></sup><sup id="cite_ref-pmid22675423_57-0" class="reference"><a href="#cite_note-pmid22675423-57">&#91;57&#93;</a></sup>
</p><p>The term "<i>de novo</i> sequencing" specifically refers to methods used to determine the sequence of DNA with no previously known sequence. <i>De novo</i> translates from Latin as "from the beginning". Gaps in the assembled sequence may be filled by <a href="/wiki/Primer_walking" title="Primer walking">primer walking</a>. The different strategies have different tradeoffs in speed and accuracy; <a href="/wiki/Shotgun_sequencing" title="Shotgun sequencing">shotgun methods</a> are often used for sequencing large genomes, but its assembly is complex and difficult, particularly with <a href="/wiki/Microsatellite_(genetics)" class="mw-redirect" title="Microsatellite (genetics)">sequence repeats</a> often causing gaps in genome assembly.
</p><p>Most sequencing approaches use an <i>in vitro</i> cloning step to amplify individual DNA molecules, because their molecular detection methods are not sensitive enough for single molecule sequencing. Emulsion PCR<sup id="cite_ref-Williams2006ePCR_58-0" class="reference"><a href="#cite_note-Williams2006ePCR-58">&#91;58&#93;</a></sup> isolates individual DNA molecules along with primer-coated beads in aqueous droplets within an oil phase. A <a href="/wiki/Polymerase_chain_reaction" title="Polymerase chain reaction">polymerase chain reaction</a> (PCR) then coats each bead with clonal copies of the DNA molecule followed by immobilization for later sequencing. Emulsion PCR is used in the methods developed by Marguilis et al. (commercialized by <a href="/wiki/454_Life_Sciences" title="454 Life Sciences">454 Life Sciences</a>), Shendure and Porreca et al. (also known as "<a href="/wiki/Polony_(biology)" title="Polony (biology)">polony sequencing</a>") and <a href="/wiki/ABI_Solid_Sequencing" title="ABI Solid Sequencing">SOLiD sequencing</a>, (developed by <a href="/wiki/Agencourt" title="Agencourt">Agencourt</a>, later <a href="/wiki/Applied_Biosystems" title="Applied Biosystems">Applied Biosystems</a>, now <a href="/wiki/Life_Technologies_(Thermo_Fisher_Scientific)" title="Life Technologies (Thermo Fisher Scientific)">Life Technologies</a>).<sup id="cite_ref-Margulies_2005_59-0" class="reference"><a href="#cite_note-Margulies_2005-59">&#91;59&#93;</a></sup><sup id="cite_ref-polony_sequencing_60-0" class="reference"><a href="#cite_note-polony_sequencing-60">&#91;60&#93;</a></sup><sup id="cite_ref-solid_sequencing_61-0" class="reference"><a href="#cite_note-solid_sequencing-61">&#91;61&#93;</a></sup> Emulsion PCR is also used in the GemCode and Chromium platforms developed by <a href="/wiki/10x_Genomics" title="10x Genomics">10x Genomics</a>.<sup id="cite_ref-10x-epcr_62-0" class="reference"><a href="#cite_note-10x-epcr-62">&#91;62&#93;</a></sup>
</p>
<h3><span class="mw-headline" id="Shotgun_sequencing">Shotgun sequencing</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=DNA_sequencing&amp;action=edit&amp;section=19" title="Edit section: Shotgun sequencing">edit</a><span class="mw-editsection-bracket">]</span></span></h3>
<div role="note" class="hatnote navigation-not-searchable">Main article: <a href="/wiki/Shotgun_sequencing" title="Shotgun sequencing">Shotgun sequencing</a></div>
<p>Shotgun sequencing is a sequencing method designed for analysis of DNA sequences longer than 1000 base pairs, up to and including entire chromosomes.  This method requires the target DNA to be broken into random fragments.  After sequencing individual fragments, the sequences can be reassembled on the basis of their overlapping regions.<sup id="cite_ref-63" class="reference"><a href="#cite_note-63">&#91;63&#93;</a></sup>
</p>
<h2><span class="mw-headline" id="High-throughput_methods">High-throughput methods</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=DNA_sequencing&amp;action=edit&amp;section=20" title="Edit section: High-throughput methods">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<div class="thumb tright"><div class="thumbinner" style="width:222px;"><a href="/wiki/File:Mapping_Reads.png" class="image"><img alt="" src="//upload.wikimedia.org/wikipedia/commons/thumb/2/2e/Mapping_Reads.png/220px-Mapping_Reads.png" decoding="async" width="220" height="159" class="thumbimage" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/2/2e/Mapping_Reads.png/330px-Mapping_Reads.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/2/2e/Mapping_Reads.png/440px-Mapping_Reads.png 2x" data-file-width="1283" data-file-height="927" /></a>  <div class="thumbcaption"><div class="magnify"><a href="/wiki/File:Mapping_Reads.png" class="internal" title="Enlarge"></a></div>Multiple, fragmented sequence reads must be assembled together on the basis of their overlapping areas.</div></div></div>
<p>High-throughput sequencing, which includes next-generation "short-read" and third-generation "long-read" sequencing methods,<sup id="cite_ref-64" class="reference"><a href="#cite_note-64">&#91;nt 1&#93;</a></sup> applies to exome sequencing, genome sequencing, genome resequencing, <a href="/wiki/Transcriptome" title="Transcriptome">transcriptome</a> profiling (<a href="/wiki/RNA-Seq" title="RNA-Seq">RNA-Seq</a>), DNA-protein interactions (<a href="/wiki/ChIP-sequencing" title="ChIP-sequencing">ChIP-sequencing</a>), and <a href="/wiki/Epigenome" title="Epigenome">epigenome</a> characterization.<sup id="cite_ref-pmid19900591_65-0" class="reference"><a href="#cite_note-pmid19900591-65">&#91;64&#93;</a></sup> Resequencing is necessary, because the genome of a single individual of a species will not indicate all of the genome variations among other individuals of the same species.
</p><p>The high demand for low-cost sequencing has driven the development of high-throughput sequencing technologies that <a href="/wiki/Multiplex_(assay)" title="Multiplex (assay)">parallelize</a> the sequencing process, producing thousands or millions of sequences concurrently.<sup id="cite_ref-pmid23856935_66-0" class="reference"><a href="#cite_note-pmid23856935-66">&#91;65&#93;</a></sup><sup id="cite_ref-hall2007_67-0" class="reference"><a href="#cite_note-hall2007-67">&#91;66&#93;</a></sup><sup id="cite_ref-church2006_68-0" class="reference"><a href="#cite_note-church2006-68">&#91;67&#93;</a></sup>  High-throughput sequencing technologies are intended to lower the cost of DNA sequencing beyond what is possible with standard dye-terminator methods.<sup id="cite_ref-pmid18165802_69-0" class="reference"><a href="#cite_note-pmid18165802-69">&#91;68&#93;</a></sup>  In ultra-high-throughput sequencing as many as 500,000 sequencing-by-synthesis operations may be run in parallel.<sup id="cite_ref-kalb1992_70-0" class="reference"><a href="#cite_note-kalb1992-70">&#91;69&#93;</a></sup><sup id="cite_ref-tenBosch2008_71-0" class="reference"><a href="#cite_note-tenBosch2008-71">&#91;70&#93;</a></sup><sup id="cite_ref-Tucker2009_72-0" class="reference"><a href="#cite_note-Tucker2009-72">&#91;71&#93;</a></sup>  Such technologies led to the ability to sequence an entire human genome in as little as one day.<sup id="cite_ref-:2_73-0" class="reference"><a href="#cite_note-:2-73">&#91;72&#93;</a></sup>  As of 2019<sup class="plainlinks noexcerpt noprint asof-tag update" style="display:none;"><a class="external text" href="https://en.wikipedia.org/w/index.php?title=DNA_sequencing&amp;action=edit">&#91;update&#93;</a></sup>, corporate leaders in the development of high-throughput sequencing products included <a href="/wiki/Illumina,_Inc." title="Illumina, Inc.">Illumina</a>, <a href="/wiki/Qiagen" title="Qiagen">Qiagen</a> and <a href="/wiki/Thermo_Fisher_Scientific" title="Thermo Fisher Scientific">ThermoFisher Scientific</a>.<sup id="cite_ref-:2_73-1" class="reference"><a href="#cite_note-:2-73">&#91;72&#93;</a></sup>
</p>
<table class="wikitable" style="font-size:0.9em;">
<caption>Comparison of high-throughput sequencing methods<sup id="cite_ref-quail2012_74-0" class="reference"><a href="#cite_note-quail2012-74">&#91;73&#93;</a></sup><sup id="cite_ref-lin2012_75-0" class="reference"><a href="#cite_note-lin2012-75">&#91;74&#93;</a></sup>
</caption>
<tbody><tr>
<th>Method</th>
<th><b>Read length</b></th>
<th><b>Accuracy (single read not consensus)</b></th>
<th><b>Reads per run</b></th>
<th><b>Time per run</b></th>
<th><b>Cost per 1 billion bases (in US$)</b></th>
<th><b>Advantages</b></th>
<th><b>Disadvantages</b>
</th></tr>
<tr>
<td><b>Single-molecule real-time sequencing (Pacific Biosciences)</b></td>
<td>30,000 bp (<a href="/wiki/N50_statistic" class="mw-redirect" title="N50 statistic">N50</a>);
<p>maximum read length &gt;100,000 bases<sup id="cite_ref-sequel21_76-0" class="reference"><a href="#cite_note-sequel21-76">&#91;75&#93;</a></sup><sup id="cite_ref-autogenerated1_77-0" class="reference"><a href="#cite_note-autogenerated1-77">&#91;76&#93;</a></sup><sup id="cite_ref-78" class="reference"><a href="#cite_note-78">&#91;77&#93;</a></sup>
</p>
</td>
<td>87% raw-read accuracy<sup id="cite_ref-pmid23644548_79-0" class="reference"><a href="#cite_note-pmid23644548-79">&#91;78&#93;</a></sup></td>
<td>4,000,000 per Sequel 2 SMRT cell, 100–200 gigabases<sup id="cite_ref-sequel21_76-1" class="reference"><a href="#cite_note-sequel21-76">&#91;75&#93;</a></sup><sup id="cite_ref-flxlexblog.wordpress.com_80-0" class="reference"><a href="#cite_note-flxlexblog.wordpress.com-80">&#91;79&#93;</a></sup><sup id="cite_ref-rasko2011_81-0" class="reference"><a href="#cite_note-rasko2011-81">&#91;80&#93;</a></sup></td>
<td>30 minutes to 20 hours<sup id="cite_ref-sequel21_76-2" class="reference"><a href="#cite_note-sequel21-76">&#91;75&#93;</a></sup><sup id="cite_ref-tran2012_82-0" class="reference"><a href="#cite_note-tran2012-82">&#91;81&#93;</a></sup></td>
<td>$7.2-$43.3
</td>
<td>Fast. Detects 4mC, 5mC, 6mA.<sup id="cite_ref-83" class="reference"><a href="#cite_note-83">&#91;82&#93;</a></sup></td>
<td>Moderate throughput. Equipment can be very expensive.
</td></tr>
<tr>
<td><b>Ion semiconductor (Ion Torrent sequencing)</b></td>
<td>up to 600 bp<sup id="cite_ref-84" class="reference"><a href="#cite_note-84">&#91;83&#93;</a></sup></td>
<td>99.6%<sup id="cite_ref-85" class="reference"><a href="#cite_note-85">&#91;84&#93;</a></sup></td>
<td>up to 80 million</td>
<td>2 hours</td>
<td>$66.8-$950</td>
<td>Less expensive equipment. Fast.</td>
<td>Homopolymer errors.
</td></tr>
<tr>
<td><b>Pyrosequencing (454)</b></td>
<td>700 bp</td>
<td>99.9%</td>
<td>1 million</td>
<td>24 hours</td>
<td>$10,000</td>
<td>Long read size. Fast.</td>
<td>Runs are expensive. Homopolymer errors.
</td></tr>
<tr>
<td><b>Sequencing by synthesis (Illumina)</b></td>
<td>MiniSeq, NextSeq: 75–300 bp;
<p>MiSeq: 50–600 bp;
</p><p>HiSeq 2500: 50–500 bp;
</p><p>HiSeq 3/4000: 50–300 bp;
</p><p>HiSeq X: 300 bp
</p>
</td>
<td>99.9% (Phred30)</td>
<td>MiniSeq/MiSeq: 1–25 Million;
<p>NextSeq: 130-00 Million;
</p><p>HiSeq 2500: 300 million – 2 billion;
</p><p>HiSeq 3/4000 2.5 billion;
</p><p>HiSeq X: 3 billion
</p>
</td>
<td>1 to 11 days, depending upon sequencer and specified read length<sup id="cite_ref-vliet2010_86-0" class="reference"><a href="#cite_note-vliet2010-86">&#91;85&#93;</a></sup></td>
<td>$5 to $150</td>
<td>Potential for high sequence yield, depending upon sequencer model and desired application.</td>
<td>Equipment can be very expensive. Requires high concentrations of DNA.
</td></tr>
<tr>
<td><b>Combinatorial probe anchor synthesis (cPAS-  BGI/MGI)</b>
</td>
<td>BGISEQ-50: 35-50bp;
<p>MGISEQ 200: 50-200bp;
</p><p>BGISEQ-500, MGISEQ-2000: 50-300bp<sup id="cite_ref-87" class="reference"><a href="#cite_note-87">&#91;86&#93;</a></sup>
</p>
</td>
<td>99.9% (Phred30)
</td>
<td>BGISEQ-50: 160M;
<p>MGISEQ 200: 300M;
</p><p>BGISEQ-500: 1300M per flow cell;
</p><p>MGISEQ-2000: 375M FCS flow cell, 1500M FCL flow cell per flow cell.
</p>
</td>
<td>1 to 9 days depending on instrument, read length and number of flow cells run at a time.
</td>
<td>$5- $120
</td>
<td>
</td>
<td>
</td></tr>
<tr>
<td><b>Sequencing by ligation (SOLiD sequencing)</b></td>
<td>50+35 or 50+50 bp</td>
<td>99.9%</td>
<td>1.2 to 1.4 billion</td>
<td>1 to 2 weeks</td>
<td>$60-130</td>
<td>Low cost per base.</td>
<td>Slower than other methods. Has issues sequencing palindromic sequences.<sup id="cite_ref-Yu-Feng_Huang,_Sheng-Chung_Chen,_Yih-Shien_Chiang,_Tzu-Han_Chen_&amp;_Kuo-Ping_Chiu_2012_S10_88-0" class="reference"><a href="#cite_note-Yu-Feng_Huang,_Sheng-Chung_Chen,_Yih-Shien_Chiang,_Tzu-Han_Chen_&amp;_Kuo-Ping_Chiu_2012_S10-88">&#91;87&#93;</a></sup>
</td></tr>
<tr>
<td><b>Nanopore Sequencing</b></td>
<td>Dependent on library preparation, not the device, so user chooses read length (up to 2,272,580 bp reported<sup id="cite_ref-89" class="reference"><a href="#cite_note-89">&#91;88&#93;</a></sup>).</td>
<td>~92–97% single read</td>
<td>dependent on read length selected by user</td>
<td>data streamed in real time. Choose 1 min to 48 hrs</td>
<td>$7-100</td>
<td>Longest individual reads. Accessible user community. Portable (Palm sized).</td>
<td>Lower throughput than other machines, Single read accuracy in 90s.
</td></tr>
<tr>
<td><b>GenapSys Sequencing</b>
</td>
<td>Around 150 bp single-end
</td>
<td>99.9% (Phred30)
</td>
<td>1 to 16 million
</td>
<td>Around 24 hours
</td>
<td>$667
</td>
<td>Low-cost of instrument ($10,000)
</td>
<td>
</td></tr>
<tr>
<td><b>Chain termination (Sanger sequencing)</b></td>
<td>400 to 900 bp</td>
<td>99.9%</td>
<td>N/A</td>
<td>20 minutes to 3 hours</td>
<td>$2,400,000</td>
<td>Useful for many applications.</td>
<td>More expensive and impractical for larger sequencing projects. This method also requires the time-consuming step of plasmid cloning or PCR.
</td></tr></tbody></table>
<h3><span class="mw-headline" id="Long-read_sequencing_methods">Long-read sequencing methods</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=DNA_sequencing&amp;action=edit&amp;section=21" title="Edit section: Long-read sequencing methods">edit</a><span class="mw-editsection-bracket">]</span></span></h3>
<div role="note" class="hatnote navigation-not-searchable">Further information: <a href="/wiki/Long-read_sequencing" class="mw-redirect" title="Long-read sequencing">Long-read sequencing</a></div>
<h4><span id="Single_molecule_real_time_.28SMRT.29_sequencing"></span><span class="mw-headline" id="Single_molecule_real_time_(SMRT)_sequencing">Single molecule real time (SMRT) sequencing</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=DNA_sequencing&amp;action=edit&amp;section=22" title="Edit section: Single molecule real time (SMRT) sequencing">edit</a><span class="mw-editsection-bracket">]</span></span></h4>
<div role="note" class="hatnote navigation-not-searchable">Main article: <a href="/wiki/Single_molecule_real_time_sequencing" class="mw-redirect" title="Single molecule real time sequencing">Single molecule real time sequencing</a></div>
<p>SMRT sequencing is based on the sequencing by synthesis approach. The DNA is synthesized in zero-mode wave-guides (ZMWs)&#160;– small well-like containers with the capturing tools located at the bottom of the well. The sequencing is performed with use of unmodified polymerase (attached to the ZMW bottom) and fluorescently labelled nucleotides flowing freely in the solution. The wells are constructed in a way that only the fluorescence occurring by the bottom of the well is detected. The fluorescent label is detached from the nucleotide upon its incorporation into the DNA strand, leaving an unmodified DNA strand. According to <a href="/wiki/Pacific_Biosciences" title="Pacific Biosciences">Pacific Biosciences</a> (PacBio), the SMRT technology developer, this methodology allows detection of nucleotide modifications (such as cytosine methylation). This happens through the observation of polymerase kinetics. This approach allows reads of 20,000 nucleotides or more, with average read lengths of 5 kilobases.<sup id="cite_ref-flxlexblog.wordpress.com_80-1" class="reference"><a href="#cite_note-flxlexblog.wordpress.com-80">&#91;79&#93;</a></sup><sup id="cite_ref-90" class="reference"><a href="#cite_note-90">&#91;89&#93;</a></sup> In 2015, Pacific Biosciences announced the launch of a new sequencing instrument called the Sequel System, with 1 million ZMWs compared to 150,000 ZMWs in the PacBio RS II instrument.<sup id="cite_ref-91" class="reference"><a href="#cite_note-91">&#91;90&#93;</a></sup><sup id="cite_ref-92" class="reference"><a href="#cite_note-92">&#91;91&#93;</a></sup> SMRT sequencing is referred to as "<a href="/wiki/Third-generation_sequencing" title="Third-generation sequencing">third-generation</a>" or "long-read" sequencing.
</p>
<h4><span class="mw-headline" id="Nanopore_DNA_sequencing">Nanopore DNA sequencing</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=DNA_sequencing&amp;action=edit&amp;section=23" title="Edit section: Nanopore DNA sequencing">edit</a><span class="mw-editsection-bracket">]</span></span></h4>
<div role="note" class="hatnote navigation-not-searchable">Main article: <a href="/wiki/Nanopore_sequencing" title="Nanopore sequencing">Nanopore sequencing</a></div>
<p>The DNA passing through the nanopore changes its ion current. This change is dependent on the shape, size and length of the DNA sequence. Each type of the nucleotide blocks the ion flow through the pore for a different period of time. The method does not require modified nucleotides and is performed in real time. Nanopore sequencing is referred to as "<a href="/wiki/Third-generation_sequencing" title="Third-generation sequencing">third-generation</a>" or "long-read" sequencing, along with SMRT sequencing.
</p><p>Early industrial research into this method was based on a technique called 'exonuclease sequencing', where the readout of electrical signals occurred as nucleotides passed by <a href="/wiki/Hemolysin" title="Hemolysin">alpha(α)-hemolysin</a> pores covalently bound with <a href="/wiki/Cyclodextrin" title="Cyclodextrin">cyclodextrin</a>.<sup id="cite_ref-93" class="reference"><a href="#cite_note-93">&#91;92&#93;</a></sup> However the subsequent commercial method, 'strand sequencing', sequenced DNA bases in an intact strand.
</p><p>Two main areas of nanopore sequencing in development are solid state nanopore sequencing, and protein based nanopore sequencing. Protein nanopore sequencing utilizes membrane protein complexes such as α-hemolysin, MspA (<i><a href="/wiki/Mycobacterium_smegmatis" title="Mycobacterium smegmatis">Mycobacterium smegmatis</a></i> Porin A) or CssG, which show great promise given their ability to distinguish between individual and groups of nucleotides.<sup id="cite_ref-Torre_2012_94-0" class="reference"><a href="#cite_note-Torre_2012-94">&#91;93&#93;</a></sup> In contrast, solid-state nanopore sequencing utilizes synthetic materials such as silicon nitride and aluminum oxide and it is preferred for its superior mechanical ability and thermal and chemical stability.<sup id="cite_ref-Pathak_2012_95-0" class="reference"><a href="#cite_note-Pathak_2012-95">&#91;94&#93;</a></sup> The fabrication method is essential for this type of sequencing given that the nanopore array can contain hundreds of pores with diameters smaller than eight nanometers.<sup id="cite_ref-Torre_2012_94-1" class="reference"><a href="#cite_note-Torre_2012-94">&#91;93&#93;</a></sup>
</p><p>The concept originated from the idea that single stranded DNA or RNA molecules can be electrophoretically driven in a strict linear sequence through a biological pore that can be less than eight nanometers, and can be detected given that the molecules release an ionic current while moving through the pore.  The pore contains a detection region capable of recognizing different bases, with each base generating various time specific signals corresponding to the sequence of bases as they cross the pore which are then evaluated.<sup id="cite_ref-Pathak_2012_95-1" class="reference"><a href="#cite_note-Pathak_2012-95">&#91;94&#93;</a></sup> Precise control over the DNA transport through the pore is crucial for success. Various enzymes such as exonucleases and polymerases have been used to moderate this process by positioning them near the pore's entrance.<sup id="cite_ref-Korlach_2008_96-0" class="reference"><a href="#cite_note-Korlach_2008-96">&#91;95&#93;</a></sup>
</p>
<h3><span class="mw-headline" id="Short-read_sequencing_methods">Short-read sequencing methods <span id="Next-generation_methods"></span></span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=DNA_sequencing&amp;action=edit&amp;section=24" title="Edit section: Short-read sequencing methods">edit</a><span class="mw-editsection-bracket">]</span></span></h3>
<div role="note" class="hatnote navigation-not-searchable">Further information: <a href="/wiki/Short-read_sequencing" class="mw-redirect" title="Short-read sequencing">Short-read sequencing</a></div> 
<h4><span id="Massively_parallel_signature_sequencing_.28MPSS.29"></span><span class="mw-headline" id="Massively_parallel_signature_sequencing_(MPSS)">Massively parallel signature sequencing (MPSS)</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=DNA_sequencing&amp;action=edit&amp;section=25" title="Edit section: Massively parallel signature sequencing (MPSS)">edit</a><span class="mw-editsection-bracket">]</span></span></h4>
<p>The first of the high-throughput sequencing technologies, <a href="/wiki/Massively_parallel_signature_sequencing" title="Massively parallel signature sequencing">massively parallel signature sequencing</a> (or MPSS), was developed in the 1990s at Lynx Therapeutics, a company founded in 1992 by <a href="/wiki/Sydney_Brenner" title="Sydney Brenner">Sydney Brenner</a> and <a href="/wiki/Applied_Biosystems#History" title="Applied Biosystems">Sam Eletr</a>. MPSS was a bead-based method that used a complex approach of adapter ligation followed by adapter decoding, reading the sequence in increments of four nucleotides. This method made it susceptible to sequence-specific bias or loss of specific sequences. Because the technology was so complex, MPSS was only performed 'in-house' by Lynx Therapeutics and no DNA sequencing machines were sold to independent laboratories. Lynx Therapeutics merged with Solexa (later acquired by <a href="/wiki/Illumina_(company)" class="mw-redirect" title="Illumina (company)">Illumina</a>) in 2004, leading to the development of sequencing-by-synthesis, a simpler approach acquired from <a href="/wiki/Manteia_Predictive_Medicine" title="Manteia Predictive Medicine">Manteia Predictive Medicine</a>, which rendered MPSS obsolete. However, the essential properties of the MPSS output were typical of later high-throughput data types, including hundreds of thousands of short DNA sequences. In the case of MPSS, these were typically used for sequencing <a href="/wiki/CDNA" class="mw-redirect" title="CDNA">cDNA</a> for measurements of <a href="/wiki/Gene_expression" title="Gene expression">gene expression</a> levels.<sup id="cite_ref-Brenner_2000_52-1" class="reference"><a href="#cite_note-Brenner_2000-52">&#91;52&#93;</a></sup>
</p>
<h4><span class="mw-headline" id="Polony_sequencing">Polony sequencing</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=DNA_sequencing&amp;action=edit&amp;section=26" title="Edit section: Polony sequencing">edit</a><span class="mw-editsection-bracket">]</span></span></h4>
<div role="note" class="hatnote navigation-not-searchable">Main article: <a href="/wiki/Polony_sequencing" title="Polony sequencing">Polony sequencing</a></div>
<p>The <a href="/wiki/Polony_sequencing" title="Polony sequencing">polony sequencing</a> method, developed in the laboratory of <a href="/wiki/George_M._Church" class="mw-redirect" title="George M. Church">George M. Church</a> at Harvard, was among the first high-throughput sequencing systems and was used to sequence a full <i><a href="/wiki/E._coli" class="mw-redirect" title="E. coli">E. coli</a></i> genome in 2005.<sup id="cite_ref-Shendure2005_97-0" class="reference"><a href="#cite_note-Shendure2005-97">&#91;96&#93;</a></sup> It combined an in vitro paired-tag library with emulsion PCR, an automated microscope, and ligation-based sequencing chemistry to sequence an <i>E. coli</i> genome at an accuracy of &gt;99.9999% and a cost approximately 1/9 that of Sanger sequencing.<sup id="cite_ref-Shendure2005_97-1" class="reference"><a href="#cite_note-Shendure2005-97">&#91;96&#93;</a></sup> The technology was licensed to Agencourt Biosciences, subsequently spun out into Agencourt Personal Genomics, and eventually incorporated into the <a href="/wiki/Applied_Biosystems" title="Applied Biosystems">Applied Biosystems</a> SOLiD platform. Applied Biosystems was later acquired by <a href="/wiki/Life_Technologies_(Thermo_Fisher_Scientific)" title="Life Technologies (Thermo Fisher Scientific)">Life Technologies</a>, now part of <a href="/wiki/Thermo_Fisher_Scientific" title="Thermo Fisher Scientific">Thermo Fisher Scientific</a>.
</p>
<h4><span class="mw-headline" id="454_pyrosequencing">454 pyrosequencing</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=DNA_sequencing&amp;action=edit&amp;section=27" title="Edit section: 454 pyrosequencing">edit</a><span class="mw-editsection-bracket">]</span></span></h4>
<div role="note" class="hatnote navigation-not-searchable">Main article: <a href="/wiki/454_Life_Sciences#Technology" title="454 Life Sciences">454 Life Sciences §&#160;Technology</a></div>
<p>A parallelized version of <a href="/wiki/Pyrosequencing" title="Pyrosequencing">pyrosequencing</a> was developed by <a href="/wiki/454_Life_Sciences" title="454 Life Sciences">454 Life Sciences</a>, which has since been acquired by <a href="/wiki/Roche_Diagnostics" title="Roche Diagnostics">Roche Diagnostics</a>. The method amplifies DNA inside water droplets in an oil solution (emulsion PCR), with each droplet containing a single DNA template attached to a single primer-coated bead that then forms a clonal colony. The sequencing machine contains many <a href="/wiki/Picoliter" class="mw-redirect" title="Picoliter">picoliter</a>-volume wells each containing a single bead and sequencing enzymes. Pyrosequencing uses <a href="/wiki/Luciferase" title="Luciferase">luciferase</a> to generate light for detection of the individual nucleotides added to the nascent DNA, and the combined data are used to generate sequence <a href="/wiki/Read_(biology)" title="Read (biology)">reads</a>.<sup id="cite_ref-Margulies_2005_59-1" class="reference"><a href="#cite_note-Margulies_2005-59">&#91;59&#93;</a></sup> This technology provides intermediate read length and price per base compared to Sanger sequencing on one end and Solexa and SOLiD on the other.<sup id="cite_ref-pmid18165802_69-1" class="reference"><a href="#cite_note-pmid18165802-69">&#91;68&#93;</a></sup>
</p>
<h4><span id="Illumina_.28Solexa.29_sequencing"></span><span class="mw-headline" id="Illumina_(Solexa)_sequencing">Illumina (Solexa) sequencing</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=DNA_sequencing&amp;action=edit&amp;section=28" title="Edit section: Illumina (Solexa) sequencing">edit</a><span class="mw-editsection-bracket">]</span></span></h4>
<div role="note" class="hatnote navigation-not-searchable">Main article: <a href="/wiki/Illumina_dye_sequencing" title="Illumina dye sequencing">Illumina dye sequencing</a></div>
<p><a href="/wiki/Solexa" class="mw-redirect" title="Solexa">Solexa</a>, now part of <a href="/wiki/Illumina_(company)" class="mw-redirect" title="Illumina (company)">Illumina</a>, was founded by <a href="/wiki/Shankar_Balasubramanian" title="Shankar Balasubramanian">Shankar Balasubramanian</a> and <a href="/wiki/David_Klenerman" title="David Klenerman">David Klenerman</a> in 1998, and developed a sequencing method based on reversible dye-terminators technology, and engineered polymerases.<sup id="cite_ref-Bentley_2008_98-0" class="reference"><a href="#cite_note-Bentley_2008-98">&#91;97&#93;</a></sup> The reversible terminated chemistry concept was invented by Bruno Canard and Simon Sarfati at the Pasteur Institute in Paris.<sup id="cite_ref-99" class="reference"><a href="#cite_note-99">&#91;98&#93;</a></sup><sup id="cite_ref-100" class="reference"><a href="#cite_note-100">&#91;99&#93;</a></sup> It was developed internally at Solexa by those named on the relevant patents. In 2004, Solexa acquired the company <a href="/wiki/Manteia_Predictive_Medicine" title="Manteia Predictive Medicine">Manteia Predictive Medicine</a> in order to gain a massively parallel sequencing technology invented in 1997 by Pascal Mayer and Laurent Farinelli.<sup id="cite_ref-DNA_colony_patents_49-1" class="reference"><a href="#cite_note-DNA_colony_patents-49">&#91;49&#93;</a></sup> It is based on "DNA clusters" or "DNA colonies", which involves the clonal amplification of DNA on a surface. The cluster technology was co-acquired with Lynx Therapeutics of California. Solexa Ltd. later merged with Lynx to form Solexa Inc.
</p>
<div class="thumb tright"><div class="thumbinner" style="width:222px;"><a href="/wiki/File:Illumina_HiSeq_2500.jpg" class="image"><img alt="" src="//upload.wikimedia.org/wikipedia/commons/thumb/9/9a/Illumina_HiSeq_2500.jpg/220px-Illumina_HiSeq_2500.jpg" decoding="async" width="220" height="171" class="thumbimage" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/9/9a/Illumina_HiSeq_2500.jpg/330px-Illumina_HiSeq_2500.jpg 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/9/9a/Illumina_HiSeq_2500.jpg/440px-Illumina_HiSeq_2500.jpg 2x" data-file-width="2184" data-file-height="1700" /></a>  <div class="thumbcaption"><div class="magnify"><a href="/wiki/File:Illumina_HiSeq_2500.jpg" class="internal" title="Enlarge"></a></div>An Illumina HiSeq 2500 sequencer</div></div></div>
<div class="thumb tright"><div class="thumbinner" style="width:222px;"><a href="/wiki/File:Illumina_NovaSeq_6000_flow_cell.jpg" class="image"><img alt="" src="//upload.wikimedia.org/wikipedia/commons/thumb/d/d4/Illumina_NovaSeq_6000_flow_cell.jpg/220px-Illumina_NovaSeq_6000_flow_cell.jpg" decoding="async" width="220" height="103" class="thumbimage" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/d/d4/Illumina_NovaSeq_6000_flow_cell.jpg/330px-Illumina_NovaSeq_6000_flow_cell.jpg 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/d/d4/Illumina_NovaSeq_6000_flow_cell.jpg/440px-Illumina_NovaSeq_6000_flow_cell.jpg 2x" data-file-width="4160" data-file-height="1946" /></a>  <div class="thumbcaption"><div class="magnify"><a href="/wiki/File:Illumina_NovaSeq_6000_flow_cell.jpg" class="internal" title="Enlarge"></a></div>Illumina NovaSeq 6000 flow cell</div></div></div>
<p>In this method, DNA molecules and primers are first attached on a slide or flow cell and amplified with <a href="/wiki/Polymerase" title="Polymerase">polymerase</a> so that local clonal DNA colonies, later coined "DNA clusters", are formed. To determine the sequence, four types of reversible terminator bases (RT-bases) are added and non-incorporated nucleotides are washed away. A camera takes images of the <a href="/wiki/Fluorescent_labeling" class="mw-redirect" title="Fluorescent labeling">fluorescently labeled</a> nucleotides. Then the dye, along with the terminal 3' blocker, is chemically removed from the DNA, allowing for the next cycle to begin. Unlike pyrosequencing, the DNA chains are extended one nucleotide at a time and image acquisition can be performed at a delayed moment, allowing for very large arrays of DNA colonies to be captured by sequential images taken from a single camera.
</p>
<div class="thumb tright"><div class="thumbinner" style="width:222px;"><a href="/wiki/File:Illumina_MiSeq_sequencer.jpg" class="image"><img alt="" src="//upload.wikimedia.org/wikipedia/commons/thumb/7/7b/Illumina_MiSeq_sequencer.jpg/220px-Illumina_MiSeq_sequencer.jpg" decoding="async" width="220" height="165" class="thumbimage" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/7/7b/Illumina_MiSeq_sequencer.jpg/330px-Illumina_MiSeq_sequencer.jpg 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/7/7b/Illumina_MiSeq_sequencer.jpg/440px-Illumina_MiSeq_sequencer.jpg 2x" data-file-width="3264" data-file-height="2448" /></a>  <div class="thumbcaption"><div class="magnify"><a href="/wiki/File:Illumina_MiSeq_sequencer.jpg" class="internal" title="Enlarge"></a></div>An Illumina MiSeq sequencer</div></div></div>
<p>Decoupling the enzymatic reaction and the image capture allows for optimal throughput and theoretically unlimited sequencing capacity.  With an optimal configuration, the ultimately reachable instrument throughput is thus dictated solely by the analog-to-digital conversion rate of the camera, multiplied by the number of cameras and divided by the number of pixels per DNA colony required for visualizing them optimally (approximately 10 pixels/colony). In 2012, with cameras operating at more than 10&#160;MHz A/D conversion rates and available optics, fluidics and enzymatics, throughput can be multiples of 1 million nucleotides/second, corresponding roughly to 1 human genome equivalent at 1x <a href="/wiki/Coverage_(genetics)" title="Coverage (genetics)">coverage</a> per hour per instrument, and 1 human genome re-sequenced (at approx. 30x) per day per instrument (equipped with a single camera).<sup id="cite_ref-pmid18576944_101-0" class="reference"><a href="#cite_note-pmid18576944-101">&#91;100&#93;</a></sup>
</p>
<h4><span id="Combinatorial_probe_anchor_synthesis_.28cPAS.29"></span><span class="mw-headline" id="Combinatorial_probe_anchor_synthesis_(cPAS)">Combinatorial probe anchor synthesis (cPAS)</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=DNA_sequencing&amp;action=edit&amp;section=29" title="Edit section: Combinatorial probe anchor synthesis (cPAS)">edit</a><span class="mw-editsection-bracket">]</span></span></h4>
<p>This method is an upgraded modification to combinatorial probe anchor ligation technology (cPAL) described by <a href="/wiki/Complete_Genomics" title="Complete Genomics">Complete Genomics</a><sup id="cite_ref-:0_102-0" class="reference"><a href="#cite_note-:0-102">&#91;101&#93;</a></sup> which has since become part of Chinese genomics company <a href="/wiki/Beijing_Genomics_Institute" class="mw-redirect" title="Beijing Genomics Institute">BGI</a> in 2013.<sup id="cite_ref-103" class="reference"><a href="#cite_note-103">&#91;102&#93;</a></sup> The two companies have refined the technology to allow for longer read lengths, reaction time reductions and faster time to results. In addition, data are now generated as contiguous full-length reads in the standard FASTQ file format and can be used as-is in most short-read-based bioinformatics analysis pipelines.<sup id="cite_ref-:1_104-0" class="reference"><a href="#cite_note-:1-104">&#91;103&#93;</a></sup><sup class="noprint Inline-Template Template-Fact" style="white-space:nowrap;">&#91;<i><a href="/wiki/Wikipedia:Citation_needed" title="Wikipedia:Citation needed"><span title="This claim needs references to reliable sources. (July 2018)">citation needed</span></a></i>&#93;</sup>
</p><p>The two technologies that form the basis for this high-throughput sequencing technology are <a href="/wiki/DNA_nanoball_sequencing" title="DNA nanoball sequencing">DNA nanoballs</a> (DNB) and patterned arrays for nanoball attachment to a solid surface.<sup id="cite_ref-:0_102-1" class="reference"><a href="#cite_note-:0-102">&#91;101&#93;</a></sup> DNA nanoballs are simply formed by denaturing double stranded, adapter ligated libraries and ligating the forward strand only to a splint oligonucleotide to form a ssDNA circle. Faithful copies of the circles containing the DNA insert are produced utilizing Rolling Circle Amplification that generates approximately 300–500 copies. The long strand of ssDNA folds upon itself to produce a three-dimensional nanoball structure that is approximately 220&#160;nm in diameter. Making DNBs replaces the need to generate PCR copies of the library on the flow cell and as such can remove large proportions of duplicate reads, adapter-adapter ligations and PCR induced errors.<sup id="cite_ref-:1_104-1" class="reference"><a href="#cite_note-:1-104">&#91;103&#93;</a></sup><sup class="noprint Inline-Template Template-Fact" style="white-space:nowrap;">&#91;<i><a href="/wiki/Wikipedia:Citation_needed" title="Wikipedia:Citation needed"><span title="This claim needs references to reliable sources. (July 2018)">citation needed</span></a></i>&#93;</sup>
</p>
<div class="thumb tright"><div class="thumbinner" style="width:222px;"><a href="/wiki/File:MGISEQ-2000RS.jpg" class="image"><img alt="" src="//upload.wikimedia.org/wikipedia/commons/thumb/f/f8/MGISEQ-2000RS.jpg/220px-MGISEQ-2000RS.jpg" decoding="async" width="220" height="165" class="thumbimage" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/f/f8/MGISEQ-2000RS.jpg/330px-MGISEQ-2000RS.jpg 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/f/f8/MGISEQ-2000RS.jpg/440px-MGISEQ-2000RS.jpg 2x" data-file-width="4608" data-file-height="3456" /></a>  <div class="thumbcaption"><div class="magnify"><a href="/wiki/File:MGISEQ-2000RS.jpg" class="internal" title="Enlarge"></a></div>A BGI MGISEQ-2000RS sequencer</div></div></div>
<p>The patterned array of positively charged spots is fabricated through photolithography and etching techniques followed by chemical modification to generate a sequencing flow cell.  Each spot on the flow cell is approximately 250&#160;nm in diameter, are separated by 700&#160;nm  (centre to centre) and allows easy attachment of a single negatively charged DNB to the flow cell and thus reducing under or over-clustering on the flow cell.<sup id="cite_ref-:0_102-2" class="reference"><a href="#cite_note-:0-102">&#91;101&#93;</a></sup><sup class="noprint Inline-Template Template-Fact" style="white-space:nowrap;">&#91;<i><a href="/wiki/Wikipedia:Citation_needed" title="Wikipedia:Citation needed"><span title="This claim needs references to reliable sources. (July 2018)">citation needed</span></a></i>&#93;</sup>
</p><p>Sequencing is then performed by addition of an oligonucleotide probe that attaches in combination to specific sites within the DNB. The probe acts as an anchor that then allows one of four single reversibly inactivated, labelled nucleotides to bind after flowing across the flow cell. Unbound nucleotides are washed away before laser excitation of the attached labels then emit fluorescence and signal is captured by cameras that is converted to a digital output for base calling. The attached base has its terminator and label chemically cleaved at completion of the cycle. The cycle is repeated with another flow of free, labelled nucleotides across the flow cell to allow the next nucleotide to bind and have its signal captured. This process is completed a number of times (usually 50 to 300 times) to determine the sequence of the inserted piece of DNA at a rate of approximately 40 million nucleotides per second as of 2018.<sup class="noprint Inline-Template Template-Fact" style="white-space:nowrap;">&#91;<i><a href="/wiki/Wikipedia:Citation_needed" title="Wikipedia:Citation needed"><span title="This claim needs references to reliable sources. (July 2018)">citation needed</span></a></i>&#93;</sup>
</p>
<h4><span class="mw-headline" id="SOLiD_sequencing">SOLiD sequencing</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=DNA_sequencing&amp;action=edit&amp;section=30" title="Edit section: SOLiD sequencing">edit</a><span class="mw-editsection-bracket">]</span></span></h4>
<div class="thumb tright"><div class="thumbinner" style="width:222px;"><a href="/wiki/File:Library_preparation_for_the_SOLiD_platform.svg" class="image"><img alt="" src="//upload.wikimedia.org/wikipedia/commons/thumb/1/18/Library_preparation_for_the_SOLiD_platform.svg/220px-Library_preparation_for_the_SOLiD_platform.svg.png" decoding="async" width="220" height="220" class="thumbimage" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/1/18/Library_preparation_for_the_SOLiD_platform.svg/330px-Library_preparation_for_the_SOLiD_platform.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/1/18/Library_preparation_for_the_SOLiD_platform.svg/440px-Library_preparation_for_the_SOLiD_platform.svg.png 2x" data-file-width="406" data-file-height="406" /></a>  <div class="thumbcaption"><div class="magnify"><a href="/wiki/File:Library_preparation_for_the_SOLiD_platform.svg" class="internal" title="Enlarge"></a></div>Library preparation for the SOLiD platform</div></div></div>
<div role="note" class="hatnote navigation-not-searchable">Main article: <a href="/wiki/2_base_encoding" title="2 base encoding">2 base encoding</a></div>
<p><a href="/wiki/Applied_Biosystems" title="Applied Biosystems">Applied Biosystems</a>' (now a <a href="/wiki/Life_Technologies_(Thermo_Fisher_Scientific)" title="Life Technologies (Thermo Fisher Scientific)">Life Technologies</a> brand) SOLiD technology employs <a href="/wiki/Sequencing_by_ligation" title="Sequencing by ligation">sequencing by ligation</a>. Here, a pool of all possible oligonucleotides of a fixed length are labeled according to the sequenced position. Oligonucleotides are annealed and ligated; the preferential ligation by <a href="/wiki/DNA_ligase" title="DNA ligase">DNA ligase</a> for matching sequences results in a signal informative of the nucleotide at that position. Before sequencing, the DNA is amplified by emulsion PCR. The resulting beads, each containing single copies of the same DNA molecule, are deposited on a glass slide.<sup id="cite_ref-pmid18477713_105-0" class="reference"><a href="#cite_note-pmid18477713-105">&#91;104&#93;</a></sup> The result is sequences of quantities and lengths comparable to Illumina sequencing.<sup id="cite_ref-pmid18165802_69-2" class="reference"><a href="#cite_note-pmid18165802-69">&#91;68&#93;</a></sup> This <a href="/wiki/Sequencing_by_ligation" title="Sequencing by ligation">sequencing by ligation</a> method has been reported to have some issue sequencing palindromic sequences.<sup id="cite_ref-Yu-Feng_Huang,_Sheng-Chung_Chen,_Yih-Shien_Chiang,_Tzu-Han_Chen_&amp;_Kuo-Ping_Chiu_2012_S10_88-1" class="reference"><a href="#cite_note-Yu-Feng_Huang,_Sheng-Chung_Chen,_Yih-Shien_Chiang,_Tzu-Han_Chen_&amp;_Kuo-Ping_Chiu_2012_S10-88">&#91;87&#93;</a></sup>
</p>
<h4><span class="mw-headline" id="Ion_Torrent_semiconductor_sequencing">Ion Torrent semiconductor sequencing</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=DNA_sequencing&amp;action=edit&amp;section=31" title="Edit section: Ion Torrent semiconductor sequencing">edit</a><span class="mw-editsection-bracket">]</span></span></h4>
<div role="note" class="hatnote navigation-not-searchable">Main article: <a href="/wiki/Ion_semiconductor_sequencing" title="Ion semiconductor sequencing">Ion semiconductor sequencing</a></div>
<p>Ion Torrent Systems Inc. (now owned by <a href="/wiki/Life_Technologies_(Thermo_Fisher_Scientific)" title="Life Technologies (Thermo Fisher Scientific)">Life Technologies</a>) developed a system based on using standard sequencing chemistry, but with a novel, semiconductor-based detection system. This method of sequencing is based on the detection of <a href="/wiki/Hydrogen_ion" title="Hydrogen ion">hydrogen ions</a> that are released during the <a href="/wiki/DNA_polymerase" title="DNA polymerase">polymerisation</a> of <a href="/wiki/DNA" title="DNA">DNA</a>, as opposed to the optical methods used in other sequencing systems. A microwell containing a template DNA strand to be sequenced is flooded with a single type of <a href="/wiki/Nucleotide" title="Nucleotide">nucleotide</a>. If the introduced nucleotide is <a href="/wiki/Complementarity_(molecular_biology)" title="Complementarity (molecular biology)">complementary</a> to the leading template nucleotide it is incorporated into the growing complementary strand. This causes the release of a hydrogen ion that triggers a hypersensitive ion sensor, which indicates that a reaction has occurred. If <a href="/wiki/Homopolymer" class="mw-redirect" title="Homopolymer">homopolymer</a> repeats are present in the template sequence, multiple nucleotides will be incorporated in a single cycle. This leads to a corresponding number of released hydrogens and a proportionally higher electronic signal.<sup id="cite_ref-rusk_106-0" class="reference"><a href="#cite_note-rusk-106">&#91;105&#93;</a></sup>
</p>
<div class="thumb tright"><div class="thumbinner" style="width:222px;"><a href="/wiki/File:From_second_to_fourth-generation_sequencing,_illustration_on_TAGGCT_template.svg" class="image"><img alt="" src="//upload.wikimedia.org/wikipedia/commons/thumb/2/2e/From_second_to_fourth-generation_sequencing%2C_illustration_on_TAGGCT_template.svg/220px-From_second_to_fourth-generation_sequencing%2C_illustration_on_TAGGCT_template.svg.png" decoding="async" width="220" height="261" class="thumbimage" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/2/2e/From_second_to_fourth-generation_sequencing%2C_illustration_on_TAGGCT_template.svg/330px-From_second_to_fourth-generation_sequencing%2C_illustration_on_TAGGCT_template.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/2/2e/From_second_to_fourth-generation_sequencing%2C_illustration_on_TAGGCT_template.svg/440px-From_second_to_fourth-generation_sequencing%2C_illustration_on_TAGGCT_template.svg.png 2x" data-file-width="401" data-file-height="475" /></a>  <div class="thumbcaption"><div class="magnify"><a href="/wiki/File:From_second_to_fourth-generation_sequencing,_illustration_on_TAGGCT_template.svg" class="internal" title="Enlarge"></a></div>Sequencing of the TAGGCT template with IonTorrent, PacBioRS and GridION</div></div></div>
<h4><span class="mw-headline" id="DNA_nanoball_sequencing">DNA nanoball sequencing</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=DNA_sequencing&amp;action=edit&amp;section=32" title="Edit section: DNA nanoball sequencing">edit</a><span class="mw-editsection-bracket">]</span></span></h4>
<div role="note" class="hatnote navigation-not-searchable">Main article: <a href="/wiki/DNA_nanoball_sequencing" title="DNA nanoball sequencing">DNA nanoball sequencing</a></div>
<p><a href="/wiki/DNA_nanoball_sequencing" title="DNA nanoball sequencing">DNA nanoball sequencing</a> is a type of high throughput sequencing technology used to determine the entire <a href="/wiki/Genomic_sequence" class="mw-redirect" title="Genomic sequence">genomic sequence</a> of an organism.  The company <a href="/wiki/Complete_Genomics" title="Complete Genomics">Complete Genomics</a> uses this technology to sequence samples submitted by independent researchers.  The method uses <a href="/wiki/Rolling_circle_replication" title="Rolling circle replication">rolling circle replication</a> to amplify small fragments of genomic DNA into DNA nanoballs.  Unchained sequencing by ligation is then used to determine the nucleotide sequence.<sup id="cite_ref-Drmanac_2010_107-0" class="reference"><a href="#cite_note-Drmanac_2010-107">&#91;106&#93;</a></sup>  This method of DNA sequencing allows large numbers of DNA nanoballs to be sequenced per run and at low <a href="/wiki/Reagent" title="Reagent">reagent</a> costs compared to other high-throughput sequencing platforms.<sup id="cite_ref-108" class="reference"><a href="#cite_note-108">&#91;107&#93;</a></sup>  However, only short sequences of DNA are determined from each DNA nanoball which makes mapping the short reads to a <a href="/wiki/Reference_genome" title="Reference genome">reference genome</a> difficult.<sup id="cite_ref-Drmanac_2010_107-1" class="reference"><a href="#cite_note-Drmanac_2010-107">&#91;106&#93;</a></sup> This technology has been used for multiple genome sequencing projects and is scheduled to be used for more.<sup id="cite_ref-109" class="reference"><a href="#cite_note-109">&#91;108&#93;</a></sup>
</p>
<h4><span class="mw-headline" id="Heliscope_single_molecule_sequencing">Heliscope single molecule sequencing</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=DNA_sequencing&amp;action=edit&amp;section=33" title="Edit section: Heliscope single molecule sequencing">edit</a><span class="mw-editsection-bracket">]</span></span></h4>
<div role="note" class="hatnote navigation-not-searchable">Main article: <a href="/wiki/Helicos_single_molecule_fluorescent_sequencing" title="Helicos single molecule fluorescent sequencing">Helicos single molecule fluorescent sequencing</a></div>
<p>Heliscope sequencing is a method of single-molecule sequencing developed by <a href="/wiki/Helicos_Biosciences" title="Helicos Biosciences">Helicos Biosciences</a>. It uses DNA fragments with added poly-A tail adapters which are attached to the flow cell surface. The next steps involve extension-based sequencing with cyclic washes of the flow cell with fluorescently labeled nucleotides (one nucleotide type at a time, as with the Sanger method). The reads are performed by the Heliscope sequencer.<sup id="cite_ref-110" class="reference"><a href="#cite_note-110">&#91;109&#93;</a></sup><sup id="cite_ref-111" class="reference"><a href="#cite_note-111">&#91;110&#93;</a></sup> The reads are short, averaging 35 bp.<sup id="cite_ref-112" class="reference"><a href="#cite_note-112">&#91;111&#93;</a></sup> What made this technology especially novel was that it was the first of its class to sequence non-amplified DNA, thus preventing any read errors associated with amplification steps.<sup id="cite_ref-113" class="reference"><a href="#cite_note-113">&#91;112&#93;</a></sup> In 2009 a human genome was sequenced using the Heliscope, however in 2012 the company went bankrupt.<sup id="cite_ref-114" class="reference"><a href="#cite_note-114">&#91;113&#93;</a></sup>
</p>
<h4><span class="mw-headline" id="Microfluidic_Systems">Microfluidic Systems</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=DNA_sequencing&amp;action=edit&amp;section=34" title="Edit section: Microfluidic Systems">edit</a><span class="mw-editsection-bracket">]</span></span></h4>
<p>There are two main microfluidic systems that are used to sequence DNA; <a href="/wiki/Droplet-based_microfluidics" title="Droplet-based microfluidics">droplet based microfluidics</a> and <a href="/wiki/Digital_microfluidics" title="Digital microfluidics">digital microfluidics</a>. Microfluidic devices solve many of the current limitations of current sequencing arrays.
</p><p>Abate et al. studied the use of droplet-based microfluidic devices for DNA sequencing.<sup id="cite_ref-:3_4-2" class="reference"><a href="#cite_note-:3-4">&#91;4&#93;</a></sup> These devices have the ability to form and process picoliter sized droplets at the rate of thousands per second. The devices were created from <a href="/wiki/Polydimethylsiloxane" title="Polydimethylsiloxane">polydimethylsiloxane (PDMS)</a>and used Forster resonance energy transfer, <a href="/wiki/F%C3%B6rster_resonance_energy_transfer" title="Förster resonance energy transfer">FRET assays</a> to read the sequences of DNA encompassed in the droplets. Each position on the array tested for a specific 15 base sequence.<sup id="cite_ref-:3_4-3" class="reference"><a href="#cite_note-:3-4">&#91;4&#93;</a></sup>
</p><p>Fair et al. used digital microfluidic devices to study DNA <a href="/wiki/Pyrosequencing" title="Pyrosequencing">pyrosequencing</a>.<sup id="cite_ref-:4_115-0" class="reference"><a href="#cite_note-:4-115">&#91;114&#93;</a></sup> Significant advantages include the portability of the device, reagent volume, speed of analysis, mass manufacturing abilities, and high throughput. This study provided a proof of concept showing that digital devices can be used for pyrosequencing; the study included using synthesis, which involves the extension of the enzymes and addition of labeled nucleotides.<sup id="cite_ref-:4_115-1" class="reference"><a href="#cite_note-:4-115">&#91;114&#93;</a></sup>
</p><p>Boles et al. also studied pyrosequencing on digital microfluidic devices.<sup id="cite_ref-:5_116-0" class="reference"><a href="#cite_note-:5-116">&#91;115&#93;</a></sup> They used an electro-wetting device to create, mix, and split droplets. The sequencing uses a three-enzyme protocol and DNA templates anchored with magnetic beads. The device was tested using two protocols and resulted in 100% accuracy based on raw pyrogram levels. The advantages of these digital microfluidic devices include size, cost, and achievable levels of functional integration.<sup id="cite_ref-:5_116-1" class="reference"><a href="#cite_note-:5-116">&#91;115&#93;</a></sup>
</p><p>DNA sequencing research, using microfluidics, also has the ability to be applied to the <a href="/wiki/RNA-Seq" title="RNA-Seq">sequencing of RNA</a>, using similar droplet microfluidic techniques, such as the method, inDrops.<sup id="cite_ref-117" class="reference"><a href="#cite_note-117">&#91;116&#93;</a></sup> This shows that many of these DNA sequencing techniques will be able to be applied further and be used to understand more about genomes and transcriptomes.
</p>
<h2><span class="mw-headline" id="Methods_in_development">Methods in development</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=DNA_sequencing&amp;action=edit&amp;section=35" title="Edit section: Methods in development">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<p>DNA sequencing methods currently under development include reading the sequence as a DNA strand transits through <a href="/wiki/Nanopore_sequencing" title="Nanopore sequencing">nanopores</a> (a method that is now commercial but subsequent generations such as solid-state nanopores are still in development),<sup id="cite_ref-118" class="reference"><a href="#cite_note-118">&#91;117&#93;</a></sup><sup id="cite_ref-Physorg_119-0" class="reference"><a href="#cite_note-Physorg-119">&#91;118&#93;</a></sup> and microscopy-based techniques, such as <a href="/wiki/Atomic_force_microscope" class="mw-redirect" title="Atomic force microscope">atomic force microscopy</a> or <a href="/wiki/Transmission_electron_microscopy_DNA_sequencing" title="Transmission electron microscopy DNA sequencing">transmission electron microscopy</a> that are used to identify the positions of individual nucleotides within long DNA fragments (&gt;5,000 bp) by nucleotide labeling with heavier elements (e.g., halogens) for visual detection and recording.<sup id="cite_ref-120" class="reference"><a href="#cite_note-120">&#91;119&#93;</a></sup><sup id="cite_ref-121" class="reference"><a href="#cite_note-121">&#91;120&#93;</a></sup>
<a href="/wiki/Third-generation_sequencing" title="Third-generation sequencing">Third generation technologies</a> aim to increase throughput and decrease the time to result and cost by eliminating the need for excessive reagents and harnessing the processivity of DNA polymerase.<sup id="cite_ref-122" class="reference"><a href="#cite_note-122">&#91;121&#93;</a></sup>
</p>
<h3><span class="mw-headline" id="Tunnelling_currents_DNA_sequencing">Tunnelling currents DNA sequencing</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=DNA_sequencing&amp;action=edit&amp;section=36" title="Edit section: Tunnelling currents DNA sequencing">edit</a><span class="mw-editsection-bracket">]</span></span></h3>
<p>Another approach uses measurements of the electrical tunnelling currents across single-strand DNA as it moves through a channel. Depending on its electronic structure, each base affects the tunnelling current differently,<sup id="cite_ref-123" class="reference"><a href="#cite_note-123">&#91;122&#93;</a></sup> allowing differentiation between different bases.<sup id="cite_ref-124" class="reference"><a href="#cite_note-124">&#91;123&#93;</a></sup>
</p><p>The use of tunnelling currents has the potential to sequence orders of magnitude faster than ionic current methods and the sequencing of several DNA oligomers and micro-RNA has already been achieved.<sup id="cite_ref-pmid22787559_125-0" class="reference"><a href="#cite_note-pmid22787559-125">&#91;124&#93;</a></sup>
</p>
<h3><span class="mw-headline" id="Sequencing_by_hybridization">Sequencing by hybridization</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=DNA_sequencing&amp;action=edit&amp;section=37" title="Edit section: Sequencing by hybridization">edit</a><span class="mw-editsection-bracket">]</span></span></h3>
<p><i><a href="/wiki/Sequencing_by_hybridization" title="Sequencing by hybridization">Sequencing by hybridization</a></i> is a non-enzymatic method that uses a <a href="/wiki/DNA_microarray" title="DNA microarray">DNA microarray</a>. A single pool of DNA whose sequence is to be determined is fluorescently labeled and hybridized to an array containing known sequences. Strong hybridization signals from a given spot on the array identifies its sequence in the DNA being sequenced.<sup id="cite_ref-126" class="reference"><a href="#cite_note-126">&#91;125&#93;</a></sup>
</p><p>This method of sequencing utilizes binding characteristics of a library of short single stranded DNA molecules (oligonucleotides), also called DNA probes, to reconstruct a target DNA sequence. Non-specific hybrids are removed by washing and the target DNA is eluted.<sup id="cite_ref-Morey_127-0" class="reference"><a href="#cite_note-Morey-127">&#91;126&#93;</a></sup> Hybrids are re-arranged such that the DNA sequence can be reconstructed. The benefit of this sequencing type is its ability to capture a large number of targets with a homogenous coverage.<sup id="cite_ref-Qin_128-0" class="reference"><a href="#cite_note-Qin-128">&#91;127&#93;</a></sup> A large number of chemicals and starting DNA is usually required. However, with the advent of solution-based hybridization, much less equipment and chemicals are necessary.<sup id="cite_ref-Morey_127-1" class="reference"><a href="#cite_note-Morey-127">&#91;126&#93;</a></sup>
</p>
<h3><span class="mw-headline" id="Sequencing_with_mass_spectrometry">Sequencing with mass spectrometry</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=DNA_sequencing&amp;action=edit&amp;section=38" title="Edit section: Sequencing with mass spectrometry">edit</a><span class="mw-editsection-bracket">]</span></span></h3>
<p><a href="/wiki/Mass_spectrometry" title="Mass spectrometry">Mass spectrometry</a> may be used to determine DNA sequences. Matrix-assisted laser desorption ionization time-of-flight mass spectrometry, or <a href="/wiki/Matrix-assisted_laser_desorption/ionization" title="Matrix-assisted laser desorption/ionization">MALDI-TOF MS</a>, has specifically been investigated as an alternative method to gel electrophoresis for visualizing DNA fragments. With this method, DNA fragments generated by chain-termination sequencing reactions are compared by mass rather than by size. The mass of each nucleotide is different from the others and this difference is detectable by mass spectrometry. Single-nucleotide mutations in a fragment can be more easily detected with MS than by gel electrophoresis alone. MALDI-TOF MS can more easily detect differences between RNA fragments, so researchers may indirectly sequence DNA with MS-based methods by converting it to RNA first.<sup id="cite_ref-129" class="reference"><a href="#cite_note-129">&#91;128&#93;</a></sup>
</p><p>The higher resolution of DNA fragments permitted by MS-based methods is of special interest to researchers in forensic science, as they may wish to find <a href="/wiki/Single-nucleotide_polymorphisms" class="mw-redirect" title="Single-nucleotide polymorphisms">single-nucleotide polymorphisms</a> in human DNA samples to identify individuals. These samples may be highly degraded so forensic researchers often prefer <a href="/wiki/Mitochondrial_DNA" title="Mitochondrial DNA">mitochondrial DNA</a> for its higher stability and applications for lineage studies. MS-based sequencing methods have been used to compare the sequences of human mitochondrial DNA from samples in a <a href="/wiki/Federal_Bureau_of_Investigation" title="Federal Bureau of Investigation">Federal Bureau of Investigation</a> database<sup id="cite_ref-130" class="reference"><a href="#cite_note-130">&#91;129&#93;</a></sup> and from bones found in mass graves of World War I soldiers.<sup id="cite_ref-131" class="reference"><a href="#cite_note-131">&#91;130&#93;</a></sup>
</p><p>Early chain-termination and TOF MS methods demonstrated read lengths of up to 100 base pairs.<sup id="cite_ref-132" class="reference"><a href="#cite_note-132">&#91;131&#93;</a></sup> Researchers have been unable to exceed this average read size; like chain-termination sequencing alone, MS-based DNA sequencing may not be suitable for large <i>de novo</i> sequencing projects. Even so, a recent study did use the short sequence reads and mass spectroscopy to compare single-nucleotide polymorphisms in pathogenic <i><a href="/wiki/Streptococcus" title="Streptococcus">Streptococcus</a></i> strains.<sup id="cite_ref-133" class="reference"><a href="#cite_note-133">&#91;132&#93;</a></sup>
</p>
<h3><span class="mw-headline" id="Microfluidic_Sanger_sequencing">Microfluidic Sanger sequencing</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=DNA_sequencing&amp;action=edit&amp;section=39" title="Edit section: Microfluidic Sanger sequencing">edit</a><span class="mw-editsection-bracket">]</span></span></h3>
<div role="note" class="hatnote navigation-not-searchable">Main article: <a href="/wiki/Sanger_sequencing" title="Sanger sequencing">Sanger sequencing</a></div>
<p>In microfluidic Sanger sequencing the entire thermocycling amplification of DNA fragments as well as their separation by electrophoresis is done on a single glass wafer (approximately 10&#160;cm in diameter) thus reducing the reagent usage as well as cost.<sup id="cite_ref-134" class="reference"><a href="#cite_note-134">&#91;133&#93;</a></sup> In some instances researchers have shown that they can increase the throughput of conventional sequencing through the use of microchips.<sup id="cite_ref-135" class="reference"><a href="#cite_note-135">&#91;134&#93;</a></sup> Research will still need to be done in order to make this use of technology effective.
</p>
<h3><span class="mw-headline" id="Microscopy-based_techniques">Microscopy-based techniques</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=DNA_sequencing&amp;action=edit&amp;section=40" title="Edit section: Microscopy-based techniques">edit</a><span class="mw-editsection-bracket">]</span></span></h3>
<div role="note" class="hatnote navigation-not-searchable">Main article: <a href="/wiki/Transmission_electron_microscopy_DNA_sequencing" title="Transmission electron microscopy DNA sequencing">Transmission electron microscopy DNA sequencing</a></div>
<p>This approach directly visualizes the sequence of DNA molecules using electron microscopy. The first identification of DNA base pairs within intact DNA molecules by enzymatically incorporating modified bases, which contain atoms of increased atomic number, direct visualization and identification of individually labeled bases within a synthetic 3,272 base-pair DNA molecule and a 7,249 base-pair viral genome has been demonstrated.<sup id="cite_ref-136" class="reference"><a href="#cite_note-136">&#91;135&#93;</a></sup>
</p>
<h3><span class="mw-headline" id="RNAP_sequencing">RNAP sequencing</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=DNA_sequencing&amp;action=edit&amp;section=41" title="Edit section: RNAP sequencing">edit</a><span class="mw-editsection-bracket">]</span></span></h3>
<p>This method is based on use of <a href="/wiki/RNA_polymerase" title="RNA polymerase">RNA polymerase</a> (RNAP), which is attached to a <a href="/wiki/Polystyrene" title="Polystyrene">polystyrene</a> bead. One end of DNA to be sequenced is attached to another bead, with both beads being placed in optical traps. RNAP motion during transcription brings the beads in closer and their relative distance changes, which can then be recorded at a single nucleotide resolution. The sequence is deduced based on the four readouts with lowered concentrations of each of the four nucleotide types, similarly to the Sanger method.<sup id="cite_ref-137" class="reference"><a href="#cite_note-137">&#91;136&#93;</a></sup> A comparison is made between regions and sequence information is deduced by comparing the known sequence regions to the unknown sequence regions.<sup id="cite_ref-Pareek_CS_138-0" class="reference"><a href="#cite_note-Pareek_CS-138">&#91;137&#93;</a></sup>
</p>
<h3><span class="mw-headline" id="In_vitro_virus_high-throughput_sequencing"><i>In vitro</i> virus high-throughput sequencing</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=DNA_sequencing&amp;action=edit&amp;section=42" title="Edit section: In vitro virus high-throughput sequencing">edit</a><span class="mw-editsection-bracket">]</span></span></h3>
<p>A method has been developed to analyze full sets of <a href="/wiki/Interactome" title="Interactome">protein interactions</a> using a combination of 454 pyrosequencing and an  <i>in vitro</i> virus <a href="/wiki/MRNA_display" title="MRNA display">mRNA display</a> method. Specifically, this method covalently links proteins of interest to the mRNAs encoding them, then detects the mRNA pieces using reverse transcription <a href="/wiki/Polymerase_chain_reaction" title="Polymerase chain reaction">PCRs</a>. The mRNA may then be amplified and sequenced. The combined method was titled IVV-HiTSeq and can be performed under cell-free conditions, though its results may not be representative of <i>in vivo</i> conditions.<sup id="cite_ref-139" class="reference"><a href="#cite_note-139">&#91;138&#93;</a></sup>
</p>
<h2><span class="mw-headline" id="Sample_preparation">Sample preparation</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=DNA_sequencing&amp;action=edit&amp;section=43" title="Edit section: Sample preparation">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<p>The success of any DNA sequencing protocol relies upon the DNA or RNA sample extraction and preparation from the biological material of interest.
</p>
<ul><li>A successful DNA extraction will yield a DNA sample with long, non-degraded strands.</li>
<li>A successful RNA extraction will yield a RNA sample that should be converted to complementary DNA (cDNA) using reverse transcriptase—a DNA polymerase that synthesizes a complementary DNA based on existing strands of RNA in a PCR-like manner.<sup id="cite_ref-140" class="reference"><a href="#cite_note-140">&#91;139&#93;</a></sup> Complementary DNA can then be processed the same way as genomic DNA.</li></ul>
<p>According to the sequencing technology to be used, the samples resulting from either the DNA or the RNA extraction require further preparation.  For Sanger sequencing, either cloning procedures or PCR are required prior to sequencing. In the case of next-generation sequencing methods, library preparation is required before processing.<sup id="cite_ref-141" class="reference"><a href="#cite_note-141">&#91;140&#93;</a></sup> Assessing the quality and quantity of nucleic acids both after extraction and after library preparation identifies degraded, fragmented, and low-purity samples and yields high-quality sequencing data.<sup id="cite_ref-142" class="reference"><a href="#cite_note-142">&#91;141&#93;</a></sup>
</p><p>The high-throughput nature of current DNA/RNA sequencing technologies has posed a challenge for sample preparation method to scale-up. Several liquid handling instruments are being used for the preparation of higher numbers of samples with a lower total hands-on time:
</p>
<table class="wikitable" style="font-size:0.9em;">
<tbody><tr>
<th width="9%">company
</th>
<th width="22%">Liquid handlers / Automation
</th>
<th width="67%">landing_url
</th></tr>
<tr>
<td><a href="/w/index.php?title=Opentrons&amp;action=edit&amp;redlink=1" class="new" title="Opentrons (page does not exist)">Opentrons</a>
</td>
<td>OpenTrons OT-2
</td>
<td><a rel="nofollow" class="external free" href="https://www.opentrons.com/">https://www.opentrons.com/</a>
</td></tr>
<tr>
<td><a href="/wiki/Agilent_Technologies" title="Agilent Technologies">Agilent</a>
</td>
<td>Agilent Bravo NGS
</td>
<td><a rel="nofollow" class="external free" href="https://www.agilent.com/en/products/automated-liquid-handling/automated-liquid-handling-applications/bravo-ngs">https://www.agilent.com/en/products/automated-liquid-handling/automated-liquid-handling-applications/bravo-ngs</a>
</td></tr>
<tr>
<td><a href="/wiki/Beckman_Coulter" title="Beckman Coulter">Beckman Coulter</a>
</td>
<td>Beckman Coulter Biomek iSeries
</td>
<td><a rel="nofollow" class="external free" href="https://www.beckman.com/liquid-handlers/biomek-i7/features">https://www.beckman.com/liquid-handlers/biomek-i7/features</a>
</td></tr>
<tr>
<td><a href="/wiki/Eppendorf_(company)" title="Eppendorf (company)">Eppendorf</a>
</td>
<td>Eppendorf epMotion 5075t
</td>
<td><a rel="nofollow" class="external free" href="https://www.eppendorf.com/epmotion/">https://www.eppendorf.com/epmotion/</a>
</td></tr>
<tr>
<td>Hamilton
</td>
<td>NGS STAR
</td>
<td><a rel="nofollow" class="external free" href="http://www.hamiltonrobotics.com/">http://www.hamiltonrobotics.com/</a>
</td></tr>
<tr>
<td><a href="/wiki/PerkinElmer" title="PerkinElmer">PerkinElmer</a>
</td>
<td>Sciclone G3 NGS and NGSx Workstation
</td>
<td><a rel="nofollow" class="external free" href="https://www.perkinelmer.com/uk/product/sciclone-g3-ngs-workstation-cls145321">https://www.perkinelmer.com/uk/product/sciclone-g3-ngs-workstation-cls145321</a>
</td></tr>
<tr>
<td><a href="/wiki/Tecan" title="Tecan">Tecan</a>
</td>
<td>Tecan Freedom EVO NGS
</td>
<td><a rel="nofollow" class="external free" href="https://lifesciences.tecan.com/ngs-sample-preparation">https://lifesciences.tecan.com/ngs-sample-preparation</a>
</td></tr>
<tr>
<td><a href="/w/index.php?title=Hudson_Robotics&amp;action=edit&amp;redlink=1" class="new" title="Hudson Robotics (page does not exist)">Hudson Robotics</a>
</td>
<td>Hudson Robotics SOLO
</td>
<td><a rel="nofollow" class="external free" href="https://hudsonrobotics.com/products/applications/automated-solutions-next-generation-sequencing-ngs/">https://hudsonrobotics.com/products/applications/automated-solutions-next-generation-sequencing-ngs/</a>
</td></tr></tbody></table>
<p><br />
</p>
<h2><span class="mw-headline" id="Development_initiatives">Development initiatives</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=DNA_sequencing&amp;action=edit&amp;section=44" title="Edit section: Development initiatives">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<div class="thumb tright"><div class="thumbinner" style="width:282px;"><a href="/wiki/File:Historic_cost_of_sequencing_a_human_genome.svg" class="image"><img alt="" src="//upload.wikimedia.org/wikipedia/commons/thumb/e/e7/Historic_cost_of_sequencing_a_human_genome.svg/280px-Historic_cost_of_sequencing_a_human_genome.svg.png" decoding="async" width="280" height="187" class="thumbimage" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/e/e7/Historic_cost_of_sequencing_a_human_genome.svg/420px-Historic_cost_of_sequencing_a_human_genome.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/e/e7/Historic_cost_of_sequencing_a_human_genome.svg/560px-Historic_cost_of_sequencing_a_human_genome.svg.png 2x" data-file-width="1200" data-file-height="800" /></a>  <div class="thumbcaption"><div class="magnify"><a href="/wiki/File:Historic_cost_of_sequencing_a_human_genome.svg" class="internal" title="Enlarge"></a></div>Total cost of sequencing a human genome over time as calculated by the <a href="/wiki/National_Human_Genome_Research_Institute" title="National Human Genome Research Institute">NHGRI</a>.</div></div></div>
<p>In October 2006, the <a href="/wiki/X_Prize_Foundation" title="X Prize Foundation">X Prize Foundation</a> established an initiative to promote the development of <a href="/wiki/Full_genome_sequencing" class="mw-redirect" title="Full genome sequencing">full genome sequencing</a> technologies, called the <a href="/wiki/Archon_X_Prize" title="Archon X Prize">Archon X Prize</a>, intending to award $10 million to "the first Team that can build a device and use it to sequence 100 human genomes within 10 days or less, with an accuracy of no more than one error in every 100,000 bases sequenced, with sequences accurately covering at least 98% of the genome, and at a recurring cost of no more than $10,000 (US) per genome."<sup id="cite_ref-143" class="reference"><a href="#cite_note-143">&#91;142&#93;</a></sup>
</p><p>Each year the <a href="/wiki/National_Human_Genome_Research_Institute" title="National Human Genome Research Institute">National Human Genome Research Institute</a>, or NHGRI, promotes grants for new research and developments in <a href="/wiki/Genomics" title="Genomics">genomics</a>. 2010 grants and 2011 candidates include continuing work in microfluidic, polony and base-heavy sequencing methodologies.<sup id="cite_ref-144" class="reference"><a href="#cite_note-144">&#91;143&#93;</a></sup>
</p>
<h2><span class="mw-headline" id="Computational_challenges">Computational challenges</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=DNA_sequencing&amp;action=edit&amp;section=45" title="Edit section: Computational challenges">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<p>The sequencing technologies described here produce raw data that needs to be assembled into longer sequences such as complete genomes (<a href="/wiki/Sequence_assembly" title="Sequence assembly">sequence assembly</a>). There are many computational challenges to achieve this, such as the evaluation of the raw sequence data which is done by programs and algorithms such as <a href="/wiki/Phred_base_calling" title="Phred base calling">Phred</a> and <a href="/wiki/Phrap" title="Phrap">Phrap</a>. Other challenges have to deal with <a href="/wiki/Repetitive_DNA" class="mw-redirect" title="Repetitive DNA">repetitive</a> sequences that often prevent complete genome assemblies because they occur in many places of the genome. As a consequence, many sequences may not be assigned to particular <a href="/wiki/Chromosome" title="Chromosome">chromosomes</a>. The production of raw sequence data is only the beginning of its detailed <a href="/wiki/Bioinformatics" title="Bioinformatics">bioinformatical</a> analysis.<sup id="cite_ref-pmid24727769_145-0" class="reference"><a href="#cite_note-pmid24727769-145">&#91;144&#93;</a></sup> Yet new methods for sequencing and correcting sequencing errors were developed.<sup id="cite_ref-146" class="reference"><a href="#cite_note-146">&#91;145&#93;</a></sup>
</p>
<h3><span class="mw-headline" id="Read_trimming">Read trimming</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=DNA_sequencing&amp;action=edit&amp;section=46" title="Edit section: Read trimming">edit</a><span class="mw-editsection-bracket">]</span></span></h3>
<p>Sometimes, the raw reads produced by the sequencer are correct and precise only in a fraction of their length. Using the entire read may introduce artifacts in the downstream analyses like genome assembly, snp calling, or gene expression estimation. Two classes of trimming programs have been introduced, based on the window-based or the running-sum classes of algorithms.<sup id="cite_ref-147" class="reference"><a href="#cite_note-147">&#91;146&#93;</a></sup> This is a partial list of the trimming algorithms currently available, specifying the algorithm class they belong to:
</p>
<table class="wikitable">
<caption>Read Trimming Algorithms
</caption>
<tbody><tr>
<th>Name of algorithm</th>
<th>Type of algorithm</th>
<th>Link
</th></tr>
<tr>
<td>Cutadapt<sup id="cite_ref-cutadapt_148-0" class="reference"><a href="#cite_note-cutadapt-148">&#91;147&#93;</a></sup></td>
<td>Running sum</td>
<td><a rel="nofollow" class="external text" href="https://code.google.com/p/cutadapt/">Cutadapt</a>
</td></tr>
<tr>
<td>ConDeTri<sup id="cite_ref-condetri_149-0" class="reference"><a href="#cite_note-condetri-149">&#91;148&#93;</a></sup></td>
<td>Window based</td>
<td><a rel="nofollow" class="external text" href="https://code.google.com/p/condetri/">ConDeTri</a>
</td></tr>
<tr>
<td>ERNE-FILTER<sup id="cite_ref-erne-bs5_150-0" class="reference"><a href="#cite_note-erne-bs5-150">&#91;149&#93;</a></sup></td>
<td>Running sum</td>
<td><a rel="nofollow" class="external text" href="http://erne.sourceforge.net/">ERNE-FILTER</a>
</td></tr>
<tr>
<td>FASTX quality trimmer</td>
<td>Window based</td>
<td><a rel="nofollow" class="external text" href="http://hannonlab.cshl.edu/fastx_toolkit/download.html">FASTX quality trimmer</a>
</td></tr>
<tr>
<td>PRINSEQ<sup id="cite_ref-prinseq_151-0" class="reference"><a href="#cite_note-prinseq-151">&#91;150&#93;</a></sup></td>
<td>Window based</td>
<td><a rel="nofollow" class="external text" href="http://sourceforge.net/projects/prinseq/files/">PRINSEQ</a>
</td></tr>
<tr>
<td>Trimmomatic<sup id="cite_ref-trimmomatic_152-0" class="reference"><a href="#cite_note-trimmomatic-152">&#91;151&#93;</a></sup></td>
<td>Window based</td>
<td><a rel="nofollow" class="external text" href="http://www.usadellab.org/cms/index.php?page=trimmomatic">Trimmomatic</a>
</td></tr>
<tr>
<td>SolexaQA<sup id="cite_ref-solexaqa_153-0" class="reference"><a href="#cite_note-solexaqa-153">&#91;152&#93;</a></sup></td>
<td>Window based</td>
<td><a rel="nofollow" class="external text" href="http://sourceforge.net/projects/solexaqa/files/">SolexaQA</a>
</td></tr>
<tr>
<td>SolexaQA-BWA</td>
<td>Running sum</td>
<td><a rel="nofollow" class="external text" href="http://sourceforge.net/projects/solexaqa/files/">SolexaQA-BWA</a>
</td></tr>
<tr>
<td>Sickle</td>
<td>Window based</td>
<td><a rel="nofollow" class="external text" href="https://github.com/najoshi/sickle">Sickle</a>
</td></tr></tbody></table>
<h2><span class="mw-headline" id="Ethical_issues">Ethical issues</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=DNA_sequencing&amp;action=edit&amp;section=47" title="Edit section: Ethical issues">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<table class="box-Expand_section plainlinks metadata ambox mbox-small-left ambox-content" role="presentation"><tbody><tr><td class="mbox-image"><a href="/wiki/File:Wiki_letter_w_cropped.svg" class="image"><img alt="[icon]" src="//upload.wikimedia.org/wikipedia/commons/thumb/1/1c/Wiki_letter_w_cropped.svg/20px-Wiki_letter_w_cropped.svg.png" decoding="async" width="20" height="14" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/1/1c/Wiki_letter_w_cropped.svg/30px-Wiki_letter_w_cropped.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/1/1c/Wiki_letter_w_cropped.svg/40px-Wiki_letter_w_cropped.svg.png 2x" data-file-width="44" data-file-height="31" /></a></td><td class="mbox-text"><div class="mbox-text-span">This section <b>needs expansion</b>. <small>You can help by <a class="external text" href="https://en.wikipedia.org/w/index.php?title=DNA_sequencing&amp;action=edit&amp;section=">adding to it</a>.</small>  <small class="date-container"><i>(<span class="date">May 2015</span>)</i></small></div></td></tr></tbody></table>
<div role="note" class="hatnote navigation-not-searchable">Further information: <a href="/wiki/Bioethics" title="Bioethics">Bioethics</a></div>
<p>Human genetics have been included within the field of <a href="/wiki/Bioethics" title="Bioethics">bioethics</a> since the early 1970s<sup id="cite_ref-ethics-murray_154-0" class="reference"><a href="#cite_note-ethics-murray-154">&#91;153&#93;</a></sup> and the growth in the use of DNA sequencing (particularly high-throughput sequencing) has introduced a number of ethical issues. One key issue is the ownership of an individual's DNA and the data produced when that DNA is sequenced.<sup id="cite_ref-usd1000-genome-ethics_155-0" class="reference"><a href="#cite_note-usd1000-genome-ethics-155">&#91;154&#93;</a></sup> Regarding the DNA molecule itself, the leading legal case on this topic, <i><a href="/wiki/Moore_v._Regents_of_the_University_of_California" title="Moore v. Regents of the University of California">Moore v. Regents of the University of California</a></i> (1990) ruled that individuals have no property rights to discarded cells or any profits made using these cells (for instance, as a patented <a href="/wiki/Cell_line" class="mw-redirect" title="Cell line">cell line</a>). However, individuals have a right to informed consent regarding removal and use of cells. Regarding the data produced through DNA sequencing, <i>Moore</i> gives the individual no rights to the information derived from their DNA.<sup id="cite_ref-usd1000-genome-ethics_155-1" class="reference"><a href="#cite_note-usd1000-genome-ethics-155">&#91;154&#93;</a></sup>
</p><p>As DNA sequencing becomes more widespread, the storage, security and sharing of genomic data has also become more important.<sup id="cite_ref-usd1000-genome-ethics_155-2" class="reference"><a href="#cite_note-usd1000-genome-ethics-155">&#91;154&#93;</a></sup><sup id="cite_ref-guardian-ethics_156-0" class="reference"><a href="#cite_note-guardian-ethics-156">&#91;155&#93;</a></sup> For instance, one concern is that insurers may use an individual's genomic data to modify their quote, depending on the perceived future health of the individual based on their DNA.<sup id="cite_ref-guardian-ethics_156-1" class="reference"><a href="#cite_note-guardian-ethics-156">&#91;155&#93;</a></sup><sup id="cite_ref-ny-times-insurance-ethics_157-0" class="reference"><a href="#cite_note-ny-times-insurance-ethics-157">&#91;156&#93;</a></sup> In May 2008, the <a href="/wiki/Genetic_Information_Nondiscrimination_Act" title="Genetic Information Nondiscrimination Act">Genetic Information Nondiscrimination Act</a> (GINA) was signed in the United States, prohibiting discrimination on the basis of genetic information with respect to health insurance and employment.<sup id="cite_ref-OMB_support_158-0" class="reference"><a href="#cite_note-OMB_support-158">&#91;157&#93;</a></sup><sup id="cite_ref-Signing_159-0" class="reference"><a href="#cite_note-Signing-159">&#91;158&#93;</a></sup> In 2012, the US <a href="/wiki/Presidential_Commission_for_the_Study_of_Bioethical_Issues" title="Presidential Commission for the Study of Bioethical Issues">Presidential Commission for the Study of Bioethical Issues</a> reported that existing privacy legislation for DNA sequencing data such as GINA and the <a href="/wiki/Health_Insurance_Portability_and_Accountability_Act" title="Health Insurance Portability and Accountability Act">Health Insurance Portability and Accountability Act</a> were insufficient, noting that whole-genome sequencing data was particularly sensitive, as it could be used to identify not only the individual from which the data was created, but also their relatives.<sup id="cite_ref-nature-ethics_160-0" class="reference"><a href="#cite_note-nature-ethics-160">&#91;159&#93;</a></sup><sup id="cite_ref-privacy-progress-report_161-0" class="reference"><a href="#cite_note-privacy-progress-report-161">&#91;160&#93;</a></sup>
</p><p>Ethical issues have also been raised by the increasing use of genetic variation screening, both in newborns, and in adults by companies such as <a href="/wiki/23andMe" title="23andMe">23andMe</a>.<sup id="cite_ref-ethics-newborns_162-0" class="reference"><a href="#cite_note-ethics-newborns-162">&#91;161&#93;</a></sup><sup id="cite_ref-ethics-hughes_163-0" class="reference"><a href="#cite_note-ethics-hughes-163">&#91;162&#93;</a></sup> It has been asserted that screening for genetic variations can be harmful, increasing <a href="/wiki/Anxiety" title="Anxiety">anxiety</a> in individuals who have been found to have an increased risk of disease.<sup id="cite_ref-ethics-bloss_164-0" class="reference"><a href="#cite_note-ethics-bloss-164">&#91;163&#93;</a></sup> For example, in one case noted in <i><a href="/wiki/Time_(magazine)" title="Time (magazine)">Time</a></i>, doctors screening an ill baby for genetic variants chose not to inform the parents of an unrelated variant linked to <a href="/wiki/Dementia" title="Dementia">dementia</a> due to the harm it would cause to the parents.<sup id="cite_ref-ethics-time_165-0" class="reference"><a href="#cite_note-ethics-time-165">&#91;164&#93;</a></sup> However, a 2011 study in <i><a href="/wiki/The_New_England_Journal_of_Medicine" title="The New England Journal of Medicine">The New England Journal of Medicine</a></i> has shown that individuals undergoing disease risk profiling did not show increased levels of anxiety.<sup id="cite_ref-ethics-bloss_164-1" class="reference"><a href="#cite_note-ethics-bloss-164">&#91;163&#93;</a></sup>
</p>
<h2><span class="mw-headline" id="See_also">See also</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=DNA_sequencing&amp;action=edit&amp;section=48" title="Edit section: See also">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<div>

&#32;
<table class="multicol" role="presentation" style="border-collapse: collapse; padding: 0; border: 0; background:transparent; width:100%;"><tbody><tr>
<td style="width: 50%;text-align: left; vertical-align: top;">
<ul><li><a href="/wiki/Bioinformatics" title="Bioinformatics">Bioinformatics</a></li>
<li><a href="/wiki/Cancer_genome_sequencing" title="Cancer genome sequencing">Cancer genome sequencing</a></li>
<li><a href="/wiki/DNA_computing" title="DNA computing">DNA computing</a></li>
<li><a href="/wiki/DNA_field-effect_transistor" title="DNA field-effect transistor">DNA field-effect transistor</a></li>
<li><a href="/wiki/DNA_sequencing_theory" title="DNA sequencing theory">DNA sequencing theory</a></li>
<li><a href="/wiki/DNA_sequencer" title="DNA sequencer">DNA sequencer</a></li>
<li><a href="/wiki/Genome_project" title="Genome project">Genome project</a></li>
<li><a href="/wiki/Genome_sequencing_of_endangered_species" title="Genome sequencing of endangered species">Genome sequencing of endangered species</a></li>
<li><a href="/wiki/Genome_skimming" title="Genome skimming">Genome skimming</a></li>
<li><a href="/wiki/Jumping_library" title="Jumping library">Jumping library</a></li></ul>
<p>&#32;
</p>
</td>
<td style="width: 50%;text-align: left; vertical-align: top;">
<ul><li><a href="/wiki/Nucleic_acid_sequence" title="Nucleic acid sequence">Nucleic acid sequence</a></li>
<li><a href="/wiki/Multiplex_ligation-dependent_probe_amplification" title="Multiplex ligation-dependent probe amplification">Multiplex ligation-dependent probe amplification</a></li>
<li><a href="/wiki/Personalized_medicine" title="Personalized medicine">Personalized medicine</a></li>
<li><a href="/wiki/Protein_sequencing" title="Protein sequencing">Protein sequencing</a></li>
<li><a href="/wiki/Sequence_mining" class="mw-redirect" title="Sequence mining">Sequence mining</a></li>
<li><a href="/wiki/Sequence_profiling_tool" title="Sequence profiling tool">Sequence profiling tool</a></li>
<li><a href="/wiki/Sequencing_by_hybridization" title="Sequencing by hybridization">Sequencing by hybridization</a></li>
<li><a href="/wiki/Sequencing_by_ligation" title="Sequencing by ligation">Sequencing by ligation</a></li>
<li><a href="/wiki/Transmission_electron_microscopy_DNA_sequencing" title="Transmission electron microscopy DNA sequencing">Transmission electron microscopy DNA sequencing</a></li></ul>
<p>&#32;
</p>
</td></tr></tbody></table></div>
<h2><span class="mw-headline" id="Notes">Notes</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=DNA_sequencing&amp;action=edit&amp;section=49" title="Edit section: Notes">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<div class="reflist" style="list-style-type: decimal;">
<div class="mw-references-wrap"><ol class="references">
<li id="cite_note-64"><span class="mw-cite-backlink"><b><a href="#cite_ref-64">^</a></b></span> <span class="reference-text">"Next-generation" remains in broad use as of 2019.  For instance, <cite class="citation journal">Straiton J, Free T, Sawyer A, Martin J (February 2019). "From Sanger Sequencing to Genome Databases and Beyond". <i>BioTechniques</i>. <b>66</b> (2): 60–63. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.2144%2Fbtn-2019-0011">10.2144/btn-2019-0011</a>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/30744413">30744413</a>. <q>Next-generation sequencing (NGS) technologies have revolutionized genomic research. (opening sentence of the article)</q></cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=BioTechniques&amp;rft.atitle=From+Sanger+Sequencing+to+Genome+Databases+and+Beyond&amp;rft.volume=66&amp;rft.issue=2&amp;rft.pages=60-63&amp;rft.date=2019-02&amp;rft_id=info%3Adoi%2F10.2144%2Fbtn-2019-0011&amp;rft_id=info%3Apmid%2F30744413&amp;rft.aulast=Straiton&amp;rft.aufirst=J&amp;rft.au=Free%2C+T&amp;rft.au=Sawyer%2C+A&amp;rft.au=Martin%2C+J&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><style data-mw-deduplicate="TemplateStyles:r935243608">.mw-parser-output cite.citation{font-style:inherit}.mw-parser-output .citation q{quotes:"\"""\"""'""'"}.mw-parser-output .id-lock-free a,.mw-parser-output .citation .cs1-lock-free a{background:url("//upload.wikimedia.org/wikipedia/commons/thumb/6/65/Lock-green.svg/9px-Lock-green.svg.png")no-repeat;background-position:right .1em center}.mw-parser-output .id-lock-limited a,.mw-parser-output .id-lock-registration a,.mw-parser-output .citation .cs1-lock-limited a,.mw-parser-output .citation .cs1-lock-registration a{background:url("//upload.wikimedia.org/wikipedia/commons/thumb/d/d6/Lock-gray-alt-2.svg/9px-Lock-gray-alt-2.svg.png")no-repeat;background-position:right .1em center}.mw-parser-output .id-lock-subscription a,.mw-parser-output .citation .cs1-lock-subscription a{background:url("//upload.wikimedia.org/wikipedia/commons/thumb/a/aa/Lock-red-alt-2.svg/9px-Lock-red-alt-2.svg.png")no-repeat;background-position:right .1em center}.mw-parser-output .cs1-subscription,.mw-parser-output .cs1-registration{color:#555}.mw-parser-output .cs1-subscription span,.mw-parser-output .cs1-registration span{border-bottom:1px dotted;cursor:help}.mw-parser-output .cs1-ws-icon a{background:url("//upload.wikimedia.org/wikipedia/commons/thumb/4/4c/Wikisource-logo.svg/12px-Wikisource-logo.svg.png")no-repeat;background-position:right .1em center}.mw-parser-output code.cs1-code{color:inherit;background:inherit;border:inherit;padding:inherit}.mw-parser-output .cs1-hidden-error{display:none;font-size:100%}.mw-parser-output .cs1-visible-error{font-size:100%}.mw-parser-output .cs1-maint{display:none;color:#33aa33;margin-left:0.3em}.mw-parser-output .cs1-subscription,.mw-parser-output .cs1-registration,.mw-parser-output .cs1-format{font-size:95%}.mw-parser-output .cs1-kern-left,.mw-parser-output .cs1-kern-wl-left{padding-left:0.2em}.mw-parser-output .cs1-kern-right,.mw-parser-output .cs1-kern-wl-right{padding-right:0.2em}</style></span>
</li>
</ol></div></div>
<h2><span class="mw-headline" id="References">References</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=DNA_sequencing&amp;action=edit&amp;section=50" title="Edit section: References">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<div class="reflist" style="list-style-type: decimal;">
<div class="mw-references-wrap mw-references-columns"><ol class="references">
<li id="cite_note-1"><span class="mw-cite-backlink"><b><a href="#cite_ref-1">^</a></b></span> <span class="reference-text"><cite class="citation web"><a rel="nofollow" class="external text" href="https://theconversation.com/introducing-dark-dna-the-phenomenon-that-could-change-how-we-think-about-evolution-82867">"Introducing 'dark DNA' – the phenomenon that could change how we think about evolution"</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook&amp;rft.genre=unknown&amp;rft.btitle=Introducing+%27dark+DNA%27+%E2%80%93+the+phenomenon+that+could+change+how+we+think+about+evolution&amp;rft_id=https%3A%2F%2Ftheconversation.com%2Fintroducing-dark-dna-the-phenomenon-that-could-change-how-we-think-about-evolution-82867&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-2"><span class="mw-cite-backlink"><b><a href="#cite_ref-2">^</a></b></span> <span class="reference-text"><cite class="citation journal">Behjati S, Tarpey PS (December 2013). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3841808">"What is next generation sequencing?"</a>. <i>Archives of Disease in Childhood. Education and Practice Edition</i>. <b>98</b> (6): 236–8. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1136%2Farchdischild-2013-304340">10.1136/archdischild-2013-304340</a>. <a href="/wiki/PubMed_Central" title="PubMed Central">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3841808">3841808</a></span>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/23986538">23986538</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Archives+of+Disease+in+Childhood.+Education+and+Practice+Edition&amp;rft.atitle=What+is+next+generation+sequencing%3F&amp;rft.volume=98&amp;rft.issue=6&amp;rft.pages=236-8&amp;rft.date=2013-12&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3841808&amp;rft_id=info%3Apmid%2F23986538&amp;rft_id=info%3Adoi%2F10.1136%2Farchdischild-2013-304340&amp;rft.aulast=Behjati&amp;rft.aufirst=S&amp;rft.au=Tarpey%2C+PS&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3841808&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-3"><span class="mw-cite-backlink"><b><a href="#cite_ref-3">^</a></b></span> <span class="reference-text"><cite class="citation journal">Chmielecki J, Meyerson M (14 January 2014). "DNA sequencing of cancer: what have we learned?". <i>Annual Review of Medicine</i>. <b>65</b> (1): 63–79. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1146%2Fannurev-med-060712-200152">10.1146/annurev-med-060712-200152</a>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/24274178">24274178</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Annual+Review+of+Medicine&amp;rft.atitle=DNA+sequencing+of+cancer%3A+what+have+we+learned%3F&amp;rft.volume=65&amp;rft.issue=1&amp;rft.pages=63-79&amp;rft.date=2014-01-14&amp;rft_id=info%3Adoi%2F10.1146%2Fannurev-med-060712-200152&amp;rft_id=info%3Apmid%2F24274178&amp;rft.aulast=Chmielecki&amp;rft.aufirst=J&amp;rft.au=Meyerson%2C+M&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-:3-4"><span class="mw-cite-backlink">^ <a href="#cite_ref-:3_4-0"><sup><i><b>a</b></i></sup></a> <a href="#cite_ref-:3_4-1"><sup><i><b>b</b></i></sup></a> <a href="#cite_ref-:3_4-2"><sup><i><b>c</b></i></sup></a> <a href="#cite_ref-:3_4-3"><sup><i><b>d</b></i></sup></a></span> <span class="reference-text"><cite class="citation journal">Abate AR, Hung T, Sperling RA, Mary P, Rotem A, Agresti JJ,  et al. (December 2013). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC4090915">"DNA sequence analysis with droplet-based microfluidics"</a>. <i>Lab on a Chip</i>. <b>13</b> (24): 4864–9. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1039%2Fc3lc50905b">10.1039/c3lc50905b</a>. <a href="/wiki/PubMed_Central" title="PubMed Central">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC4090915">4090915</a></span>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/24185402">24185402</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Lab+on+a+Chip&amp;rft.atitle=DNA+sequence+analysis+with+droplet-based+microfluidics&amp;rft.volume=13&amp;rft.issue=24&amp;rft.pages=4864-9&amp;rft.date=2013-12&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4090915&amp;rft_id=info%3Apmid%2F24185402&amp;rft_id=info%3Adoi%2F10.1039%2Fc3lc50905b&amp;rft.aulast=Abate&amp;rft.aufirst=AR&amp;rft.au=Hung%2C+T&amp;rft.au=Sperling%2C+RA&amp;rft.au=Mary%2C+P&amp;rft.au=Rotem%2C+A&amp;rft.au=Agresti%2C+JJ&amp;rft.au=Weiner%2C+MA&amp;rft.au=Weitz%2C+DA&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4090915&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-5"><span class="mw-cite-backlink"><b><a href="#cite_ref-5">^</a></b></span> <span class="reference-text"><cite class="citation journal">Pekin D, Skhiri Y, Baret JC, Le Corre D, Mazutis L, Salem CB,  et al. (July 2011). "Quantitative and sensitive detection of rare mutations using droplet-based microfluidics". <i>Lab on a Chip</i>. <b>11</b> (13): 2156–66. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1039%2Fc1lc20128j">10.1039/c1lc20128j</a>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/21594292">21594292</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Lab+on+a+Chip&amp;rft.atitle=Quantitative+and+sensitive+detection+of+rare+mutations+using+droplet-based+microfluidics&amp;rft.volume=11&amp;rft.issue=13&amp;rft.pages=2156-66&amp;rft.date=2011-07&amp;rft_id=info%3Adoi%2F10.1039%2Fc1lc20128j&amp;rft_id=info%3Apmid%2F21594292&amp;rft.aulast=Pekin&amp;rft.aufirst=D&amp;rft.au=Skhiri%2C+Y&amp;rft.au=Baret%2C+JC&amp;rft.au=Le+Corre%2C+D&amp;rft.au=Mazutis%2C+L&amp;rft.au=Salem%2C+CB&amp;rft.au=Millot%2C+F&amp;rft.au=El+Harrak%2C+A&amp;rft.au=Hutchison%2C+JB&amp;rft.au=Larson%2C+JW&amp;rft.au=Link%2C+DR&amp;rft.au=Laurent-Puig%2C+P&amp;rft.au=Griffiths%2C+AD&amp;rft.au=Taly%2C+V&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-olsvik1993-6"><span class="mw-cite-backlink"><b><a href="#cite_ref-olsvik1993_6-0">^</a></b></span> <span class="reference-text"><cite class="citation journal">Olsvik O, Wahlberg J, Petterson B, Uhlén M, Popovic T, Wachsmuth IK, Fields PI (January 1993). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC262614">"Use of automated sequencing of polymerase chain reaction-generated amplicons to identify three types of cholera toxin subunit B in Vibrio cholerae O1 strains"</a>. <i><a href="/wiki/J._Clin._Microbiol." class="mw-redirect" title="J. Clin. Microbiol.">J. Clin. Microbiol.</a></i> <b>31</b> (1): 22–25. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1128%2FJCM.31.1.22-25.1993">10.1128/JCM.31.1.22-25.1993</a>. <a href="/wiki/PubMed_Central" title="PubMed Central">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC262614">262614</a></span>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/7678018">7678018</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=J.+Clin.+Microbiol.&amp;rft.atitle=Use+of+automated+sequencing+of+polymerase+chain+reaction-generated+amplicons+to+identify+three+types+of+cholera+toxin+subunit+B+in+Vibrio+cholerae+O1+strains&amp;rft.volume=31&amp;rft.issue=1&amp;rft.pages=22-25&amp;rft.date=1993-01&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC262614&amp;rft_id=info%3Apmid%2F7678018&amp;rft_id=info%3Adoi%2F10.1128%2FJCM.31.1.22-25.1993&amp;rft.aulast=Olsvik&amp;rft.aufirst=O&amp;rft.au=Wahlberg%2C+J&amp;rft.au=Petterson%2C+B&amp;rft.au=Uhl%C3%A9n%2C+M&amp;rft.au=Popovic%2C+T&amp;rft.au=Wachsmuth%2C+IK&amp;rft.au=Fields%2C+PI&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC262614&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/><span style="position:relative; top: -2px;"><a href="/wiki/Open_access" title="open access publication – free to read"><img alt="open access" src="//upload.wikimedia.org/wikipedia/commons/thumb/7/77/Open_Access_logo_PLoS_transparent.svg/9px-Open_Access_logo_PLoS_transparent.svg.png" decoding="async" width="9" height="14" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/7/77/Open_Access_logo_PLoS_transparent.svg/14px-Open_Access_logo_PLoS_transparent.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/7/77/Open_Access_logo_PLoS_transparent.svg/18px-Open_Access_logo_PLoS_transparent.svg.png 2x" data-file-width="640" data-file-height="1000" /></a></span></span>
</li>
<li id="cite_note-pmid18992322-7"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid18992322_7-0">^</a></b></span> <span class="reference-text"><cite class="citation journal">Pettersson E, Lundeberg J, Ahmadian A (February 2009). "Generations of sequencing technologies". <i>Genomics</i>. <b>93</b> (2): 105–11. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1016%2Fj.ygeno.2008.10.003">10.1016/j.ygeno.2008.10.003</a>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/18992322">18992322</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Genomics&amp;rft.atitle=Generations+of+sequencing+technologies&amp;rft.volume=93&amp;rft.issue=2&amp;rft.pages=105-11&amp;rft.date=2009-02&amp;rft_id=info%3Adoi%2F10.1016%2Fj.ygeno.2008.10.003&amp;rft_id=info%3Apmid%2F18992322&amp;rft.aulast=Pettersson&amp;rft.aufirst=E&amp;rft.au=Lundeberg%2C+J&amp;rft.au=Ahmadian%2C+A&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-:0c-8"><span class="mw-cite-backlink">^ <a href="#cite_ref-:0c_8-0"><sup><i><b>a</b></i></sup></a> <a href="#cite_ref-:0c_8-1"><sup><i><b>b</b></i></sup></a></span> <span class="reference-text"><cite class="citation journal">Castro, Christina; Marine, Rachel; Ramos, Edward; Ng, Terry Fei Fan (2019). <a rel="nofollow" class="external text" href="https://www.biorxiv.org/content/10.1101/815480v1">"The effect of variant interference on de novo assembly for viral deep sequencing"</a>. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1101%2F815480">10.1101/815480</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.atitle=The+effect+of+variant+interference+on+de+novo+assembly+for+viral+deep+sequencing&amp;rft.date=2019&amp;rft_id=info%3Adoi%2F10.1101%2F815480&amp;rft.aulast=Castro&amp;rft.aufirst=Christina&amp;rft.au=Marine%2C+Rachel&amp;rft.au=Ramos%2C+Edward&amp;rft.au=Ng%2C+Terry+Fei+Fan&amp;rft_id=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F815480v1&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span> <span class="cs1-hidden-error error citation-comment">Cite journal requires <code class="cs1-code">&#124;journal=</code> (<a href="/wiki/Help:CS1_errors#missing_periodical" title="Help:CS1 errors">help</a>)</span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-9"><span class="mw-cite-backlink"><b><a href="#cite_ref-9">^</a></b></span> <span class="reference-text"><a rel="nofollow" class="external text" href="https://www.ncbi.nlm.nih.gov/pubmed/28425484CASTB">Mycobacterium tuberculosis resistance prediction and lineage classification from genome sequencing: comparison of automated analysis tools.</a></span>
</li>
<li id="cite_note-10"><span class="mw-cite-backlink"><b><a href="#cite_ref-10">^</a></b></span> <span class="reference-text"><a rel="nofollow" class="external text" href="https://www.ncbi.nlm.nih.gov/pubmed/31106066">A large scale evaluation of TBProfiler and Mykrobe for antibiotic resistance prediction in Mycobacterium tuberculosis.</a></span>
</li>
<li id="cite_note-11"><span class="mw-cite-backlink"><b><a href="#cite_ref-11">^</a></b></span> <span class="reference-text"><a rel="nofollow" class="external text" href="https://innovation.ox.ac.uk/wp-content/uploads/2015/04/Mykrobe-predictor-Poster.pdf">Mykrobe predictor –Antibiotic resistance prediction for S. aureus and M. tuberculosis from whole genome sequence data</a></span>
</li>
<li id="cite_note-12"><span class="mw-cite-backlink"><b><a href="#cite_ref-12">^</a></b></span> <span class="reference-text"><a rel="nofollow" class="external text" href="https://www.nature.com/articles/ncomms10063">Rapid antibiotic-resistance predictions from genome sequence data for Staphylococcus aureus and Mycobacterium tuberculosis</a></span>
</li>
<li id="cite_note-13"><span class="mw-cite-backlink"><b><a href="#cite_ref-13">^</a></b></span> <span class="reference-text"><a rel="nofollow" class="external text" href="https://www.tvo.org/transcript/115187X/michael-mosley-vs-the-superbugs">Michael Mosley vs the superbugs</a></span>
</li>
<li id="cite_note-14"><span class="mw-cite-backlink"><b><a href="#cite_ref-14">^</a></b></span> <span class="reference-text"><a rel="nofollow" class="external text" href="https://github.com/Mykrobe-tools/mykrobe">Mykrobe Predictor github</a></span>
</li>
<li id="cite_note-15"><span class="mw-cite-backlink"><b><a href="#cite_ref-15">^</a></b></span> <span class="reference-text"><cite class="citation news">Curtis C, Hereward J (29 August 2017). <a rel="nofollow" class="external text" href="https://theconversation.com/from-the-crime-scene-to-the-courtroom-the-journey-of-a-dna-sample-82250">"From the crime scene to the courtroom: the journey of a DNA sample"</a>. <i>The Conversation</i>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=The+Conversation&amp;rft.atitle=From+the+crime+scene+to+the+courtroom%3A+the+journey+of+a+DNA+sample&amp;rft.date=2017-08-29&amp;rft.aulast=Curtis&amp;rft.aufirst=Caitlin&amp;rft.au=Hereward%2C+James&amp;rft_id=https%3A%2F%2Ftheconversation.com%2Ffrom-the-crime-scene-to-the-courtroom-the-journey-of-a-dna-sample-82250&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-16"><span class="mw-cite-backlink"><b><a href="#cite_ref-16">^</a></b></span> <span class="reference-text"><cite class="citation journal">Moréra S, Larivière L, Kurzeck J, Aschke-Sonnenborn U, Freemont PS, Janin J, Rüger W (August 2001). "High resolution crystal structures of T4 phage beta-glucosyltransferase: induced fit and effect of substrate and metal binding". <i>Journal of Molecular Biology</i>. <b>311</b> (3): 569–77. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1006%2Fjmbi.2001.4905">10.1006/jmbi.2001.4905</a>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/11493010">11493010</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Journal+of+Molecular+Biology&amp;rft.atitle=High+resolution+crystal+structures+of+T4+phage+beta-glucosyltransferase%3A+induced+fit+and+effect+of+substrate+and+metal+binding&amp;rft.volume=311&amp;rft.issue=3&amp;rft.pages=569-77&amp;rft.date=2001-08&amp;rft_id=info%3Adoi%2F10.1006%2Fjmbi.2001.4905&amp;rft_id=info%3Apmid%2F11493010&amp;rft.aulast=Mor%C3%A9ra&amp;rft.aufirst=S&amp;rft.au=Larivi%C3%A8re%2C+L&amp;rft.au=Kurzeck%2C+J&amp;rft.au=Aschke-Sonnenborn%2C+U&amp;rft.au=Freemont%2C+PS&amp;rft.au=Janin%2C+J&amp;rft.au=R%C3%BCger%2C+W&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-17"><span class="mw-cite-backlink"><b><a href="#cite_ref-17">^</a></b></span> <span class="reference-text"><cite class="citation journal">Ehrlich M, Gama-Sosa MA, Huang LH, Midgett RM, Kuo KC, McCune RA, Gehrke C (April 1982). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC320645">"Amount and distribution of 5-methylcytosine in human DNA from different types of tissues of cells"</a>. <i>Nucleic Acids Research</i>. <b>10</b> (8): 2709–21. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fnar%2F10.8.2709">10.1093/nar/10.8.2709</a>. <a href="/wiki/PubMed_Central" title="PubMed Central">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC320645">320645</a></span>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/7079182">7079182</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Nucleic+Acids+Research&amp;rft.atitle=Amount+and+distribution+of+5-methylcytosine+in+human+DNA+from+different+types+of+tissues+of+cells&amp;rft.volume=10&amp;rft.issue=8&amp;rft.pages=2709-21&amp;rft.date=1982-04&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC320645&amp;rft_id=info%3Apmid%2F7079182&amp;rft_id=info%3Adoi%2F10.1093%2Fnar%2F10.8.2709&amp;rft.aulast=Ehrlich&amp;rft.aufirst=M&amp;rft.au=Gama-Sosa%2C+MA&amp;rft.au=Huang%2C+LH&amp;rft.au=Midgett%2C+RM&amp;rft.au=Kuo%2C+KC&amp;rft.au=McCune%2C+RA&amp;rft.au=Gehrke%2C+C&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC320645&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-18"><span class="mw-cite-backlink"><b><a href="#cite_ref-18">^</a></b></span> <span class="reference-text"><cite class="citation journal">Ehrlich M, Wang RY (June 1981). "5-Methylcytosine in eukaryotic DNA". <i>Science</i>. <b>212</b> (4501): 1350–7. <a href="/wiki/Bibcode" title="Bibcode">Bibcode</a>:<a rel="nofollow" class="external text" href="https://ui.adsabs.harvard.edu/abs/1981Sci...212.1350E">1981Sci...212.1350E</a>. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1126%2Fscience.6262918">10.1126/science.6262918</a>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/6262918">6262918</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Science&amp;rft.atitle=5-Methylcytosine+in+eukaryotic+DNA&amp;rft.volume=212&amp;rft.issue=4501&amp;rft.pages=1350-7&amp;rft.date=1981-06&amp;rft_id=info%3Apmid%2F6262918&amp;rft_id=info%3Adoi%2F10.1126%2Fscience.6262918&amp;rft_id=info%3Abibcode%2F1981Sci...212.1350E&amp;rft.aulast=Ehrlich&amp;rft.aufirst=M&amp;rft.au=Wang%2C+RY&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-19"><span class="mw-cite-backlink"><b><a href="#cite_ref-19">^</a></b></span> <span class="reference-text"><cite class="citation journal">Song CX, Clark TA, Lu XY, Kislyuk A, Dai Q, Turner SW,  et al. (November 2011). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3646335">"Sensitive and specific single-molecule sequencing of 5-hydroxymethylcytosine"</a>. <i>Nature Methods</i>. <b>9</b> (1): 75–7. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1038%2Fnmeth.1779">10.1038/nmeth.1779</a>. <a href="/wiki/PubMed_Central" title="PubMed Central">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3646335">3646335</a></span>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/22101853">22101853</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Nature+Methods&amp;rft.atitle=Sensitive+and+specific+single-molecule+sequencing+of+5-hydroxymethylcytosine&amp;rft.volume=9&amp;rft.issue=1&amp;rft.pages=75-7&amp;rft.date=2011-11&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3646335&amp;rft_id=info%3Apmid%2F22101853&amp;rft_id=info%3Adoi%2F10.1038%2Fnmeth.1779&amp;rft.aulast=Song&amp;rft.aufirst=CX&amp;rft.au=Clark%2C+TA&amp;rft.au=Lu%2C+XY&amp;rft.au=Kislyuk%2C+A&amp;rft.au=Dai%2C+Q&amp;rft.au=Turner%2C+SW&amp;rft.au=He%2C+C&amp;rft.au=Korlach%2C+J&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3646335&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-pmid13168976-20"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid13168976_20-0">^</a></b></span> <span class="reference-text"><cite class="citation journal">Watson JD, Crick FH (1953). "The structure of DNA". <i>Cold Spring Harb. Symp. Quant. Biol</i>. <b>18</b>: 123–31. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1101%2FSQB.1953.018.01.020">10.1101/SQB.1953.018.01.020</a>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/13168976">13168976</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Cold+Spring+Harb.+Symp.+Quant.+Biol.&amp;rft.atitle=The+structure+of+DNA&amp;rft.volume=18&amp;rft.pages=123-31&amp;rft.date=1953&amp;rft_id=info%3Adoi%2F10.1101%2FSQB.1953.018.01.020&amp;rft_id=info%3Apmid%2F13168976&amp;rft.aulast=Watson&amp;rft.aufirst=JD&amp;rft.au=Crick%2C+FH&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-21"><span class="mw-cite-backlink"><b><a href="#cite_ref-21">^</a></b></span> <span class="reference-text"><a rel="nofollow" class="external text" href="http://www.whatisbiotechnology.org/exhibitions/sanger/path">Marks, L, The path to DNA sequencing: The life and work of Frederick Sanger</a>.</span>
</li>
<li id="cite_note-22"><span class="mw-cite-backlink"><b><a href="#cite_ref-22">^</a></b></span> <span class="reference-text"><cite class="citation journal">Min Jou W, Haegeman G, Ysebaert M, Fiers W (May 1972). "Nucleotide sequence of the gene coding for the bacteriophage MS2 coat protein". <i>Nature</i>. <b>237</b> (5350): 82–8. <a href="/wiki/Bibcode" title="Bibcode">Bibcode</a>:<a rel="nofollow" class="external text" href="https://ui.adsabs.harvard.edu/abs/1972Natur.237...82J">1972Natur.237...82J</a>. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1038%2F237082a0">10.1038/237082a0</a>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/4555447">4555447</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Nature&amp;rft.atitle=Nucleotide+sequence+of+the+gene+coding+for+the+bacteriophage+MS2+coat+protein&amp;rft.volume=237&amp;rft.issue=5350&amp;rft.pages=82-8&amp;rft.date=1972-05&amp;rft_id=info%3Apmid%2F4555447&amp;rft_id=info%3Adoi%2F10.1038%2F237082a0&amp;rft_id=info%3Abibcode%2F1972Natur.237...82J&amp;rft.aulast=Min+Jou&amp;rft.aufirst=W&amp;rft.au=Haegeman%2C+G&amp;rft.au=Ysebaert%2C+M&amp;rft.au=Fiers%2C+W&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-23"><span class="mw-cite-backlink"><b><a href="#cite_ref-23">^</a></b></span> <span class="reference-text"><cite class="citation journal">Fiers W, Contreras R, Duerinck F, Haegeman G, Iserentant D, Merregaert J, Min Jou W, Molemans F, Raeymaekers A, Van den Berghe A, Volckaert G, Ysebaert M (April 1976). "Complete nucleotide sequence of bacteriophage MS2 RNA: primary and secondary structure of the replicase gene". <i>Nature</i>. <b>260</b> (5551): 500–7. <a href="/wiki/Bibcode" title="Bibcode">Bibcode</a>:<a rel="nofollow" class="external text" href="https://ui.adsabs.harvard.edu/abs/1976Natur.260..500F">1976Natur.260..500F</a>. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1038%2F260500a0">10.1038/260500a0</a>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/1264203">1264203</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Nature&amp;rft.atitle=Complete+nucleotide+sequence+of+bacteriophage+MS2+RNA%3A+primary+and+secondary+structure+of+the+replicase+gene&amp;rft.volume=260&amp;rft.issue=5551&amp;rft.pages=500-7&amp;rft.date=1976-04&amp;rft_id=info%3Apmid%2F1264203&amp;rft_id=info%3Adoi%2F10.1038%2F260500a0&amp;rft_id=info%3Abibcode%2F1976Natur.260..500F&amp;rft.aulast=Fiers&amp;rft.aufirst=W&amp;rft.au=Contreras%2C+R&amp;rft.au=Duerinck%2C+F&amp;rft.au=Haegeman%2C+G&amp;rft.au=Iserentant%2C+D&amp;rft.au=Merregaert%2C+J&amp;rft.au=Min+Jou%2C+W&amp;rft.au=Molemans%2C+F&amp;rft.au=Raeymaekers%2C+A&amp;rft.au=Van+den+Berghe%2C+A&amp;rft.au=Volckaert%2C+G&amp;rft.au=Ysebaert%2C+M&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-24"><span class="mw-cite-backlink"><b><a href="#cite_ref-24">^</a></b></span> <span class="reference-text"><cite class="citation journal">Ozsolak F, Milos PM (February 2011). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3031867">"RNA sequencing: advances, challenges and opportunities"</a>. <i>Nature Reviews. Genetics</i>. <b>12</b> (2): 87–98. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1038%2Fnrg2934">10.1038/nrg2934</a>. <a href="/wiki/PubMed_Central" title="PubMed Central">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3031867">3031867</a></span>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/21191423">21191423</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Nature+Reviews.+Genetics&amp;rft.atitle=RNA+sequencing%3A+advances%2C+challenges+and+opportunities&amp;rft.volume=12&amp;rft.issue=2&amp;rft.pages=87-98&amp;rft.date=2011-02&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3031867&amp;rft_id=info%3Apmid%2F21191423&amp;rft_id=info%3Adoi%2F10.1038%2Fnrg2934&amp;rft.aulast=Ozsolak&amp;rft.aufirst=F&amp;rft.au=Milos%2C+PM&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3031867&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-25"><span class="mw-cite-backlink"><b><a href="#cite_ref-25">^</a></b></span> <span class="reference-text"><cite class="citation web"><a rel="nofollow" class="external text" href="https://web.archive.org/web/20090304121126/http://www.mbg.cornell.edu/faculty-staff/faculty/wu.cfm">"Ray Wu Faculty Profile"</a>. Cornell University. Archived from <a rel="nofollow" class="external text" href="http://www.mbg.cornell.edu/faculty-staff/faculty/wu.cfm">the original</a> on 4 March 2009.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook&amp;rft.genre=unknown&amp;rft.btitle=Ray+Wu+Faculty+Profile&amp;rft.pub=Cornell+University&amp;rft_id=http%3A%2F%2Fwww.mbg.cornell.edu%2Ffaculty-staff%2Ffaculty%2Fwu.cfm&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-26"><span class="mw-cite-backlink"><b><a href="#cite_ref-26">^</a></b></span> <span class="reference-text"><cite class="citation journal">Padmanabhan R, Jay E, Wu R (June 1974). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC388489">"Chemical synthesis of a primer and its use in the sequence analysis of the lysozyme gene of bacteriophage T4"</a>. <i>Proceedings of the National Academy of Sciences of the United States of America</i>. <b>71</b> (6): 2510–4. <a href="/wiki/Bibcode" title="Bibcode">Bibcode</a>:<a rel="nofollow" class="external text" href="https://ui.adsabs.harvard.edu/abs/1974PNAS...71.2510P">1974PNAS...71.2510P</a>. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1073%2Fpnas.71.6.2510">10.1073/pnas.71.6.2510</a>. <a href="/wiki/PubMed_Central" title="PubMed Central">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC388489">388489</a></span>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/4526223">4526223</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Proceedings+of+the+National+Academy+of+Sciences+of+the+United+States+of+America&amp;rft.atitle=Chemical+synthesis+of+a+primer+and+its+use+in+the+sequence+analysis+of+the+lysozyme+gene+of+bacteriophage+T4&amp;rft.volume=71&amp;rft.issue=6&amp;rft.pages=2510-4&amp;rft.date=1974-06&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC388489&amp;rft_id=info%3Apmid%2F4526223&amp;rft_id=info%3Adoi%2F10.1073%2Fpnas.71.6.2510&amp;rft_id=info%3Abibcode%2F1974PNAS...71.2510P&amp;rft.aulast=Padmanabhan&amp;rft.aufirst=R&amp;rft.au=Jay%2C+E&amp;rft.au=Wu%2C+R&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC388489&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-27"><span class="mw-cite-backlink"><b><a href="#cite_ref-27">^</a></b></span> <span class="reference-text"><cite class="citation journal">Onaga LA (June 2014). "Ray Wu as Fifth Business: Demonstrating Collective Memory in the History of DNA Sequencing". <i>Studies in the History and Philosophy of Science</i>. Part C. <b>46</b>: 1–14. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1016%2Fj.shpsc.2013.12.006">10.1016/j.shpsc.2013.12.006</a>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/24565976">24565976</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Studies+in+the+History+and+Philosophy+of+Science&amp;rft.atitle=Ray+Wu+as+Fifth+Business%3A+Demonstrating+Collective+Memory+in+the+History+of+DNA+Sequencing&amp;rft.volume=46&amp;rft.pages=1-14&amp;rft.date=2014-06&amp;rft_id=info%3Adoi%2F10.1016%2Fj.shpsc.2013.12.006&amp;rft_id=info%3Apmid%2F24565976&amp;rft.aulast=Onaga&amp;rft.aufirst=LA&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-pmid4553110-28"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid4553110_28-0">^</a></b></span> <span class="reference-text"><cite class="citation journal">Wu R (1972). "Nucleotide sequence analysis of DNA". <i>Nature New Biology</i>. <b>236</b> (68): 198–200. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1038%2Fnewbio236198a0">10.1038/newbio236198a0</a>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/4553110">4553110</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Nature+New+Biology&amp;rft.atitle=Nucleotide+sequence+analysis+of+DNA&amp;rft.volume=236&amp;rft.issue=68&amp;rft.pages=198-200&amp;rft.date=1972&amp;rft_id=info%3Adoi%2F10.1038%2Fnewbio236198a0&amp;rft_id=info%3Apmid%2F4553110&amp;rft.aulast=Wu&amp;rft.aufirst=R&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-pmid4560009-29"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid4560009_29-0">^</a></b></span> <span class="reference-text"><cite class="citation journal">Padmanabhan R, Wu R (1972). "Nucleotide sequence analysis of DNA. IX. Use of oligonucleotides of defined sequence as primers in DNA sequence analysis". <i>Biochem. Biophys. Res. Commun</i>. <b>48</b> (5): 1295–302. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1016%2F0006-291X%2872%2990852-2">10.1016/0006-291X(72)90852-2</a>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/4560009">4560009</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Biochem.+Biophys.+Res.+Commun.&amp;rft.atitle=Nucleotide+sequence+analysis+of+DNA.+IX.+Use+of+oligonucleotides+of+defined+sequence+as+primers+in+DNA+sequence+analysis&amp;rft.volume=48&amp;rft.issue=5&amp;rft.pages=1295-302&amp;rft.date=1972&amp;rft_id=info%3Adoi%2F10.1016%2F0006-291X%2872%2990852-2&amp;rft_id=info%3Apmid%2F4560009&amp;rft.aulast=Padmanabhan&amp;rft.aufirst=R&amp;rft.au=Wu%2C+R&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-pmid4358929-30"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid4358929_30-0">^</a></b></span> <span class="reference-text"><cite class="citation journal">Wu R, Tu CD, Padmanabhan R (1973). "Nucleotide sequence analysis of DNA. XII. The chemical synthesis and sequence analysis of a dodecadeoxynucleotide which binds to the endolysin gene of bacteriophage lambda". <i>Biochem. Biophys. Res. Commun</i>. <b>55</b> (4): 1092–99. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1016%2FS0006-291X%2873%2980007-5">10.1016/S0006-291X(73)80007-5</a>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/4358929">4358929</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Biochem.+Biophys.+Res.+Commun.&amp;rft.atitle=Nucleotide+sequence+analysis+of+DNA.+XII.+The+chemical+synthesis+and+sequence+analysis+of+a+dodecadeoxynucleotide+which+binds+to+the+endolysin+gene+of+bacteriophage+lambda&amp;rft.volume=55&amp;rft.issue=4&amp;rft.pages=1092-99&amp;rft.date=1973&amp;rft_id=info%3Adoi%2F10.1016%2FS0006-291X%2873%2980007-5&amp;rft_id=info%3Apmid%2F4358929&amp;rft.aulast=Wu&amp;rft.aufirst=R&amp;rft.au=Tu%2C+CD&amp;rft.au=Padmanabhan%2C+R&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-31"><span class="mw-cite-backlink"><b><a href="#cite_ref-31">^</a></b></span> <span class="reference-text"><cite class="citation journal">Jay E, Bambara R, Padmanabhan R, Wu R (March 1974). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC344020">"DNA sequence analysis: a general, simple and rapid method for sequencing large oligodeoxyribonucleotide fragments by mapping"</a>. <i>Nucleic Acids Research</i>. <b>1</b> (3): 331–53. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fnar%2F1.3.331">10.1093/nar/1.3.331</a>. <a href="/wiki/PubMed_Central" title="PubMed Central">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC344020">344020</a></span>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/10793670">10793670</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Nucleic+Acids+Research&amp;rft.atitle=DNA+sequence+analysis%3A+a+general%2C+simple+and+rapid+method+for+sequencing+large+oligodeoxyribonucleotide+fragments+by+mapping&amp;rft.volume=1&amp;rft.issue=3&amp;rft.pages=331-53&amp;rft.date=1974-03&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC344020&amp;rft_id=info%3Apmid%2F10793670&amp;rft_id=info%3Adoi%2F10.1093%2Fnar%2F1.3.331&amp;rft.aulast=Jay&amp;rft.aufirst=E&amp;rft.au=Bambara%2C+R&amp;rft.au=Padmanabhan%2C+R&amp;rft.au=Wu%2C+R&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC344020&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-Sanger1977-32"><span class="mw-cite-backlink">^ <a href="#cite_ref-Sanger1977_32-0"><sup><i><b>a</b></i></sup></a> <a href="#cite_ref-Sanger1977_32-1"><sup><i><b>b</b></i></sup></a></span> <span class="reference-text"><cite class="citation journal">Sanger F, Nicklen S, Coulson AR (December 1977). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC431765">"DNA sequencing with chain-terminating inhibitors"</a>. <i>Proc. Natl. Acad. Sci. USA</i>. <b>74</b> (12): 5463–77. <a href="/wiki/Bibcode" title="Bibcode">Bibcode</a>:<a rel="nofollow" class="external text" href="https://ui.adsabs.harvard.edu/abs/1977PNAS...74.5463S">1977PNAS...74.5463S</a>. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1073%2Fpnas.74.12.5463">10.1073/pnas.74.12.5463</a>. <a href="/wiki/PubMed_Central" title="PubMed Central">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC431765">431765</a></span>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/271968">271968</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Proc.+Natl.+Acad.+Sci.+USA&amp;rft.atitle=DNA+sequencing+with+chain-terminating+inhibitors&amp;rft.volume=74&amp;rft.issue=12&amp;rft.pages=5463-77&amp;rft.date=1977-12&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC431765&amp;rft_id=info%3Apmid%2F271968&amp;rft_id=info%3Adoi%2F10.1073%2Fpnas.74.12.5463&amp;rft_id=info%3Abibcode%2F1977PNAS...74.5463S&amp;rft.aulast=Sanger&amp;rft.aufirst=F&amp;rft.au=Nicklen%2C+S&amp;rft.au=Coulson%2C+AR&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC431765&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-Maxam77-33"><span class="mw-cite-backlink">^ <a href="#cite_ref-Maxam77_33-0"><sup><i><b>a</b></i></sup></a> <a href="#cite_ref-Maxam77_33-1"><sup><i><b>b</b></i></sup></a> <a href="#cite_ref-Maxam77_33-2"><sup><i><b>c</b></i></sup></a></span> <span class="reference-text"><cite class="citation journal">Maxam AM, Gilbert W (February 1977). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC392330">"A new method for sequencing DNA"</a>. <i>Proc. Natl. Acad. Sci. USA</i>. <b>74</b> (2): 560–64. <a href="/wiki/Bibcode" title="Bibcode">Bibcode</a>:<a rel="nofollow" class="external text" href="https://ui.adsabs.harvard.edu/abs/1977PNAS...74..560M">1977PNAS...74..560M</a>. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1073%2Fpnas.74.2.560">10.1073/pnas.74.2.560</a>. <a href="/wiki/PubMed_Central" title="PubMed Central">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC392330">392330</a></span>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/265521">265521</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Proc.+Natl.+Acad.+Sci.+USA&amp;rft.atitle=A+new+method+for+sequencing+DNA&amp;rft.volume=74&amp;rft.issue=2&amp;rft.pages=560-64&amp;rft.date=1977-02&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC392330&amp;rft_id=info%3Apmid%2F265521&amp;rft_id=info%3Adoi%2F10.1073%2Fpnas.74.2.560&amp;rft_id=info%3Abibcode%2F1977PNAS...74..560M&amp;rft.aulast=Maxam&amp;rft.aufirst=AM&amp;rft.au=Gilbert%2C+W&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC392330&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-34"><span class="mw-cite-backlink"><b><a href="#cite_ref-34">^</a></b></span> <span class="reference-text">Gilbert, W. <a rel="nofollow" class="external text" href="http://nobelprize.org/nobel_prizes/chemistry/laureates/1980/gilbert-lecture.pdf">DNA sequencing and gene structure</a>. Nobel lecture, 8 December 1980.</span>
</li>
<li id="cite_note-35"><span class="mw-cite-backlink"><b><a href="#cite_ref-35">^</a></b></span> <span class="reference-text"><cite class="citation journal">Gilbert W, Maxam A (December 1973). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC427284">"The Nucleotide Sequence of the lac Operator"</a>. <i>Proc. Natl. Acad. Sci. U.S.A</i>. <b>70</b> (12): 3581–84. <a href="/wiki/Bibcode" title="Bibcode">Bibcode</a>:<a rel="nofollow" class="external text" href="https://ui.adsabs.harvard.edu/abs/1973PNAS...70.3581G">1973PNAS...70.3581G</a>. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1073%2Fpnas.70.12.3581">10.1073/pnas.70.12.3581</a>. <a href="/wiki/PubMed_Central" title="PubMed Central">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC427284">427284</a></span>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/4587255">4587255</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Proc.+Natl.+Acad.+Sci.+U.S.A.&amp;rft.atitle=The+Nucleotide+Sequence+of+the+lac+Operator&amp;rft.volume=70&amp;rft.issue=12&amp;rft.pages=3581-84&amp;rft.date=1973-12&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC427284&amp;rft_id=info%3Apmid%2F4587255&amp;rft_id=info%3Adoi%2F10.1073%2Fpnas.70.12.3581&amp;rft_id=info%3Abibcode%2F1973PNAS...70.3581G&amp;rft.aulast=Gilbert&amp;rft.aufirst=W&amp;rft.au=Maxam%2C+A&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC427284&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-36"><span class="mw-cite-backlink"><b><a href="#cite_ref-36">^</a></b></span> <span class="reference-text"><cite class="citation journal">Sanger F, Air GM, Barrell BG, Brown NL, Coulson AR, Fiddes CA, Hutchison CA, Slocombe PM, Smith M (February 1977). "Nucleotide sequence of bacteriophage phi X174 DNA". <i>Nature</i>. <b>265</b> (5596): 687–95. <a href="/wiki/Bibcode" title="Bibcode">Bibcode</a>:<a rel="nofollow" class="external text" href="https://ui.adsabs.harvard.edu/abs/1977Natur.265..687S">1977Natur.265..687S</a>. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1038%2F265687a0">10.1038/265687a0</a>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/870828">870828</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Nature&amp;rft.atitle=Nucleotide+sequence+of+bacteriophage+phi+X174+DNA&amp;rft.volume=265&amp;rft.issue=5596&amp;rft.pages=687-95&amp;rft.date=1977-02&amp;rft_id=info%3Apmid%2F870828&amp;rft_id=info%3Adoi%2F10.1038%2F265687a0&amp;rft_id=info%3Abibcode%2F1977Natur.265..687S&amp;rft.aulast=Sanger&amp;rft.aufirst=F&amp;rft.au=Air%2C+GM&amp;rft.au=Barrell%2C+BG&amp;rft.au=Brown%2C+NL&amp;rft.au=Coulson%2C+AR&amp;rft.au=Fiddes%2C+CA&amp;rft.au=Hutchison%2C+CA&amp;rft.au=Slocombe%2C+PM&amp;rft.au=Smith%2C+M&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-37"><span class="mw-cite-backlink"><b><a href="#cite_ref-37">^</a></b></span> <span class="reference-text"><a rel="nofollow" class="external text" href="http://www.whatisbiotechnology.org/exhibitions/sanger/sequencing">"The Next Frontier: Human Viruses"</a> , whatisbiotechnology.org, Retrieved 3 May 2017</span>
</li>
<li id="cite_note-38"><span class="mw-cite-backlink"><b><a href="#cite_ref-38">^</a></b></span> <span class="reference-text"><cite class="citation journal">Beck S, Pohl FM (1984). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC557787">"DNA sequencing with direct blotting electrophoresis"</a>. <i>EMBO J</i>. <b>3</b> (12): 2905–09. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1002%2Fj.1460-2075.1984.tb02230.x">10.1002/j.1460-2075.1984.tb02230.x</a>. <a href="/wiki/PubMed_Central" title="PubMed Central">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC557787">557787</a></span>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/6396083">6396083</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=EMBO+J&amp;rft.atitle=DNA+sequencing+with+direct+blotting+electrophoresis&amp;rft.volume=3&amp;rft.issue=12&amp;rft.pages=2905-09&amp;rft.date=1984&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC557787&amp;rft_id=info%3Apmid%2F6396083&amp;rft_id=info%3Adoi%2F10.1002%2Fj.1460-2075.1984.tb02230.x&amp;rft.aulast=Beck&amp;rft.aufirst=S&amp;rft.au=Pohl%2C+FM&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC557787&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-39"><span class="mw-cite-backlink"><b><a href="#cite_ref-39">^</a></b></span> <span class="reference-text">United States Patent 4,631,122 (1986)</span>
</li>
<li id="cite_note-Feldmann_1994-40"><span class="mw-cite-backlink"><b><a href="#cite_ref-Feldmann_1994_40-0">^</a></b></span> <span class="reference-text"><cite class="citation journal">Feldmann H,  et al. (1994). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC395553">"Complete DNA sequence of yeast chromosome II"</a>. <i>EMBO J</i>. <b>13</b> (24): 5795–809. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1002%2Fj.1460-2075.1994.tb06923.x">10.1002/j.1460-2075.1994.tb06923.x</a>. <a href="/wiki/PubMed_Central" title="PubMed Central">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC395553">395553</a></span>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/7813418">7813418</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=EMBO+J.&amp;rft.atitle=Complete+DNA+sequence+of+yeast+chromosome+II&amp;rft.volume=13&amp;rft.issue=24&amp;rft.pages=5795-809&amp;rft.date=1994&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC395553&amp;rft_id=info%3Apmid%2F7813418&amp;rft_id=info%3Adoi%2F10.1002%2Fj.1460-2075.1994.tb06923.x&amp;rft.aulast=Feldmann&amp;rft.aufirst=H&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC395553&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-41"><span class="mw-cite-backlink"><b><a href="#cite_ref-41">^</a></b></span> <span class="reference-text"><cite class="citation journal">Smith LM, Sanders JZ, Kaiser RJ, Hughes P, Dodd C, Connell CR, Heiner C, Kent SB, Hood LE (12 June 1986). "Fluorescence Detection in Automated DNA Sequence Analysis". <i>Nature</i>. <b>321</b> (6071): 674–79. <a href="/wiki/Bibcode" title="Bibcode">Bibcode</a>:<a rel="nofollow" class="external text" href="https://ui.adsabs.harvard.edu/abs/1986Natur.321..674S">1986Natur.321..674S</a>. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1038%2F321674a0">10.1038/321674a0</a>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/3713851">3713851</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Nature&amp;rft.atitle=Fluorescence+Detection+in+Automated+DNA+Sequence+Analysis&amp;rft.volume=321&amp;rft.issue=6071&amp;rft.pages=674-79&amp;rft.date=1986-06-12&amp;rft_id=info%3Apmid%2F3713851&amp;rft_id=info%3Adoi%2F10.1038%2F321674a0&amp;rft_id=info%3Abibcode%2F1986Natur.321..674S&amp;rft.aulast=Smith&amp;rft.aufirst=LM&amp;rft.au=Sanders%2C+JZ&amp;rft.au=Kaiser%2C+RJ&amp;rft.au=Hughes%2C+P&amp;rft.au=Dodd%2C+C&amp;rft.au=Connell%2C+CR&amp;rft.au=Heiner%2C+C&amp;rft.au=Kent%2C+SB&amp;rft.au=Hood%2C+LE&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-42"><span class="mw-cite-backlink"><b><a href="#cite_ref-42">^</a></b></span> <span class="reference-text"><cite class="citation journal">Prober JM, Trainor GL, Dam RJ, Hobbs FW, Robertson CW, Zagursky RJ, Cocuzza AJ, Jensen MA, Baumeister K (16 October 1987). "A system for rapid DNA sequencing with fluorescent chain-terminating dideoxynucleotides". <i>Science</i>. <b>238</b> (4825): 336–41. <a href="/wiki/Bibcode" title="Bibcode">Bibcode</a>:<a rel="nofollow" class="external text" href="https://ui.adsabs.harvard.edu/abs/1987Sci...238..336P">1987Sci...238..336P</a>. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1126%2Fscience.2443975">10.1126/science.2443975</a>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/2443975">2443975</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Science&amp;rft.atitle=A+system+for+rapid+DNA+sequencing+with+fluorescent+chain-terminating+dideoxynucleotides&amp;rft.volume=238&amp;rft.issue=4825&amp;rft.pages=336-41&amp;rft.date=1987-10-16&amp;rft_id=info%3Apmid%2F2443975&amp;rft_id=info%3Adoi%2F10.1126%2Fscience.2443975&amp;rft_id=info%3Abibcode%2F1987Sci...238..336P&amp;rft.aulast=Prober&amp;rft.aufirst=JM&amp;rft.au=Trainor%2C+GL&amp;rft.au=Dam%2C+RJ&amp;rft.au=Hobbs%2C+FW&amp;rft.au=Robertson%2C+CW&amp;rft.au=Zagursky%2C+RJ&amp;rft.au=Cocuzza%2C+AJ&amp;rft.au=Jensen%2C+MA&amp;rft.au=Baumeister%2C+K&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-pmid2047873-43"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid2047873_43-0">^</a></b></span> <span class="reference-text"><cite class="citation journal">Adams MD, Kelley JM, Gocayne JD, Dubnick M, Polymeropoulos MH, Xiao H, Merril CR, Wu A, Olde B, Moreno RF (June 1991). "Complementary DNA sequencing: expressed sequence tags and human genome project". <i>Science</i>. <b>252</b> (5013): 1651–56. <a href="/wiki/Bibcode" title="Bibcode">Bibcode</a>:<a rel="nofollow" class="external text" href="https://ui.adsabs.harvard.edu/abs/1991Sci...252.1651A">1991Sci...252.1651A</a>. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1126%2Fscience.2047873">10.1126/science.2047873</a>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/2047873">2047873</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Science&amp;rft.atitle=Complementary+DNA+sequencing%3A+expressed+sequence+tags+and+human+genome+project&amp;rft.volume=252&amp;rft.issue=5013&amp;rft.pages=1651-56&amp;rft.date=1991-06&amp;rft_id=info%3Apmid%2F2047873&amp;rft_id=info%3Adoi%2F10.1126%2Fscience.2047873&amp;rft_id=info%3Abibcode%2F1991Sci...252.1651A&amp;rft.aulast=Adams&amp;rft.aufirst=MD&amp;rft.au=Kelley%2C+JM&amp;rft.au=Gocayne%2C+JD&amp;rft.au=Dubnick%2C+M&amp;rft.au=Polymeropoulos%2C+MH&amp;rft.au=Xiao%2C+H&amp;rft.au=Merril%2C+CR&amp;rft.au=Wu%2C+A&amp;rft.au=Olde%2C+B&amp;rft.au=Moreno%2C+RF&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-44"><span class="mw-cite-backlink"><b><a href="#cite_ref-44">^</a></b></span> <span class="reference-text"><cite class="citation journal">Fleischmann RD, Adams MD, White O, Clayton RA, Kirkness EF, Kerlavage AR, Bult CJ, Tomb JF, Dougherty BA, Merrick JM (July 1995). "Whole-genome random sequencing and assembly of <i>Haemophilus influenzae Rd</i>". <i>Science</i>. <b>269</b> (5223): 496–512. <a href="/wiki/Bibcode" title="Bibcode">Bibcode</a>:<a rel="nofollow" class="external text" href="https://ui.adsabs.harvard.edu/abs/1995Sci...269..496F">1995Sci...269..496F</a>. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1126%2Fscience.7542800">10.1126/science.7542800</a>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/7542800">7542800</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Science&amp;rft.atitle=Whole-genome+random+sequencing+and+assembly+of+Haemophilus+influenzae+Rd&amp;rft.volume=269&amp;rft.issue=5223&amp;rft.pages=496-512&amp;rft.date=1995-07&amp;rft_id=info%3Apmid%2F7542800&amp;rft_id=info%3Adoi%2F10.1126%2Fscience.7542800&amp;rft_id=info%3Abibcode%2F1995Sci...269..496F&amp;rft.aulast=Fleischmann&amp;rft.aufirst=RD&amp;rft.au=Adams%2C+MD&amp;rft.au=White%2C+O&amp;rft.au=Clayton%2C+RA&amp;rft.au=Kirkness%2C+EF&amp;rft.au=Kerlavage%2C+AR&amp;rft.au=Bult%2C+CJ&amp;rft.au=Tomb%2C+JF&amp;rft.au=Dougherty%2C+BA&amp;rft.au=Merrick%2C+JM&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-Lander_2001-45"><span class="mw-cite-backlink"><b><a href="#cite_ref-Lander_2001_45-0">^</a></b></span> <span class="reference-text"><cite class="citation journal">Lander ES, Linton LM, Birren B, Nusbaum C, Zody MC,  et al. (February 2001). <a rel="nofollow" class="external text" href="https://deepblue.lib.umich.edu/bitstream/2027.42/62798/1/409860a0.pdf">"Initial sequencing and analysis of the human genome"</a> <span class="cs1-format">(PDF)</span>. <i>Nature</i>. <b>409</b> (6822): 860–921. <a href="/wiki/Bibcode" title="Bibcode">Bibcode</a>:<a rel="nofollow" class="external text" href="https://ui.adsabs.harvard.edu/abs/2001Natur.409..860L">2001Natur.409..860L</a>. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1038%2F35057062">10.1038/35057062</a>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/11237011">11237011</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Nature&amp;rft.atitle=Initial+sequencing+and+analysis+of+the+human+genome&amp;rft.volume=409&amp;rft.issue=6822&amp;rft.pages=860-921&amp;rft.date=2001-02&amp;rft_id=info%3Apmid%2F11237011&amp;rft_id=info%3Adoi%2F10.1038%2F35057062&amp;rft_id=info%3Abibcode%2F2001Natur.409..860L&amp;rft.aulast=Lander&amp;rft.aufirst=ES&amp;rft.au=Linton%2C+LM&amp;rft.au=Birren%2C+B&amp;rft.au=Nusbaum%2C+C&amp;rft.au=Zody%2C+MC&amp;rft_id=https%3A%2F%2Fdeepblue.lib.umich.edu%2Fbitstream%2F2027.42%2F62798%2F1%2F409860a0.pdf&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-Venter_2001-46"><span class="mw-cite-backlink"><b><a href="#cite_ref-Venter_2001_46-0">^</a></b></span> <span class="reference-text"><cite class="citation journal">Venter JC, Adams MD,  et al. (February 2001). "The sequence of the human genome". <i>Science</i>. <b>291</b> (5507): 1304–51. <a href="/wiki/Bibcode" title="Bibcode">Bibcode</a>:<a rel="nofollow" class="external text" href="https://ui.adsabs.harvard.edu/abs/2001Sci...291.1304V">2001Sci...291.1304V</a>. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1126%2Fscience.1058040">10.1126/science.1058040</a>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/11181995">11181995</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Science&amp;rft.atitle=The+sequence+of+the+human+genome&amp;rft.volume=291&amp;rft.issue=5507&amp;rft.pages=1304-51&amp;rft.date=2001-02&amp;rft_id=info%3Apmid%2F11181995&amp;rft_id=info%3Adoi%2F10.1126%2Fscience.1058040&amp;rft_id=info%3Abibcode%2F2001Sci...291.1304V&amp;rft.aulast=Venter&amp;rft.aufirst=JC&amp;rft.au=Adams%2C+MD&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-TsienPatent-47"><span class="mw-cite-backlink"><b><a href="#cite_ref-TsienPatent_47-0">^</a></b></span> <span class="reference-text"><cite class="citation web"><a rel="nofollow" class="external text" href="http://worldwide.espacenet.com/publicationDetails/biblio?FT=D&amp;date=19910516&amp;DB=EPODOC&amp;locale=en_EP&amp;CC=WO&amp;NR=9106678A1&amp;KC=A1&amp;ND=4">"Espacenet – Bibliographic data"</a>. <i>worldwide.espacenet.com</i>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=unknown&amp;rft.jtitle=worldwide.espacenet.com&amp;rft.atitle=Espacenet+%E2%80%93+Bibliographic+data&amp;rft_id=http%3A%2F%2Fworldwide.espacenet.com%2FpublicationDetails%2Fbiblio%3FFT%3DD%26date%3D19910516%26DB%3DEPODOC%26locale%3Den_EP%26CC%3DWO%26NR%3D9106678A1%26KC%3DA1%26ND%3D4&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-Ronaghi-48"><span class="mw-cite-backlink"><b><a href="#cite_ref-Ronaghi_48-0">^</a></b></span> <span class="reference-text"><cite class="citation journal">Ronaghi M, Karamohamed S, Pettersson B, Uhlén M, Nyrén P (1996). "Real-time DNA sequencing using detection of pyrophosphate release". <i>Analytical Biochemistry</i>. <b>242</b> (1): 84–89. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1006%2Fabio.1996.0432">10.1006/abio.1996.0432</a>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/8923969">8923969</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Analytical+Biochemistry&amp;rft.atitle=Real-time+DNA+sequencing+using+detection+of+pyrophosphate+release&amp;rft.volume=242&amp;rft.issue=1&amp;rft.pages=84-89&amp;rft.date=1996&amp;rft_id=info%3Adoi%2F10.1006%2Fabio.1996.0432&amp;rft_id=info%3Apmid%2F8923969&amp;rft.aulast=Ronaghi&amp;rft.aufirst=M&amp;rft.au=Karamohamed%2C+S&amp;rft.au=Pettersson%2C+B&amp;rft.au=Uhl%C3%A9n%2C+M&amp;rft.au=Nyr%C3%A9n%2C+P&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-DNA_colony_patents-49"><span class="mw-cite-backlink">^ <a href="#cite_ref-DNA_colony_patents_49-0"><sup><i><b>a</b></i></sup></a> <a href="#cite_ref-DNA_colony_patents_49-1"><sup><i><b>b</b></i></sup></a></span> <span class="reference-text"><cite class="citation web">Kawashima, Eric H.; Laurent Farinelli; Pascal Mayer (12 May 2005). <a rel="nofollow" class="external text" href="https://archive.is/20130222020134/http://www.patentlens.net/patentlens/patent/WO_1998_044151_A1/en/">"Patent: Method of nucleic acid amplification"</a>. Archived from <a rel="nofollow" class="external text" href="http://www.patentlens.net/patentlens/patent/WO_1998_044151_A1/en/">the original</a> on 22 February 2013<span class="reference-accessdate">. Retrieved <span class="nowrap">22 December</span> 2012</span>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook&amp;rft.genre=unknown&amp;rft.btitle=Patent%3A+Method+of+nucleic+acid+amplification&amp;rft.date=2005-05-12&amp;rft.aulast=Kawashima&amp;rft.aufirst=Eric+H.&amp;rft.au=Laurent+Farinelli&amp;rft.au=Pascal+Mayer&amp;rft_id=http%3A%2F%2Fwww.patentlens.net%2Fpatentlens%2Fpatent%2FWO_1998_044151_A1%2Fen%2F&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-50"><span class="mw-cite-backlink"><b><a href="#cite_ref-50">^</a></b></span> <span class="reference-text"><cite class="citation journal">Ewing B, Green P (March 1998). "Base-calling of automated sequencer traces using phred. II. Error probabilities". <i>Genome Res</i>. <b>8</b> (3): 186–94. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1101%2Fgr.8.3.186">10.1101/gr.8.3.186</a>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/9521922">9521922</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Genome+Res.&amp;rft.atitle=Base-calling+of+automated+sequencer+traces+using+phred.+II.+Error+probabilities&amp;rft.volume=8&amp;rft.issue=3&amp;rft.pages=186-94&amp;rft.date=1998-03&amp;rft_id=info%3Adoi%2F10.1101%2Fgr.8.3.186&amp;rft_id=info%3Apmid%2F9521922&amp;rft.aulast=Ewing&amp;rft.aufirst=B&amp;rft.au=Green%2C+P&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-51"><span class="mw-cite-backlink"><b><a href="#cite_ref-51">^</a></b></span> <span class="reference-text"><cite class="citation web"><a rel="nofollow" class="external text" href="https://www.illumina.com/documents/products/technotes/technote_Q-Scores.pdf">"Quality Scores for Next-Generation Sequencing"</a> <span class="cs1-format">(PDF)</span>. <i>Illumina</i>. 31 October 2011<span class="reference-accessdate">. Retrieved <span class="nowrap">8 May</span> 2018</span>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=unknown&amp;rft.jtitle=Illumina&amp;rft.atitle=Quality+Scores+for+Next-Generation+Sequencing&amp;rft.date=2011-10-31&amp;rft_id=https%3A%2F%2Fwww.illumina.com%2Fdocuments%2Fproducts%2Ftechnotes%2Ftechnote_Q-Scores.pdf&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-Brenner_2000-52"><span class="mw-cite-backlink">^ <a href="#cite_ref-Brenner_2000_52-0"><sup><i><b>a</b></i></sup></a> <a href="#cite_ref-Brenner_2000_52-1"><sup><i><b>b</b></i></sup></a></span> <span class="reference-text"><cite class="citation journal">Brenner S, Johnson M, Bridgham J, Golda G, Lloyd DH, Johnson D, Luo S, McCurdy S, Foy M, Ewan M, Roth R, George D, Eletr S, Albrecht G, Vermaas E, Williams SR, Moon K, Burcham T, Pallas M, DuBridge RB, Kirchner J, Fearon K, Mao J, Corcoran K (2000). "Gene expression analysis by massively parallel signature sequencing (MPSS) on microbead arrays". <i>Nature Biotechnology</i>. <b>18</b> (6): 630–34. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1038%2F76469">10.1038/76469</a>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/10835600">10835600</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Nature+Biotechnology&amp;rft.atitle=Gene+expression+analysis+by+massively+parallel+signature+sequencing+%28MPSS%29+on+microbead+arrays&amp;rft.volume=18&amp;rft.issue=6&amp;rft.pages=630-34&amp;rft.date=2000&amp;rft_id=info%3Adoi%2F10.1038%2F76469&amp;rft_id=info%3Apmid%2F10835600&amp;rft.aulast=Brenner&amp;rft.aufirst=S&amp;rft.au=Johnson%2C+M&amp;rft.au=Bridgham%2C+J&amp;rft.au=Golda%2C+G&amp;rft.au=Lloyd%2C+DH&amp;rft.au=Johnson%2C+D&amp;rft.au=Luo%2C+S&amp;rft.au=McCurdy%2C+S&amp;rft.au=Foy%2C+M&amp;rft.au=Ewan%2C+M&amp;rft.au=Roth%2C+R&amp;rft.au=George%2C+D&amp;rft.au=Eletr%2C+S&amp;rft.au=Albrecht%2C+G&amp;rft.au=Vermaas%2C+E&amp;rft.au=Williams%2C+SR&amp;rft.au=Moon%2C+K&amp;rft.au=Burcham%2C+T&amp;rft.au=Pallas%2C+M&amp;rft.au=DuBridge%2C+RB&amp;rft.au=Kirchner%2C+J&amp;rft.au=Fearon%2C+K&amp;rft.au=Mao%2C+J&amp;rft.au=Corcoran%2C+K&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-Sanger75-53"><span class="mw-cite-backlink"><b><a href="#cite_ref-Sanger75_53-0">^</a></b></span> <span class="reference-text"><cite class="citation journal">Sanger F, Coulson AR (May 1975). "A rapid method for determining sequences in DNA by primed synthesis with DNA polymerase". <i>J. Mol. Biol</i>. <b>94</b> (3): 441–48. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1016%2F0022-2836%2875%2990213-2">10.1016/0022-2836(75)90213-2</a>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/1100841">1100841</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=J.+Mol.+Biol.&amp;rft.atitle=A+rapid+method+for+determining+sequences+in+DNA+by+primed+synthesis+with+DNA+polymerase&amp;rft.volume=94&amp;rft.issue=3&amp;rft.pages=441-48&amp;rft.date=1975-05&amp;rft_id=info%3Adoi%2F10.1016%2F0022-2836%2875%2990213-2&amp;rft_id=info%3Apmid%2F1100841&amp;rft.aulast=Sanger&amp;rft.aufirst=F&amp;rft.au=Coulson%2C+AR&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-54"><span class="mw-cite-backlink"><b><a href="#cite_ref-54">^</a></b></span> <span class="reference-text"><cite class="citation web">Wetterstrand, Kris. <a rel="nofollow" class="external text" href="https://www.genome.gov/sequencingcosts">"DNA Sequencing Costs: Data from the NHGRI Genome Sequencing Program (GSP)"</a>. <a href="/wiki/National_Human_Genome_Research_Institute" title="National Human Genome Research Institute">National Human Genome Research Institute</a><span class="reference-accessdate">. Retrieved <span class="nowrap">30 May</span> 2013</span>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook&amp;rft.genre=unknown&amp;rft.btitle=DNA+Sequencing+Costs%3A+Data+from+the+NHGRI+Genome+Sequencing+Program+%28GSP%29&amp;rft.pub=National+Human+Genome+Research+Institute&amp;rft.aulast=Wetterstrand&amp;rft.aufirst=Kris&amp;rft_id=https%3A%2F%2Fwww.genome.gov%2Fsequencingcosts&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-pmid23147856-55"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid23147856_55-0">^</a></b></span> <span class="reference-text"><cite class="citation journal">Quail MA, Gu Y, Swerdlow H, Mayho M (2012). "Evaluation and optimisation of preparative semi-automated electrophoresis systems for Illumina library preparation". <i>Electrophoresis</i>. <b>33</b> (23): 3521–28. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1002%2Felps.201200128">10.1002/elps.201200128</a>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/23147856">23147856</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Electrophoresis&amp;rft.atitle=Evaluation+and+optimisation+of+preparative+semi-automated+electrophoresis+systems+for+Illumina+library+preparation&amp;rft.volume=33&amp;rft.issue=23&amp;rft.pages=3521-28&amp;rft.date=2012&amp;rft_id=info%3Adoi%2F10.1002%2Felps.201200128&amp;rft_id=info%3Apmid%2F23147856&amp;rft.aulast=Quail&amp;rft.aufirst=MA&amp;rft.au=Gu%2C+Y&amp;rft.au=Swerdlow%2C+H&amp;rft.au=Mayho%2C+M&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-pmid22713159-56"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid22713159_56-0">^</a></b></span> <span class="reference-text"><cite class="citation journal">Duhaime MB, Deng L, Poulos BT, Sullivan MB (2012). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3466414">"Towards quantitative metagenomics of wild viruses and other ultra-low concentration DNA samples: a rigorous assessment and optimization of the linker amplification method"</a>. <i>Environ. Microbiol</i>. <b>14</b> (9): 2526–37. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1111%2Fj.1462-2920.2012.02791.x">10.1111/j.1462-2920.2012.02791.x</a>. <a href="/wiki/PubMed_Central" title="PubMed Central">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3466414">3466414</a></span>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/22713159">22713159</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Environ.+Microbiol.&amp;rft.atitle=Towards+quantitative+metagenomics+of+wild+viruses+and+other+ultra-low+concentration+DNA+samples%3A+a+rigorous+assessment+and+optimization+of+the+linker+amplification+method&amp;rft.volume=14&amp;rft.issue=9&amp;rft.pages=2526-37&amp;rft.date=2012&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3466414&amp;rft_id=info%3Apmid%2F22713159&amp;rft_id=info%3Adoi%2F10.1111%2Fj.1462-2920.2012.02791.x&amp;rft.aulast=Duhaime&amp;rft.aufirst=MB&amp;rft.au=Deng%2C+L&amp;rft.au=Poulos%2C+BT&amp;rft.au=Sullivan%2C+MB&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3466414&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-pmid22675423-57"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid22675423_57-0">^</a></b></span> <span class="reference-text"><cite class="citation journal">Peterson BK, Weber JN, Kay EH, Fisher HS, Hoekstra HE (2012). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3365034">"Double digest RADseq: an inexpensive method for de novo SNP discovery and genotyping in model and non-model species"</a>. <i>PLoS ONE</i>. <b>7</b> (5): e37135. <a href="/wiki/Bibcode" title="Bibcode">Bibcode</a>:<a rel="nofollow" class="external text" href="https://ui.adsabs.harvard.edu/abs/2012PLoSO...737135P">2012PLoSO...737135P</a>. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1371%2Fjournal.pone.0037135">10.1371/journal.pone.0037135</a>. <a href="/wiki/PubMed_Central" title="PubMed Central">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3365034">3365034</a></span>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/22675423">22675423</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=PLoS+ONE&amp;rft.atitle=Double+digest+RADseq%3A+an+inexpensive+method+for+de+novo+SNP+discovery+and+genotyping+in+model+and+non-model+species&amp;rft.volume=7&amp;rft.issue=5&amp;rft.pages=e37135&amp;rft.date=2012&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3365034&amp;rft_id=info%3Apmid%2F22675423&amp;rft_id=info%3Adoi%2F10.1371%2Fjournal.pone.0037135&amp;rft_id=info%3Abibcode%2F2012PLoSO...737135P&amp;rft.aulast=Peterson&amp;rft.aufirst=BK&amp;rft.au=Weber%2C+JN&amp;rft.au=Kay%2C+EH&amp;rft.au=Fisher%2C+HS&amp;rft.au=Hoekstra%2C+HE&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3365034&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-Williams2006ePCR-58"><span class="mw-cite-backlink"><b><a href="#cite_ref-Williams2006ePCR_58-0">^</a></b></span> <span class="reference-text"><cite class="citation journal">Williams R, Peisajovich SG, Miller OJ, Magdassi S, Tawfik DS, Griffiths AD (2006). "Amplification of complex gene libraries by emulsion PCR". <i>Nature Methods</i>. <b>3</b> (7): 545–50. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1038%2Fnmeth896">10.1038/nmeth896</a>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/16791213">16791213</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Nature+Methods&amp;rft.atitle=Amplification+of+complex+gene+libraries+by+emulsion+PCR&amp;rft.volume=3&amp;rft.issue=7&amp;rft.pages=545-50&amp;rft.date=2006&amp;rft_id=info%3Adoi%2F10.1038%2Fnmeth896&amp;rft_id=info%3Apmid%2F16791213&amp;rft.aulast=Williams&amp;rft.aufirst=R&amp;rft.au=Peisajovich%2C+SG&amp;rft.au=Miller%2C+OJ&amp;rft.au=Magdassi%2C+S&amp;rft.au=Tawfik%2C+DS&amp;rft.au=Griffiths%2C+AD&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-Margulies_2005-59"><span class="mw-cite-backlink">^ <a href="#cite_ref-Margulies_2005_59-0"><sup><i><b>a</b></i></sup></a> <a href="#cite_ref-Margulies_2005_59-1"><sup><i><b>b</b></i></sup></a></span> <span class="reference-text"><cite class="citation journal">Margulies M, Egholm M,  et al. (September 2005). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC1464427">"Genome Sequencing in Open Microfabricated High Density Picoliter Reactors"</a>. <i>Nature</i>. <b>437</b> (7057): 376–80. <a href="/wiki/Bibcode" title="Bibcode">Bibcode</a>:<a rel="nofollow" class="external text" href="https://ui.adsabs.harvard.edu/abs/2005Natur.437..376M">2005Natur.437..376M</a>. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1038%2Fnature03959">10.1038/nature03959</a>. <a href="/wiki/PubMed_Central" title="PubMed Central">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC1464427">1464427</a></span>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/16056220">16056220</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Nature&amp;rft.atitle=Genome+Sequencing+in+Open+Microfabricated+High+Density+Picoliter+Reactors&amp;rft.volume=437&amp;rft.issue=7057&amp;rft.pages=376-80&amp;rft.date=2005-09&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1464427&amp;rft_id=info%3Apmid%2F16056220&amp;rft_id=info%3Adoi%2F10.1038%2Fnature03959&amp;rft_id=info%3Abibcode%2F2005Natur.437..376M&amp;rft.aulast=Margulies&amp;rft.aufirst=M&amp;rft.au=Egholm%2C+M&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1464427&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-polony_sequencing-60"><span class="mw-cite-backlink"><b><a href="#cite_ref-polony_sequencing_60-0">^</a></b></span> <span class="reference-text"><cite class="citation journal">Shendure J, Porreca GJ, Reppas NB, Lin X, McCutcheon JP, Rosenbaum AM, Wang MD, Zhang K, Mitra RD, Church GM (2005). "Accurate Multiplex Polony Sequencing of an Evolved Bacterial Genome". <i>Science</i>. <b>309</b> (5741): 1728–32. <a href="/wiki/Bibcode" title="Bibcode">Bibcode</a>:<a rel="nofollow" class="external text" href="https://ui.adsabs.harvard.edu/abs/2005Sci...309.1728S">2005Sci...309.1728S</a>. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1126%2Fscience.1117389">10.1126/science.1117389</a>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/16081699">16081699</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Science&amp;rft.atitle=Accurate+Multiplex+Polony+Sequencing+of+an+Evolved+Bacterial+Genome&amp;rft.volume=309&amp;rft.issue=5741&amp;rft.pages=1728-32&amp;rft.date=2005&amp;rft_id=info%3Apmid%2F16081699&amp;rft_id=info%3Adoi%2F10.1126%2Fscience.1117389&amp;rft_id=info%3Abibcode%2F2005Sci...309.1728S&amp;rft.aulast=Shendure&amp;rft.aufirst=J&amp;rft.au=Porreca%2C+GJ&amp;rft.au=Reppas%2C+NB&amp;rft.au=Lin%2C+X&amp;rft.au=McCutcheon%2C+JP&amp;rft.au=Rosenbaum%2C+AM&amp;rft.au=Wang%2C+MD&amp;rft.au=Zhang%2C+K&amp;rft.au=Mitra%2C+RD&amp;rft.au=Church%2C+GM&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-solid_sequencing-61"><span class="mw-cite-backlink"><b><a href="#cite_ref-solid_sequencing_61-0">^</a></b></span> <span class="reference-text"><cite class="citation web"><a rel="nofollow" class="external text" href="https://web.archive.org/web/20080516181322/http://solid.appliedbiosystems.com/">"Applied Biosystems – File Not Found (404 Error)"</a>. 16 May 2008. Archived from <a rel="nofollow" class="external text" href="http://solid.appliedbiosystems.com/">the original</a> on 16 May 2008.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook&amp;rft.genre=unknown&amp;rft.btitle=Applied+Biosystems+%E2%80%93+File+Not+Found+%28404+Error%29&amp;rft.date=2008-05-16&amp;rft_id=http%3A%2F%2Fsolid.appliedbiosystems.com%2F&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-10x-epcr-62"><span class="mw-cite-backlink"><b><a href="#cite_ref-10x-epcr_62-0">^</a></b></span> <span class="reference-text"><cite class="citation journal">Goodwin S, McPherson JD, McCombie WR (May 2016). "Coming of age: ten years of next-generation sequencing technologies". <i>Nature Reviews. Genetics</i>. <b>17</b> (6): 333–51. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1038%2Fnrg.2016.49">10.1038/nrg.2016.49</a>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/27184599">27184599</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Nature+Reviews.+Genetics&amp;rft.atitle=Coming+of+age%3A+ten+years+of+next-generation+sequencing+technologies&amp;rft.volume=17&amp;rft.issue=6&amp;rft.pages=333-51&amp;rft.date=2016-05&amp;rft_id=info%3Adoi%2F10.1038%2Fnrg.2016.49&amp;rft_id=info%3Apmid%2F27184599&amp;rft.aulast=Goodwin&amp;rft.aufirst=S&amp;rft.au=McPherson%2C+JD&amp;rft.au=McCombie%2C+WR&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-63"><span class="mw-cite-backlink"><b><a href="#cite_ref-63">^</a></b></span> <span class="reference-text"><cite class="citation journal">Staden R (11 June 1979). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC327874">"A strategy of DNA sequencing employing computer programs"</a>. <i>Nucleic Acids Research</i>. <b>6</b> (7): 2601–10. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fnar%2F6.7.2601">10.1093/nar/6.7.2601</a>. <a href="/wiki/PubMed_Central" title="PubMed Central">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC327874">327874</a></span>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/461197">461197</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Nucleic+Acids+Research&amp;rft.atitle=A+strategy+of+DNA+sequencing+employing+computer+programs.&amp;rft.volume=6&amp;rft.issue=7&amp;rft.pages=2601-10&amp;rft.date=1979-06-11&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC327874&amp;rft_id=info%3Apmid%2F461197&amp;rft_id=info%3Adoi%2F10.1093%2Fnar%2F6.7.2601&amp;rft.aulast=Staden&amp;rft.aufirst=R&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC327874&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-pmid19900591-65"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid19900591_65-0">^</a></b></span> <span class="reference-text"><cite class="citation journal">de Magalhães JP, Finch CE, Janssens G (2010). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC2878865">"Next-generation sequencing in aging research: emerging applications, problems, pitfalls and possible solutions"</a>. <i><a href="/wiki/Ageing_Research_Reviews" title="Ageing Research Reviews">Ageing Research Reviews</a></i>. <b>9</b> (3): 315–23. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1016%2Fj.arr.2009.10.006">10.1016/j.arr.2009.10.006</a>. <a href="/wiki/PubMed_Central" title="PubMed Central">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC2878865">2878865</a></span>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/19900591">19900591</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Ageing+Research+Reviews&amp;rft.atitle=Next-generation+sequencing+in+aging+research%3A+emerging+applications%2C+problems%2C+pitfalls+and+possible+solutions&amp;rft.volume=9&amp;rft.issue=3&amp;rft.pages=315-23&amp;rft.date=2010&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2878865&amp;rft_id=info%3Apmid%2F19900591&amp;rft_id=info%3Adoi%2F10.1016%2Fj.arr.2009.10.006&amp;rft.aulast=de+Magalh%C3%A3es&amp;rft.aufirst=JP&amp;rft.au=Finch%2C+CE&amp;rft.au=Janssens%2C+G&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2878865&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-pmid23856935-66"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid23856935_66-0">^</a></b></span> <span class="reference-text"><cite class="citation journal">Grada A (August 2013). "Next-generation sequencing: methodology and application". <i>J Invest Dermatol</i>. <b>133</b> (8): e11. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1038%2Fjid.2013.248">10.1038/jid.2013.248</a>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/23856935">23856935</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=J+Invest+Dermatol&amp;rft.atitle=Next-generation+sequencing%3A+methodology+and+application&amp;rft.volume=133&amp;rft.issue=8&amp;rft.pages=e11&amp;rft.date=2013-08&amp;rft_id=info%3Adoi%2F10.1038%2Fjid.2013.248&amp;rft_id=info%3Apmid%2F23856935&amp;rft.aulast=Grada&amp;rft.aufirst=A&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-hall2007-67"><span class="mw-cite-backlink"><b><a href="#cite_ref-hall2007_67-0">^</a></b></span> <span class="reference-text"><cite class="citation journal">Hall N (May 2007). "Advanced sequencing technologies and their wider impact in microbiology". <i><a href="/wiki/J._Exp._Biol." class="mw-redirect" title="J. Exp. Biol.">J. Exp. Biol.</a></i> <b>210</b> (Pt 9): 1518–25. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1242%2Fjeb.001370">10.1242/jeb.001370</a>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/17449817">17449817</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=J.+Exp.+Biol.&amp;rft.atitle=Advanced+sequencing+technologies+and+their+wider+impact+in+microbiology&amp;rft.volume=210&amp;rft.issue=Pt+9&amp;rft.pages=1518-25&amp;rft.date=2007-05&amp;rft_id=info%3Adoi%2F10.1242%2Fjeb.001370&amp;rft_id=info%3Apmid%2F17449817&amp;rft.aulast=Hall&amp;rft.aufirst=N&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/><span style="position:relative; top: -2px;"><a href="/wiki/Open_access" title="open access publication – free to read"><img alt="open access" src="//upload.wikimedia.org/wikipedia/commons/thumb/7/77/Open_Access_logo_PLoS_transparent.svg/9px-Open_Access_logo_PLoS_transparent.svg.png" decoding="async" width="9" height="14" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/7/77/Open_Access_logo_PLoS_transparent.svg/14px-Open_Access_logo_PLoS_transparent.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/7/77/Open_Access_logo_PLoS_transparent.svg/18px-Open_Access_logo_PLoS_transparent.svg.png 2x" data-file-width="640" data-file-height="1000" /></a></span></span>
</li>
<li id="cite_note-church2006-68"><span class="mw-cite-backlink"><b><a href="#cite_ref-church2006_68-0">^</a></b></span> <span class="reference-text"><cite class="citation journal"><a href="/wiki/George_M._Church" class="mw-redirect" title="George M. Church">Church GM</a> (January 2006). "Genomes for all". <i><a href="/wiki/Sci._Am." class="mw-redirect" title="Sci. Am.">Sci. Am.</a></i> <b>294</b> (1): 46–54. <a href="/wiki/Bibcode" title="Bibcode">Bibcode</a>:<a rel="nofollow" class="external text" href="https://ui.adsabs.harvard.edu/abs/2006SciAm.294a..46C">2006SciAm.294a..46C</a>. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1038%2Fscientificamerican0106-46">10.1038/scientificamerican0106-46</a>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/16468433">16468433</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Sci.+Am.&amp;rft.atitle=Genomes+for+all&amp;rft.volume=294&amp;rft.issue=1&amp;rft.pages=46-54&amp;rft.date=2006-01&amp;rft_id=info%3Apmid%2F16468433&amp;rft_id=info%3Adoi%2F10.1038%2Fscientificamerican0106-46&amp;rft_id=info%3Abibcode%2F2006SciAm.294a..46C&amp;rft.aulast=Church&amp;rft.aufirst=GM&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/><span style="font-size:0.95em; font-size:90%; color:#555">(subscription required)</span></span>
</li>
<li id="cite_note-pmid18165802-69"><span class="mw-cite-backlink">^ <a href="#cite_ref-pmid18165802_69-0"><sup><i><b>a</b></i></sup></a> <a href="#cite_ref-pmid18165802_69-1"><sup><i><b>b</b></i></sup></a> <a href="#cite_ref-pmid18165802_69-2"><sup><i><b>c</b></i></sup></a></span> <span class="reference-text"><cite class="citation journal">Schuster SC (January 2008). "Next-generation sequencing transforms today's biology". <i>Nat. Methods</i>. <b>5</b> (1): 16–18. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1038%2Fnmeth1156">10.1038/nmeth1156</a>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/18165802">18165802</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Nat.+Methods&amp;rft.atitle=Next-generation+sequencing+transforms+today%27s+biology&amp;rft.volume=5&amp;rft.issue=1&amp;rft.pages=16-18&amp;rft.date=2008-01&amp;rft_id=info%3Adoi%2F10.1038%2Fnmeth1156&amp;rft_id=info%3Apmid%2F18165802&amp;rft.aulast=Schuster&amp;rft.aufirst=SC&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-kalb1992-70"><span class="mw-cite-backlink"><b><a href="#cite_ref-kalb1992_70-0">^</a></b></span> <span class="reference-text"><cite class="citation book">Kalb, Gilbert; Moxley, Robert (1992). <i>Massively Parallel, Optical, and Neural Computing in the United States</i>. <a href="/wiki/IOS_Press" title="IOS Press">IOS Press</a>. <a href="/wiki/International_Standard_Book_Number" title="International Standard Book Number">ISBN</a>&#160;<a href="/wiki/Special:BookSources/978-90-5199-097-3" title="Special:BookSources/978-90-5199-097-3"><bdi>978-90-5199-097-3</bdi></a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook&amp;rft.genre=book&amp;rft.btitle=Massively+Parallel%2C+Optical%2C+and+Neural+Computing+in+the+United+States&amp;rft.pub=IOS+Press&amp;rft.date=1992&amp;rft.isbn=978-90-5199-097-3&amp;rft.aulast=Kalb&amp;rft.aufirst=Gilbert&amp;rft.au=Moxley%2C+Robert&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/><sup class="noprint Inline-Template" style="white-space:nowrap;">&#91;<i><a href="/wiki/Wikipedia:Citing_sources" title="Wikipedia:Citing sources"><span title="This citation requires a reference to the specific page or range of pages in which the material appears. (June 2013)">page&#160;needed</span></a></i>&#93;</sup></span>
</li>
<li id="cite_note-tenBosch2008-71"><span class="mw-cite-backlink"><b><a href="#cite_ref-tenBosch2008_71-0">^</a></b></span> <span class="reference-text"><cite class="citation journal">ten Bosch JR, Grody WW (2008). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC2570630">"Keeping Up with the Next Generation"</a>. <i>The Journal of Molecular Diagnostics</i>. <b>10</b> (6): 484–92. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.2353%2Fjmoldx.2008.080027">10.2353/jmoldx.2008.080027</a>. <a href="/wiki/PubMed_Central" title="PubMed Central">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC2570630">2570630</a></span>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/18832462">18832462</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=The+Journal+of+Molecular+Diagnostics&amp;rft.atitle=Keeping+Up+with+the+Next+Generation&amp;rft.volume=10&amp;rft.issue=6&amp;rft.pages=484-92&amp;rft.date=2008&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2570630&amp;rft_id=info%3Apmid%2F18832462&amp;rft_id=info%3Adoi%2F10.2353%2Fjmoldx.2008.080027&amp;rft.aulast=ten+Bosch&amp;rft.aufirst=JR&amp;rft.au=Grody%2C+WW&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2570630&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/><span style="position:relative; top: -2px;"><a href="/wiki/Open_access" title="open access publication – free to read"><img alt="open access" src="//upload.wikimedia.org/wikipedia/commons/thumb/7/77/Open_Access_logo_PLoS_transparent.svg/9px-Open_Access_logo_PLoS_transparent.svg.png" decoding="async" width="9" height="14" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/7/77/Open_Access_logo_PLoS_transparent.svg/14px-Open_Access_logo_PLoS_transparent.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/7/77/Open_Access_logo_PLoS_transparent.svg/18px-Open_Access_logo_PLoS_transparent.svg.png 2x" data-file-width="640" data-file-height="1000" /></a></span></span>
</li>
<li id="cite_note-Tucker2009-72"><span class="mw-cite-backlink"><b><a href="#cite_ref-Tucker2009_72-0">^</a></b></span> <span class="reference-text"><cite class="citation journal">Tucker T, Marra M, Friedman JM (2009). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC2725244">"Massively Parallel Sequencing: The Next Big Thing in Genetic Medicine"</a>. <i>The American Journal of Human Genetics</i>. <b>85</b> (2): 142–54. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1016%2Fj.ajhg.2009.06.022">10.1016/j.ajhg.2009.06.022</a>. <a href="/wiki/PubMed_Central" title="PubMed Central">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC2725244">2725244</a></span>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/19679224">19679224</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=The+American+Journal+of+Human+Genetics&amp;rft.atitle=Massively+Parallel+Sequencing%3A+The+Next+Big+Thing+in+Genetic+Medicine&amp;rft.volume=85&amp;rft.issue=2&amp;rft.pages=142-54&amp;rft.date=2009&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2725244&amp;rft_id=info%3Apmid%2F19679224&amp;rft_id=info%3Adoi%2F10.1016%2Fj.ajhg.2009.06.022&amp;rft.aulast=Tucker&amp;rft.aufirst=T&amp;rft.au=Marra%2C+M&amp;rft.au=Friedman%2C+JM&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2725244&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/><span style="position:relative; top: -2px;"><a href="/wiki/Open_access" title="open access publication – free to read"><img alt="open access" src="//upload.wikimedia.org/wikipedia/commons/thumb/7/77/Open_Access_logo_PLoS_transparent.svg/9px-Open_Access_logo_PLoS_transparent.svg.png" decoding="async" width="9" height="14" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/7/77/Open_Access_logo_PLoS_transparent.svg/14px-Open_Access_logo_PLoS_transparent.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/7/77/Open_Access_logo_PLoS_transparent.svg/18px-Open_Access_logo_PLoS_transparent.svg.png 2x" data-file-width="640" data-file-height="1000" /></a></span></span>
</li>
<li id="cite_note-:2-73"><span class="mw-cite-backlink">^ <a href="#cite_ref-:2_73-0"><sup><i><b>a</b></i></sup></a> <a href="#cite_ref-:2_73-1"><sup><i><b>b</b></i></sup></a></span> <span class="reference-text"><cite class="citation journal">Straiton J, Free T, Sawyer A, Martin J (February 2019). "From Sanger sequencing to genome databases and beyond". <i>BioTechniques</i>. Future Science. <b>66</b> (2): 60–63. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.2144%2Fbtn-2019-0011">10.2144/btn-2019-0011</a>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/30744413">30744413</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=BioTechniques&amp;rft.atitle=From+Sanger+sequencing+to+genome+databases+and+beyond&amp;rft.volume=66&amp;rft.issue=2&amp;rft.pages=60-63&amp;rft.date=2019-02&amp;rft_id=info%3Adoi%2F10.2144%2Fbtn-2019-0011&amp;rft_id=info%3Apmid%2F30744413&amp;rft.aulast=Straiton&amp;rft.aufirst=J&amp;rft.au=Free%2C+T&amp;rft.au=Sawyer%2C+A&amp;rft.au=Martin%2C+J&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-quail2012-74"><span class="mw-cite-backlink"><b><a href="#cite_ref-quail2012_74-0">^</a></b></span> <span class="reference-text"><cite class="citation journal">Quail MA, Smith M, Coupland P, Otto TD, Harris SR, Connor TR, Bertoni A, Swerdlow HP, Gu Y (1 January 2012). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3431227">"A tale of three next generation sequencing platforms: comparison of Ion Torrent, Pacific Biosciences and illumina MiSeq sequencers"</a>. <i><a href="/wiki/BMC_Genomics" title="BMC Genomics">BMC Genomics</a></i>. <b>13</b> (1): 341. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1186%2F1471-2164-13-341">10.1186/1471-2164-13-341</a>. <a href="/wiki/PubMed_Central" title="PubMed Central">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3431227">3431227</a></span>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/22827831">22827831</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=BMC+Genomics&amp;rft.atitle=A+tale+of+three+next+generation+sequencing+platforms%3A+comparison+of+Ion+Torrent%2C+Pacific+Biosciences+and+illumina+MiSeq+sequencers&amp;rft.volume=13&amp;rft.issue=1&amp;rft.pages=341&amp;rft.date=2012-01-01&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3431227&amp;rft_id=info%3Apmid%2F22827831&amp;rft_id=info%3Adoi%2F10.1186%2F1471-2164-13-341&amp;rft.aulast=Quail&amp;rft.aufirst=MA&amp;rft.au=Smith%2C+M&amp;rft.au=Coupland%2C+P&amp;rft.au=Otto%2C+TD&amp;rft.au=Harris%2C+SR&amp;rft.au=Connor%2C+TR&amp;rft.au=Bertoni%2C+A&amp;rft.au=Swerdlow%2C+HP&amp;rft.au=Gu%2C+Y&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3431227&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/><span style="position:relative; top: -2px;"><a href="/wiki/Open_access" title="open access publication – free to read"><img alt="open access" src="//upload.wikimedia.org/wikipedia/commons/thumb/7/77/Open_Access_logo_PLoS_transparent.svg/9px-Open_Access_logo_PLoS_transparent.svg.png" decoding="async" width="9" height="14" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/7/77/Open_Access_logo_PLoS_transparent.svg/14px-Open_Access_logo_PLoS_transparent.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/7/77/Open_Access_logo_PLoS_transparent.svg/18px-Open_Access_logo_PLoS_transparent.svg.png 2x" data-file-width="640" data-file-height="1000" /></a></span></span>
</li>
<li id="cite_note-lin2012-75"><span class="mw-cite-backlink"><b><a href="#cite_ref-lin2012_75-0">^</a></b></span> <span class="reference-text"><cite class="citation journal">Liu L, Li Y, Li S, Hu N, He Y, Pong R, Lin D, Lu L, Law M (1 January 2012). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3398667">"Comparison of Next-Generation Sequencing Systems"</a>. <i>Journal of Biomedicine and Biotechnology</i>. <b>2012</b>: 251364. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1155%2F2012%2F251364">10.1155/2012/251364</a>. <a href="/wiki/PubMed_Central" title="PubMed Central">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3398667">3398667</a></span>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/22829749">22829749</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Journal+of+Biomedicine+and+Biotechnology&amp;rft.atitle=Comparison+of+Next-Generation+Sequencing+Systems&amp;rft.volume=2012&amp;rft.pages=251364&amp;rft.date=2012-01-01&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3398667&amp;rft_id=info%3Apmid%2F22829749&amp;rft_id=info%3Adoi%2F10.1155%2F2012%2F251364&amp;rft.aulast=Liu&amp;rft.aufirst=L&amp;rft.au=Li%2C+Y&amp;rft.au=Li%2C+S&amp;rft.au=Hu%2C+N&amp;rft.au=He%2C+Y&amp;rft.au=Pong%2C+R&amp;rft.au=Lin%2C+D&amp;rft.au=Lu%2C+L&amp;rft.au=Law%2C+M&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3398667&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/><span style="position:relative; top: -2px;"><a href="/wiki/Open_access" title="open access publication – free to read"><img alt="open access" src="//upload.wikimedia.org/wikipedia/commons/thumb/7/77/Open_Access_logo_PLoS_transparent.svg/9px-Open_Access_logo_PLoS_transparent.svg.png" decoding="async" width="9" height="14" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/7/77/Open_Access_logo_PLoS_transparent.svg/14px-Open_Access_logo_PLoS_transparent.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/7/77/Open_Access_logo_PLoS_transparent.svg/18px-Open_Access_logo_PLoS_transparent.svg.png 2x" data-file-width="640" data-file-height="1000" /></a></span></span>
</li>
<li id="cite_note-sequel21-76"><span class="mw-cite-backlink">^ <a href="#cite_ref-sequel21_76-0"><sup><i><b>a</b></i></sup></a> <a href="#cite_ref-sequel21_76-1"><sup><i><b>b</b></i></sup></a> <a href="#cite_ref-sequel21_76-2"><sup><i><b>c</b></i></sup></a></span> <span class="reference-text"><cite class="citation web"><a rel="nofollow" class="external text" href="https://www.pacb.com/blog/new-software-polymerase-sequel-system-boost-throughput-affordability/">"New Software, Polymerase for Sequel System Boost Throughput and Affordability – PacBio"</a>. 7 March 2018.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook&amp;rft.genre=unknown&amp;rft.btitle=New+Software%2C+Polymerase+for+Sequel+System+Boost+Throughput+and+Affordability+%E2%80%93+PacBio&amp;rft.date=2018-03-07&amp;rft_id=https%3A%2F%2Fwww.pacb.com%2Fblog%2Fnew-software-polymerase-sequel-system-boost-throughput-affordability%2F&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-autogenerated1-77"><span class="mw-cite-backlink"><b><a href="#cite_ref-autogenerated1_77-0">^</a></b></span> <span class="reference-text"><cite class="citation web"><a rel="nofollow" class="external text" href="http://www.genomeweb.com/sequencing/after-year-testing-two-early-pacbio-customers-expect-more-routine-use-rs-sequenc">"After a Year of Testing, Two Early PacBio Customers Expect More Routine Use of RS Sequencer in 2012"</a>. GenomeWeb. 10 January 2012.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook&amp;rft.genre=unknown&amp;rft.btitle=After+a+Year+of+Testing%2C+Two+Early+PacBio+Customers+Expect+More+Routine+Use+of+RS+Sequencer+in+2012&amp;rft.pub=GenomeWeb&amp;rft.date=2012-01-10&amp;rft_id=http%3A%2F%2Fwww.genomeweb.com%2Fsequencing%2Fafter-year-testing-two-early-pacbio-customers-expect-more-routine-use-rs-sequenc&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/><span style="font-size:0.95em; font-size:90%; color:#555">(<a href="/wiki/Wikipedia:Verifiability#Access_to_sources" title="Wikipedia:Verifiability">registration required</a>)</span></span>
</li>
<li id="cite_note-78"><span class="mw-cite-backlink"><b><a href="#cite_ref-78">^</a></b></span> <span class="reference-text"><cite class="citation web">Inc., Pacific Biosciences (2013). <a rel="nofollow" class="external text" href="http://globenewswire.com/news-release/2013/10/03/577891/10051072/en/Pacific-Biosciences-Introduces-New-Chemistry-With-Longer-Read-Lengths-to-Detect-Novel-Features-in-DNA-Sequence-and-Advance-Genome-Studies-of-Large-Organisms.html">"Pacific Biosciences Introduces New Chemistry With Longer Read Lengths to Detect Novel Features in DNA Sequence and Advance Genome Studies of Large Organisms"</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook&amp;rft.genre=unknown&amp;rft.btitle=Pacific+Biosciences+Introduces+New+Chemistry+With+Longer+Read+Lengths+to+Detect+Novel+Features+in+DNA+Sequence+and+Advance+Genome+Studies+of+Large+Organisms&amp;rft.date=2013&amp;rft.aulast=Inc.&amp;rft.aufirst=Pacific+Biosciences&amp;rft_id=http%3A%2F%2Fglobenewswire.com%2Fnews-release%2F2013%2F10%2F03%2F577891%2F10051072%2Fen%2FPacific-Biosciences-Introduces-New-Chemistry-With-Longer-Read-Lengths-to-Detect-Novel-Features-in-DNA-Sequence-and-Advance-Genome-Studies-of-Large-Organisms.html&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-pmid23644548-79"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid23644548_79-0">^</a></b></span> <span class="reference-text"><cite class="citation journal">Chin CS, Alexander DH, Marks P, Klammer AA, Drake J, Heiner C, Clum A, Copeland A, Huddleston J, Eichler EE, Turner SW, Korlach J (2013). "Nonhybrid, finished microbial genome assemblies from long-read SMRT sequencing data". <i>Nat. Methods</i>. <b>10</b> (6): 563–69. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1038%2Fnmeth.2474">10.1038/nmeth.2474</a>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/23644548">23644548</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Nat.+Methods&amp;rft.atitle=Nonhybrid%2C+finished+microbial+genome+assemblies+from+long-read+SMRT+sequencing+data&amp;rft.volume=10&amp;rft.issue=6&amp;rft.pages=563-69&amp;rft.date=2013&amp;rft_id=info%3Adoi%2F10.1038%2Fnmeth.2474&amp;rft_id=info%3Apmid%2F23644548&amp;rft.aulast=Chin&amp;rft.aufirst=CS&amp;rft.au=Alexander%2C+DH&amp;rft.au=Marks%2C+P&amp;rft.au=Klammer%2C+AA&amp;rft.au=Drake%2C+J&amp;rft.au=Heiner%2C+C&amp;rft.au=Clum%2C+A&amp;rft.au=Copeland%2C+A&amp;rft.au=Huddleston%2C+J&amp;rft.au=Eichler%2C+EE&amp;rft.au=Turner%2C+SW&amp;rft.au=Korlach%2C+J&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-flxlexblog.wordpress.com-80"><span class="mw-cite-backlink">^ <a href="#cite_ref-flxlexblog.wordpress.com_80-0"><sup><i><b>a</b></i></sup></a> <a href="#cite_ref-flxlexblog.wordpress.com_80-1"><sup><i><b>b</b></i></sup></a></span> <span class="reference-text"><cite class="citation web"><a rel="nofollow" class="external text" href="http://flxlexblog.wordpress.com/2013/07/05/de-novo-bacterial-genome-assembly-a-solved-problem/">"De novo bacterial genome assembly: a solved problem?"</a>. 5 July 2013.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook&amp;rft.genre=unknown&amp;rft.btitle=De+novo+bacterial+genome+assembly%3A+a+solved+problem%3F&amp;rft.date=2013-07-05&amp;rft_id=http%3A%2F%2Fflxlexblog.wordpress.com%2F2013%2F07%2F05%2Fde-novo-bacterial-genome-assembly-a-solved-problem%2F&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-rasko2011-81"><span class="mw-cite-backlink"><b><a href="#cite_ref-rasko2011_81-0">^</a></b></span> <span class="reference-text"><cite class="citation journal">Rasko DA, Webster DR, Sahl JW, Bashir A, Boisen N, Scheutz F, Paxinos EE, Sebra R, Chin CS, Iliopoulos D, Klammer A, Peluso P, Lee L, Kislyuk AO, Bullard J, Kasarskis A, Wang S, Eid J, Rank D, Redman JC, Steyert SR, Frimodt-Møller J, Struve C, Petersen AM, Krogfelt KA, Nataro JP, Schadt EE, Waldor MK (25 August 2011). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3168948">"Origins of the Strain Causing an Outbreak of Hemolytic–Uremic Syndrome in Germany"</a>. <i><a href="/wiki/N_Engl_J_Med" class="mw-redirect" title="N Engl J Med">N Engl J Med</a></i>. <b>365</b> (8): 709–17. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1056%2FNEJMoa1106920">10.1056/NEJMoa1106920</a>. <a href="/wiki/PubMed_Central" title="PubMed Central">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3168948">3168948</a></span>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/21793740">21793740</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=N+Engl+J+Med&amp;rft.atitle=Origins+of+the+Strain+Causing+an+Outbreak+of+Hemolytic%E2%80%93Uremic+Syndrome+in+Germany&amp;rft.volume=365&amp;rft.issue=8&amp;rft.pages=709-17&amp;rft.date=2011-08-25&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3168948&amp;rft_id=info%3Apmid%2F21793740&amp;rft_id=info%3Adoi%2F10.1056%2FNEJMoa1106920&amp;rft.aulast=Rasko&amp;rft.aufirst=DA&amp;rft.au=Webster%2C+DR&amp;rft.au=Sahl%2C+JW&amp;rft.au=Bashir%2C+A&amp;rft.au=Boisen%2C+N&amp;rft.au=Scheutz%2C+F&amp;rft.au=Paxinos%2C+EE&amp;rft.au=Sebra%2C+R&amp;rft.au=Chin%2C+CS&amp;rft.au=Iliopoulos%2C+D&amp;rft.au=Klammer%2C+A&amp;rft.au=Peluso%2C+P&amp;rft.au=Lee%2C+L&amp;rft.au=Kislyuk%2C+AO&amp;rft.au=Bullard%2C+J&amp;rft.au=Kasarskis%2C+A&amp;rft.au=Wang%2C+S&amp;rft.au=Eid%2C+J&amp;rft.au=Rank%2C+D&amp;rft.au=Redman%2C+JC&amp;rft.au=Steyert%2C+SR&amp;rft.au=Frimodt-M%C3%B8ller%2C+J&amp;rft.au=Struve%2C+C&amp;rft.au=Petersen%2C+AM&amp;rft.au=Krogfelt%2C+KA&amp;rft.au=Nataro%2C+JP&amp;rft.au=Schadt%2C+EE&amp;rft.au=Waldor%2C+MK&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3168948&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/><span style="position:relative; top: -2px;"><a href="/wiki/Open_access" title="open access publication – free to read"><img alt="open access" src="//upload.wikimedia.org/wikipedia/commons/thumb/7/77/Open_Access_logo_PLoS_transparent.svg/9px-Open_Access_logo_PLoS_transparent.svg.png" decoding="async" width="9" height="14" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/7/77/Open_Access_logo_PLoS_transparent.svg/14px-Open_Access_logo_PLoS_transparent.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/7/77/Open_Access_logo_PLoS_transparent.svg/18px-Open_Access_logo_PLoS_transparent.svg.png 2x" data-file-width="640" data-file-height="1000" /></a></span></span>
</li>
<li id="cite_note-tran2012-82"><span class="mw-cite-backlink"><b><a href="#cite_ref-tran2012_82-0">^</a></b></span> <span class="reference-text"><cite class="citation journal">Tran B, Brown AM, Bedard PL, Winquist E, Goss GD, Hotte SJ, Welch SA, Hirte HW, Zhang T, <a href="/wiki/Lincoln_Stein" title="Lincoln Stein">Stein LD</a>, Ferretti V, Watt S, Jiao W, Ng K, Ghai S, Shaw P, Petrocelli T, <a href="/wiki/Thomas_J._Hudson" title="Thomas J. Hudson">Hudson TJ</a>, Neel BG, Onetto N, Siu LL, McPherson JD, Kamel-Reid S, Dancey JE (1 January 2012). "Feasibility of real time next generation sequencing of cancer genes linked to drug response: Results from a clinical trial". <i><a href="/wiki/Int._J._Cancer" class="mw-redirect" title="Int. J. Cancer">Int. J. Cancer</a></i>. <b>132</b> (7): 1547–55. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1002%2Fijc.27817">10.1002/ijc.27817</a>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/22948899">22948899</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Int.+J.+Cancer&amp;rft.atitle=Feasibility+of+real+time+next+generation+sequencing+of+cancer+genes+linked+to+drug+response%3A+Results+from+a+clinical+trial&amp;rft.volume=132&amp;rft.issue=7&amp;rft.pages=1547-55&amp;rft.date=2012-01-01&amp;rft_id=info%3Adoi%2F10.1002%2Fijc.27817&amp;rft_id=info%3Apmid%2F22948899&amp;rft.aulast=Tran&amp;rft.aufirst=B&amp;rft.au=Brown%2C+AM&amp;rft.au=Bedard%2C+PL&amp;rft.au=Winquist%2C+E&amp;rft.au=Goss%2C+GD&amp;rft.au=Hotte%2C+SJ&amp;rft.au=Welch%2C+SA&amp;rft.au=Hirte%2C+HW&amp;rft.au=Zhang%2C+T&amp;rft.au=Stein%2C+LD&amp;rft.au=Ferretti%2C+V&amp;rft.au=Watt%2C+S&amp;rft.au=Jiao%2C+W&amp;rft.au=Ng%2C+K&amp;rft.au=Ghai%2C+S&amp;rft.au=Shaw%2C+P&amp;rft.au=Petrocelli%2C+T&amp;rft.au=Hudson%2C+TJ&amp;rft.au=Neel%2C+BG&amp;rft.au=Onetto%2C+N&amp;rft.au=Siu%2C+LL&amp;rft.au=McPherson%2C+JD&amp;rft.au=Kamel-Reid%2C+S&amp;rft.au=Dancey%2C+JE&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/><span style="font-size:0.95em; font-size:90%; color:#555">(subscription required)</span></span>
</li>
<li id="cite_note-83"><span class="mw-cite-backlink"><b><a href="#cite_ref-83">^</a></b></span> <span class="reference-text"><cite class="citation journal">Murray IA, Clark TA, Morgan RD, Boitano M, Anton BP, Luong K, Fomenkov A, Turner SW, Korlach J, Roberts RJ (2 October 2012). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3526280">"The methylomes of six bacteria"</a>. <i>Nucleic Acids Research</i>. <b>40</b> (22): 11450–62. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fnar%2Fgks891">10.1093/nar/gks891</a>. <a href="/wiki/PubMed_Central" title="PubMed Central">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3526280">3526280</a></span>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/23034806">23034806</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Nucleic+Acids+Research&amp;rft.atitle=The+methylomes+of+six+bacteria&amp;rft.volume=40&amp;rft.issue=22&amp;rft.pages=11450-62&amp;rft.date=2012-10-02&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3526280&amp;rft_id=info%3Apmid%2F23034806&amp;rft_id=info%3Adoi%2F10.1093%2Fnar%2Fgks891&amp;rft.aulast=Murray&amp;rft.aufirst=IA&amp;rft.au=Clark%2C+TA&amp;rft.au=Morgan%2C+RD&amp;rft.au=Boitano%2C+M&amp;rft.au=Anton%2C+BP&amp;rft.au=Luong%2C+K&amp;rft.au=Fomenkov%2C+A&amp;rft.au=Turner%2C+SW&amp;rft.au=Korlach%2C+J&amp;rft.au=Roberts%2C+RJ&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3526280&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-84"><span class="mw-cite-backlink"><b><a href="#cite_ref-84">^</a></b></span> <span class="reference-text"><cite class="citation web"><a rel="nofollow" class="external text" href="https://www.thermofisher.com/order/catalog/product/A30670">"Ion 520 &amp; Ion 530 ExT Kit-Chef – Thermo Fisher Scientific"</a>. <i>www.thermofisher.com</i>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=unknown&amp;rft.jtitle=www.thermofisher.com&amp;rft.atitle=Ion+520+%26+Ion+530+ExT+Kit-Chef+%E2%80%93+Thermo+Fisher+Scientific&amp;rft_id=https%3A%2F%2Fwww.thermofisher.com%2Forder%2Fcatalog%2Fproduct%2FA30670&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-85"><span class="mw-cite-backlink"><b><a href="#cite_ref-85">^</a></b></span> <span class="reference-text"><cite class="citation web"><a rel="nofollow" class="external text" href="https://web.archive.org/web/20180330075720/http://129.130.90.13/ion-docs/GUID-C6419130-57D8-4DE2-BCF8-47157CB3C9A2.html">"Archived copy"</a>. Archived from <a rel="nofollow" class="external text" href="http://129.130.90.13/ion-docs/GUID-C6419130-57D8-4DE2-BCF8-47157CB3C9A2.html">the original</a> on 30 March 2018<span class="reference-accessdate">. Retrieved <span class="nowrap">29 March</span> 2018</span>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook&amp;rft.genre=unknown&amp;rft.btitle=Archived+copy&amp;rft_id=http%3A%2F%2F129.130.90.13%2Fion-docs%2FGUID-C6419130-57D8-4DE2-BCF8-47157CB3C9A2.html&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><span class="cs1-maint citation-comment">CS1 maint: archived copy as title (<a href="/wiki/Category:CS1_maint:_archived_copy_as_title" title="Category:CS1 maint: archived copy as title">link</a>)</span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-vliet2010-86"><span class="mw-cite-backlink"><b><a href="#cite_ref-vliet2010_86-0">^</a></b></span> <span class="reference-text"><cite class="citation journal">van Vliet AH (1 January 2010). "Next generation sequencing of microbial transcriptomes: challenges and opportunities". <i><a href="/wiki/FEMS_Microbiology_Letters" title="FEMS Microbiology Letters">FEMS Microbiology Letters</a></i>. <b>302</b> (1): 1–7. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1111%2Fj.1574-6968.2009.01767.x">10.1111/j.1574-6968.2009.01767.x</a>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/19735299">19735299</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=FEMS+Microbiology+Letters&amp;rft.atitle=Next+generation+sequencing+of+microbial+transcriptomes%3A+challenges+and+opportunities&amp;rft.volume=302&amp;rft.issue=1&amp;rft.pages=1-7&amp;rft.date=2010-01-01&amp;rft_id=info%3Adoi%2F10.1111%2Fj.1574-6968.2009.01767.x&amp;rft_id=info%3Apmid%2F19735299&amp;rft.aulast=van+Vliet&amp;rft.aufirst=AH&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/><span style="position:relative; top: -2px;"><a href="/wiki/Open_access" title="open access publication – free to read"><img alt="open access" src="//upload.wikimedia.org/wikipedia/commons/thumb/7/77/Open_Access_logo_PLoS_transparent.svg/9px-Open_Access_logo_PLoS_transparent.svg.png" decoding="async" width="9" height="14" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/7/77/Open_Access_logo_PLoS_transparent.svg/14px-Open_Access_logo_PLoS_transparent.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/7/77/Open_Access_logo_PLoS_transparent.svg/18px-Open_Access_logo_PLoS_transparent.svg.png 2x" data-file-width="640" data-file-height="1000" /></a></span></span>
</li>
<li id="cite_note-87"><span class="mw-cite-backlink"><b><a href="#cite_ref-87">^</a></b></span> <span class="reference-text"><cite class="citation web"><a rel="nofollow" class="external text" href="http://en.mgitech.cn/product/30.html">"BGI and MGISEQ"</a>. <i>en.mgitech.cn</i><span class="reference-accessdate">. Retrieved <span class="nowrap">5 July</span> 2018</span>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=unknown&amp;rft.jtitle=en.mgitech.cn&amp;rft.atitle=BGI+and+MGISEQ&amp;rft_id=http%3A%2F%2Fen.mgitech.cn%2Fproduct%2F30.html&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-Yu-Feng_Huang,_Sheng-Chung_Chen,_Yih-Shien_Chiang,_Tzu-Han_Chen_&amp;_Kuo-Ping_Chiu_2012_S10-88"><span class="mw-cite-backlink">^ <a href="#cite_ref-Yu-Feng_Huang,_Sheng-Chung_Chen,_Yih-Shien_Chiang,_Tzu-Han_Chen_&amp;_Kuo-Ping_Chiu_2012_S10_88-0"><sup><i><b>a</b></i></sup></a> <a href="#cite_ref-Yu-Feng_Huang,_Sheng-Chung_Chen,_Yih-Shien_Chiang,_Tzu-Han_Chen_&amp;_Kuo-Ping_Chiu_2012_S10_88-1"><sup><i><b>b</b></i></sup></a></span> <span class="reference-text"><cite class="citation journal">Huang YF, Chen SC, Chiang YS, Chen TH, Chiu KP (2012). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3521181">"Palindromic sequence impedes sequencing-by-ligation mechanism"</a>. <i><a href="/wiki/BMC_Systems_Biology" title="BMC Systems Biology">BMC Systems Biology</a></i>. 6 Suppl 2: S10. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1186%2F1752-0509-6-S2-S10">10.1186/1752-0509-6-S2-S10</a>. <a href="/wiki/PubMed_Central" title="PubMed Central">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3521181">3521181</a></span>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/23281822">23281822</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=BMC+Systems+Biology&amp;rft.atitle=Palindromic+sequence+impedes+sequencing-by-ligation+mechanism&amp;rft.volume=6+Suppl+2&amp;rft.pages=S10&amp;rft.date=2012&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3521181&amp;rft_id=info%3Apmid%2F23281822&amp;rft_id=info%3Adoi%2F10.1186%2F1752-0509-6-S2-S10&amp;rft.aulast=Huang&amp;rft.aufirst=YF&amp;rft.au=Chen%2C+SC&amp;rft.au=Chiang%2C+YS&amp;rft.au=Chen%2C+TH&amp;rft.au=Chiu%2C+KP&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3521181&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-89"><span class="mw-cite-backlink"><b><a href="#cite_ref-89">^</a></b></span> <span class="reference-text"><cite class="citation biorxiv">Loose, Matthew; Rakyan, Vardhman; Holmes, Nadine; Payne, Alexander (3 May 2018). "Whale watching with BulkVis: A graphical viewer for Oxford Nanopore bulk fast5 files". <a href="/wiki/BioRxiv" title="BioRxiv">bioRxiv</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="https://doi.org/10.1101/312256">312256</a></span>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=preprint&amp;rft.jtitle=bioRxiv&amp;rft.atitle=Whale+watching+with+BulkVis%3A+A+graphical+viewer+for+Oxford+Nanopore+bulk+fast5+files&amp;rft.date=2018-05-03&amp;rft_id=%2F%2Fdoi.org%2F10.1101%2F312256&amp;rft.aulast=Loose&amp;rft.aufirst=Matthew&amp;rft.au=Rakyan%2C+Vardhman&amp;rft.au=Holmes%2C+Nadine&amp;rft.au=Payne%2C+Alexander&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-90"><span class="mw-cite-backlink"><b><a href="#cite_ref-90">^</a></b></span> <span class="reference-text"><cite class="citation web"><a rel="nofollow" class="external text" href="http://www.genomeweb.com/sequencing/pacbio-sales-start-pick-company-delivers-product-enhancements">"PacBio Sales Start to Pick Up as Company Delivers on Product Enhancements"</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook&amp;rft.genre=unknown&amp;rft.btitle=PacBio+Sales+Start+to+Pick+Up+as+Company+Delivers+on+Product+Enhancements&amp;rft_id=http%3A%2F%2Fwww.genomeweb.com%2Fsequencing%2Fpacbio-sales-start-pick-company-delivers-product-enhancements&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-91"><span class="mw-cite-backlink"><b><a href="#cite_ref-91">^</a></b></span> <span class="reference-text"><cite class="citation web"><a rel="nofollow" class="external text" href="http://www.bio-itworld.com/2015/9/30/pacbio-announces-sequel-sequencing-system.aspx">"Bio-IT World"</a>. <i>www.bio-itworld.com</i>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=unknown&amp;rft.jtitle=www.bio-itworld.com&amp;rft.atitle=Bio-IT+World&amp;rft_id=http%3A%2F%2Fwww.bio-itworld.com%2F2015%2F9%2F30%2Fpacbio-announces-sequel-sequencing-system.aspx&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-92"><span class="mw-cite-backlink"><b><a href="#cite_ref-92">^</a></b></span> <span class="reference-text"><cite class="citation web"><a rel="nofollow" class="external text" href="https://www.genomeweb.com/business-news/pacbio-launches-higher-throughput-lower-cost-single-molecule-sequencing-system">"PacBio Launches Higher-Throughput, Lower-Cost Single-Molecule Sequencing System"</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook&amp;rft.genre=unknown&amp;rft.btitle=PacBio+Launches+Higher-Throughput%2C+Lower-Cost+Single-Molecule+Sequencing+System&amp;rft_id=https%3A%2F%2Fwww.genomeweb.com%2Fbusiness-news%2Fpacbio-launches-higher-throughput-lower-cost-single-molecule-sequencing-system&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-93"><span class="mw-cite-backlink"><b><a href="#cite_ref-93">^</a></b></span> <span class="reference-text"><cite class="citation journal">Clarke J, Wu HC, Jayasinghe L, Patel A, Reid S, Bayley H (April 2009). "Continuous base identification for single-molecule nanopore DNA sequencing". <i>Nature Nanotechnology</i>. <b>4</b> (4): 265–70. <a href="/wiki/Bibcode" title="Bibcode">Bibcode</a>:<a rel="nofollow" class="external text" href="https://ui.adsabs.harvard.edu/abs/2009NatNa...4..265C">2009NatNa...4..265C</a>. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1038%2Fnnano.2009.12">10.1038/nnano.2009.12</a>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/19350039">19350039</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Nature+Nanotechnology&amp;rft.atitle=Continuous+base+identification+for+single-molecule+nanopore+DNA+sequencing&amp;rft.volume=4&amp;rft.issue=4&amp;rft.pages=265-70&amp;rft.date=2009-04&amp;rft_id=info%3Apmid%2F19350039&amp;rft_id=info%3Adoi%2F10.1038%2Fnnano.2009.12&amp;rft_id=info%3Abibcode%2F2009NatNa...4..265C&amp;rft.aulast=Clarke&amp;rft.aufirst=J&amp;rft.au=Wu%2C+HC&amp;rft.au=Jayasinghe%2C+L&amp;rft.au=Patel%2C+A&amp;rft.au=Reid%2C+S&amp;rft.au=Bayley%2C+H&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-Torre_2012-94"><span class="mw-cite-backlink">^ <a href="#cite_ref-Torre_2012_94-0"><sup><i><b>a</b></i></sup></a> <a href="#cite_ref-Torre_2012_94-1"><sup><i><b>b</b></i></sup></a></span> <span class="reference-text"><cite class="citation journal">dela Torre R, Larkin J, Singer A, Meller A (2012). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3557807">"Fabrication and characterization of solid-state nanopore arrays for high-throughput DNA sequencing"</a>. <i>Nanotechnology</i>. <b>23</b> (38): 385308. <a href="/wiki/Bibcode" title="Bibcode">Bibcode</a>:<a rel="nofollow" class="external text" href="https://ui.adsabs.harvard.edu/abs/2012Nanot..23L5308D">2012Nanot..23L5308D</a>. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1088%2F0957-4484%2F23%2F38%2F385308">10.1088/0957-4484/23/38/385308</a>. <a href="/wiki/PubMed_Central" title="PubMed Central">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3557807">3557807</a></span>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/22948520">22948520</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Nanotechnology&amp;rft.atitle=Fabrication+and+characterization+of+solid-state+nanopore+arrays+for+high-throughput+DNA+sequencing&amp;rft.volume=23&amp;rft.issue=38&amp;rft.pages=385308&amp;rft.date=2012&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3557807&amp;rft_id=info%3Apmid%2F22948520&amp;rft_id=info%3Adoi%2F10.1088%2F0957-4484%2F23%2F38%2F385308&amp;rft_id=info%3Abibcode%2F2012Nanot..23L5308D&amp;rft.aulast=dela+Torre&amp;rft.aufirst=R&amp;rft.au=Larkin%2C+J&amp;rft.au=Singer%2C+A&amp;rft.au=Meller%2C+A&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3557807&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-Pathak_2012-95"><span class="mw-cite-backlink">^ <a href="#cite_ref-Pathak_2012_95-0"><sup><i><b>a</b></i></sup></a> <a href="#cite_ref-Pathak_2012_95-1"><sup><i><b>b</b></i></sup></a></span> <span class="reference-text"><cite class="citation journal">Pathak B, Lofas H, Prasongkit J, Grigoriev A, Ahuja R, Scheicher RH (2012). <a rel="nofollow" class="external text" href="https://zenodo.org/record/890231">"Double-functionalized nanopore-embedded gold electrodes for rapid DNA sequencing"</a>. <i>Applied Physics Letters</i>. <b>100</b> (2): 023701. <a href="/wiki/Bibcode" title="Bibcode">Bibcode</a>:<a rel="nofollow" class="external text" href="https://ui.adsabs.harvard.edu/abs/2012ApPhL.100b3701P">2012ApPhL.100b3701P</a>. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1063%2F1.3673335">10.1063/1.3673335</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Applied+Physics+Letters&amp;rft.atitle=Double-functionalized+nanopore-embedded+gold+electrodes+for+rapid+DNA+sequencing&amp;rft.volume=100&amp;rft.issue=2&amp;rft.pages=023701&amp;rft.date=2012&amp;rft_id=info%3Adoi%2F10.1063%2F1.3673335&amp;rft_id=info%3Abibcode%2F2012ApPhL.100b3701P&amp;rft.aulast=Pathak&amp;rft.aufirst=B&amp;rft.au=Lofas%2C+H&amp;rft.au=Prasongkit%2C+J&amp;rft.au=Grigoriev%2C+A&amp;rft.au=Ahuja%2C+R&amp;rft.au=Scheicher%2C+RH&amp;rft_id=https%3A%2F%2Fzenodo.org%2Frecord%2F890231&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-Korlach_2008-96"><span class="mw-cite-backlink"><b><a href="#cite_ref-Korlach_2008_96-0">^</a></b></span> <span class="reference-text"><cite class="citation journal">Korlach J, Marks PJ, Cicero RL, Gray JJ, Murphy DL, Roitman DB, Pham TT, Otto GA, Foquet M, Turner SW (2008). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC2234111">"Selective aluminum passivation for targeted immobilization of single DNA polymerase molecules in zero-mode waveguide nanostructures"</a>. <i>Proceedings of the National Academy of Sciences</i>. <b>105</b> (4): 1176–81. <a href="/wiki/Bibcode" title="Bibcode">Bibcode</a>:<a rel="nofollow" class="external text" href="https://ui.adsabs.harvard.edu/abs/2008PNAS..105.1176K">2008PNAS..105.1176K</a>. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1073%2Fpnas.0710982105">10.1073/pnas.0710982105</a>. <a href="/wiki/PubMed_Central" title="PubMed Central">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC2234111">2234111</a></span>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/18216253">18216253</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Proceedings+of+the+National+Academy+of+Sciences&amp;rft.atitle=Selective+aluminum+passivation+for+targeted+immobilization+of+single+DNA+polymerase+molecules+in+zero-mode+waveguide+nanostructures&amp;rft.volume=105&amp;rft.issue=4&amp;rft.pages=1176-81&amp;rft.date=2008&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2234111&amp;rft_id=info%3Apmid%2F18216253&amp;rft_id=info%3Adoi%2F10.1073%2Fpnas.0710982105&amp;rft_id=info%3Abibcode%2F2008PNAS..105.1176K&amp;rft.aulast=Korlach&amp;rft.aufirst=J&amp;rft.au=Marks%2C+PJ&amp;rft.au=Cicero%2C+RL&amp;rft.au=Gray%2C+JJ&amp;rft.au=Murphy%2C+DL&amp;rft.au=Roitman%2C+DB&amp;rft.au=Pham%2C+TT&amp;rft.au=Otto%2C+GA&amp;rft.au=Foquet%2C+M&amp;rft.au=Turner%2C+SW&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2234111&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-Shendure2005-97"><span class="mw-cite-backlink">^ <a href="#cite_ref-Shendure2005_97-0"><sup><i><b>a</b></i></sup></a> <a href="#cite_ref-Shendure2005_97-1"><sup><i><b>b</b></i></sup></a></span> <span class="reference-text"><cite class="citation journal">Shendure J, Porreca GJ, Reppas NB, Lin X, McCutcheon JP, Rosenbaum AM, Wang MD, Zhang K, Mitra RD, Church GM (9 September 2005). "Accurate multiplex polony sequencing of an evolved bacterial genome". <i>Science</i>. <b>309</b> (5741): 1728–32. <a href="/wiki/Bibcode" title="Bibcode">Bibcode</a>:<a rel="nofollow" class="external text" href="https://ui.adsabs.harvard.edu/abs/2005Sci...309.1728S">2005Sci...309.1728S</a>. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1126%2Fscience.1117389">10.1126/science.1117389</a>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/16081699">16081699</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Science&amp;rft.atitle=Accurate+multiplex+polony+sequencing+of+an+evolved+bacterial+genome.&amp;rft.volume=309&amp;rft.issue=5741&amp;rft.pages=1728-32&amp;rft.date=2005-09-09&amp;rft_id=info%3Apmid%2F16081699&amp;rft_id=info%3Adoi%2F10.1126%2Fscience.1117389&amp;rft_id=info%3Abibcode%2F2005Sci...309.1728S&amp;rft.aulast=Shendure&amp;rft.aufirst=J&amp;rft.au=Porreca%2C+GJ&amp;rft.au=Reppas%2C+NB&amp;rft.au=Lin%2C+X&amp;rft.au=McCutcheon%2C+JP&amp;rft.au=Rosenbaum%2C+AM&amp;rft.au=Wang%2C+MD&amp;rft.au=Zhang%2C+K&amp;rft.au=Mitra%2C+RD&amp;rft.au=Church%2C+GM&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-Bentley_2008-98"><span class="mw-cite-backlink"><b><a href="#cite_ref-Bentley_2008_98-0">^</a></b></span> <span class="reference-text"><cite class="citation journal">Bentley DR, Balasubramanian S,  et al. (2008). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC2581791">"Accurate whole human genome sequencing using reversible terminator chemistry"</a>. <i>Nature</i>. <b>456</b> (7218): 53–59. <a href="/wiki/Bibcode" title="Bibcode">Bibcode</a>:<a rel="nofollow" class="external text" href="https://ui.adsabs.harvard.edu/abs/2008Natur.456...53B">2008Natur.456...53B</a>. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1038%2Fnature07517">10.1038/nature07517</a>. <a href="/wiki/PubMed_Central" title="PubMed Central">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC2581791">2581791</a></span>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/18987734">18987734</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Nature&amp;rft.atitle=Accurate+whole+human+genome+sequencing+using+reversible+terminator+chemistry&amp;rft.volume=456&amp;rft.issue=7218&amp;rft.pages=53-59&amp;rft.date=2008&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2581791&amp;rft_id=info%3Apmid%2F18987734&amp;rft_id=info%3Adoi%2F10.1038%2Fnature07517&amp;rft_id=info%3Abibcode%2F2008Natur.456...53B&amp;rft.aulast=Bentley&amp;rft.aufirst=DR&amp;rft.au=Balasubramanian%2C+S&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2581791&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-99"><span class="mw-cite-backlink"><b><a href="#cite_ref-99">^</a></b></span> <span class="reference-text"><cite id="CITEREFCanardSarfati1994" class="citation">Canard B, Sarfati S (13 October 1994), <a rel="nofollow" class="external text" href="http://www.google.ge/patents/CA2158975A1"><i>Novel derivatives usable for the sequencing of nucleic acids</i></a><span class="reference-accessdate">, retrieved <span class="nowrap">9 March</span> 2016</span></cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook&amp;rft.genre=book&amp;rft.btitle=Novel+derivatives+usable+for+the+sequencing+of+nucleic+acids&amp;rft.date=1994-10-13&amp;rft.aulast=Canard&amp;rft.aufirst=Bruno&amp;rft.au=Sarfati%2C+Simon&amp;rft_id=http%3A%2F%2Fwww.google.ge%2Fpatents%2FCA2158975A1&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-100"><span class="mw-cite-backlink"><b><a href="#cite_ref-100">^</a></b></span> <span class="reference-text"><cite class="citation journal">Canard B, Sarfati RS (October 1994). "DNA polymerase fluorescent substrates with reversible 3'-tags". <i>Gene</i>. <b>148</b> (1): 1–6. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1016%2F0378-1119%2894%2990226-7">10.1016/0378-1119(94)90226-7</a>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/7523248">7523248</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Gene&amp;rft.atitle=DNA+polymerase+fluorescent+substrates+with+reversible+3%27-tags&amp;rft.volume=148&amp;rft.issue=1&amp;rft.pages=1-6&amp;rft.date=1994-10&amp;rft_id=info%3Adoi%2F10.1016%2F0378-1119%2894%2990226-7&amp;rft_id=info%3Apmid%2F7523248&amp;rft.aulast=Canard&amp;rft.aufirst=B&amp;rft.au=Sarfati%2C+RS&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-pmid18576944-101"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid18576944_101-0">^</a></b></span> <span class="reference-text"><cite class="citation journal">Mardis ER (2008). "Next-generation DNA sequencing methods". <i>Annu Rev Genom Hum Genet</i>. <b>9</b>: 387–402. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1146%2Fannurev.genom.9.081307.164359">10.1146/annurev.genom.9.081307.164359</a>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/18576944">18576944</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Annu+Rev+Genom+Hum+Genet&amp;rft.atitle=Next-generation+DNA+sequencing+methods&amp;rft.volume=9&amp;rft.pages=387-402&amp;rft.date=2008&amp;rft_id=info%3Adoi%2F10.1146%2Fannurev.genom.9.081307.164359&amp;rft_id=info%3Apmid%2F18576944&amp;rft.aulast=Mardis&amp;rft.aufirst=ER&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-:0-102"><span class="mw-cite-backlink">^ <a href="#cite_ref-:0_102-0"><sup><i><b>a</b></i></sup></a> <a href="#cite_ref-:0_102-1"><sup><i><b>b</b></i></sup></a> <a href="#cite_ref-:0_102-2"><sup><i><b>c</b></i></sup></a></span> <span class="reference-text"><cite class="citation journal">Drmanac R, Sparks AB, Callow MJ, Halpern AL, Burns NL, Kermani BG,  et al. (January 2010). "Human genome sequencing using unchained base reads on self-assembling DNA nanoarrays". <i>Science</i>. <b>327</b> (5961): 78–81. <a href="/wiki/Bibcode" title="Bibcode">Bibcode</a>:<a rel="nofollow" class="external text" href="https://ui.adsabs.harvard.edu/abs/2010Sci...327...78D">2010Sci...327...78D</a>. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1126%2Fscience.1181498">10.1126/science.1181498</a>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/19892942">19892942</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Science&amp;rft.atitle=Human+genome+sequencing+using+unchained+base+reads+on+self-assembling+DNA+nanoarrays&amp;rft.volume=327&amp;rft.issue=5961&amp;rft.pages=78-81&amp;rft.date=2010-01&amp;rft_id=info%3Apmid%2F19892942&amp;rft_id=info%3Adoi%2F10.1126%2Fscience.1181498&amp;rft_id=info%3Abibcode%2F2010Sci...327...78D&amp;rft.aulast=Drmanac&amp;rft.aufirst=R&amp;rft.au=Sparks%2C+AB&amp;rft.au=Callow%2C+MJ&amp;rft.au=Halpern%2C+AL&amp;rft.au=Burns%2C+NL&amp;rft.au=Kermani%2C+BG&amp;rft.au=Carnevali%2C+P&amp;rft.au=Nazarenko%2C+I&amp;rft.au=Nilsen%2C+GB&amp;rft.au=Yeung%2C+G&amp;rft.au=Dahl%2C+F&amp;rft.au=Fernandez%2C+A&amp;rft.au=Staker%2C+B&amp;rft.au=Pant%2C+KP&amp;rft.au=Baccash%2C+J&amp;rft.au=Borcherding%2C+AP&amp;rft.au=Brownley%2C+A&amp;rft.au=Cedeno%2C+R&amp;rft.au=Chen%2C+L&amp;rft.au=Chernikoff%2C+D&amp;rft.au=Cheung%2C+A&amp;rft.au=Chirita%2C+R&amp;rft.au=Curson%2C+B&amp;rft.au=Ebert%2C+JC&amp;rft.au=Hacker%2C+CR&amp;rft.au=Hartlage%2C+R&amp;rft.au=Hauser%2C+B&amp;rft.au=Huang%2C+S&amp;rft.au=Jiang%2C+Y&amp;rft.au=Karpinchyk%2C+V&amp;rft.au=Koenig%2C+M&amp;rft.au=Kong%2C+C&amp;rft.au=Landers%2C+T&amp;rft.au=Le%2C+C&amp;rft.au=Liu%2C+J&amp;rft.au=McBride%2C+CE&amp;rft.au=Morenzoni%2C+M&amp;rft.au=Morey%2C+RE&amp;rft.au=Mutch%2C+K&amp;rft.au=Perazich%2C+H&amp;rft.au=Perry%2C+K&amp;rft.au=Peters%2C+BA&amp;rft.au=Peterson%2C+J&amp;rft.au=Pethiyagoda%2C+CL&amp;rft.au=Pothuraju%2C+K&amp;rft.au=Richter%2C+C&amp;rft.au=Rosenbaum%2C+AM&amp;rft.au=Roy%2C+S&amp;rft.au=Shafto%2C+J&amp;rft.au=Sharanhovich%2C+U&amp;rft.au=Shannon%2C+KW&amp;rft.au=Sheppy%2C+CG&amp;rft.au=Sun%2C+M&amp;rft.au=Thakuria%2C+JV&amp;rft.au=Tran%2C+A&amp;rft.au=Vu%2C+D&amp;rft.au=Zaranek%2C+AW&amp;rft.au=Wu%2C+X&amp;rft.au=Drmanac%2C+S&amp;rft.au=Oliphant%2C+AR&amp;rft.au=Banyai%2C+WC&amp;rft.au=Martin%2C+B&amp;rft.au=Ballinger%2C+DG&amp;rft.au=Church%2C+GM&amp;rft.au=Reid%2C+CA&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-103"><span class="mw-cite-backlink"><b><a href="#cite_ref-103">^</a></b></span> <span class="reference-text"><cite class="citation web">brandonvd. <a rel="nofollow" class="external text" href="http://www.completegenomics.com/">"About Us - Complete Genomics"</a>. <i>Complete Genomics</i><span class="reference-accessdate">. Retrieved <span class="nowrap">2 July</span> 2018</span>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=unknown&amp;rft.jtitle=Complete+Genomics&amp;rft.atitle=About+Us+-+Complete+Genomics&amp;rft.au=brandonvd&amp;rft_id=http%3A%2F%2Fwww.completegenomics.com%2F&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-:1-104"><span class="mw-cite-backlink">^ <a href="#cite_ref-:1_104-0"><sup><i><b>a</b></i></sup></a> <a href="#cite_ref-:1_104-1"><sup><i><b>b</b></i></sup></a></span> <span class="reference-text"><cite class="citation journal">Huang J, Liang X, Xuan Y, Geng C, Li Y, Lu H,  et al. (May 2017). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC5467036">"A reference human genome dataset of the BGISEQ-500 sequencer"</a>. <i>GigaScience</i>. <b>6</b> (5): 1–9. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fgigascience%2Fgix024">10.1093/gigascience/gix024</a>. <a href="/wiki/PubMed_Central" title="PubMed Central">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC5467036">5467036</a></span>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/28379488">28379488</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=GigaScience&amp;rft.atitle=A+reference+human+genome+dataset+of+the+BGISEQ-500+sequencer&amp;rft.volume=6&amp;rft.issue=5&amp;rft.pages=1-9&amp;rft.date=2017-05&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5467036&amp;rft_id=info%3Apmid%2F28379488&amp;rft_id=info%3Adoi%2F10.1093%2Fgigascience%2Fgix024&amp;rft.aulast=Huang&amp;rft.aufirst=J&amp;rft.au=Liang%2C+X&amp;rft.au=Xuan%2C+Y&amp;rft.au=Geng%2C+C&amp;rft.au=Li%2C+Y&amp;rft.au=Lu%2C+H&amp;rft.au=Qu%2C+S&amp;rft.au=Mei%2C+X&amp;rft.au=Chen%2C+H&amp;rft.au=Yu%2C+T&amp;rft.au=Sun%2C+N&amp;rft.au=Rao%2C+J&amp;rft.au=Wang%2C+J&amp;rft.au=Zhang%2C+W&amp;rft.au=Chen%2C+Y&amp;rft.au=Liao%2C+S&amp;rft.au=Jiang%2C+H&amp;rft.au=Liu%2C+X&amp;rft.au=Yang%2C+Z&amp;rft.au=Mu%2C+F&amp;rft.au=Gao%2C+S&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5467036&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-pmid18477713-105"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid18477713_105-0">^</a></b></span> <span class="reference-text"><cite class="citation journal">Valouev A, Ichikawa J, Tonthat T, Stuart J, Ranade S, Peckham H, Zeng K, Malek JA, Costa G, McKernan K, Sidow A, Fire A, Johnson SM (July 2008). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC2493394">"A high-resolution, nucleosome position map of C. elegans reveals a lack of universal sequence-dictated positioning"</a>. <i>Genome Res</i>. <b>18</b> (7): 1051–63. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1101%2Fgr.076463.108">10.1101/gr.076463.108</a>. <a href="/wiki/PubMed_Central" title="PubMed Central">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC2493394">2493394</a></span>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/18477713">18477713</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Genome+Res.&amp;rft.atitle=A+high-resolution%2C+nucleosome+position+map+of+C.+elegans+reveals+a+lack+of+universal+sequence-dictated+positioning&amp;rft.volume=18&amp;rft.issue=7&amp;rft.pages=1051-63&amp;rft.date=2008-07&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2493394&amp;rft_id=info%3Apmid%2F18477713&amp;rft_id=info%3Adoi%2F10.1101%2Fgr.076463.108&amp;rft.aulast=Valouev&amp;rft.aufirst=A&amp;rft.au=Ichikawa%2C+J&amp;rft.au=Tonthat%2C+T&amp;rft.au=Stuart%2C+J&amp;rft.au=Ranade%2C+S&amp;rft.au=Peckham%2C+H&amp;rft.au=Zeng%2C+K&amp;rft.au=Malek%2C+JA&amp;rft.au=Costa%2C+G&amp;rft.au=McKernan%2C+K&amp;rft.au=Sidow%2C+A&amp;rft.au=Fire%2C+A&amp;rft.au=Johnson%2C+SM&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2493394&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-rusk-106"><span class="mw-cite-backlink"><b><a href="#cite_ref-rusk_106-0">^</a></b></span> <span class="reference-text"><cite class="citation journal">Rusk N (2011). "Torrents of sequence". <i>Nat Methods</i>. <b>8</b> (1): 44. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1038%2Fnmeth.f.330">10.1038/nmeth.f.330</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Nat+Methods&amp;rft.atitle=Torrents+of+sequence&amp;rft.volume=8&amp;rft.issue=1&amp;rft.pages=44&amp;rft.date=2011&amp;rft_id=info%3Adoi%2F10.1038%2Fnmeth.f.330&amp;rft.aulast=Rusk&amp;rft.aufirst=N&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-Drmanac_2010-107"><span class="mw-cite-backlink">^ <a href="#cite_ref-Drmanac_2010_107-0"><sup><i><b>a</b></i></sup></a> <a href="#cite_ref-Drmanac_2010_107-1"><sup><i><b>b</b></i></sup></a></span> <span class="reference-text"><cite class="citation journal">Drmanac R, Sparks AB,  et al. (2010). "Human Genome Sequencing Using Unchained Base Reads in Self-Assembling DNA Nanoarrays". <i>Science</i>. <b>327</b> (5961): 78–81. <a href="/wiki/Bibcode" title="Bibcode">Bibcode</a>:<a rel="nofollow" class="external text" href="https://ui.adsabs.harvard.edu/abs/2010Sci...327...78D">2010Sci...327...78D</a>. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1126%2Fscience.1181498">10.1126/science.1181498</a>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/19892942">19892942</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Science&amp;rft.atitle=Human+Genome+Sequencing+Using+Unchained+Base+Reads+in+Self-Assembling+DNA+Nanoarrays&amp;rft.volume=327&amp;rft.issue=5961&amp;rft.pages=78-81&amp;rft.date=2010&amp;rft_id=info%3Apmid%2F19892942&amp;rft_id=info%3Adoi%2F10.1126%2Fscience.1181498&amp;rft_id=info%3Abibcode%2F2010Sci...327...78D&amp;rft.aulast=Drmanac&amp;rft.aufirst=R&amp;rft.au=Sparks%2C+AB&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-108"><span class="mw-cite-backlink"><b><a href="#cite_ref-108">^</a></b></span> <span class="reference-text"><cite class="citation journal">Porreca GJ (2010). "Genome Sequencing on Nanoballs". <i>Nature Biotechnology</i>. <b>28</b> (1): 43–44. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1038%2Fnbt0110-43">10.1038/nbt0110-43</a>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/20062041">20062041</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Nature+Biotechnology&amp;rft.atitle=Genome+Sequencing+on+Nanoballs&amp;rft.volume=28&amp;rft.issue=1&amp;rft.pages=43-44&amp;rft.date=2010&amp;rft_id=info%3Adoi%2F10.1038%2Fnbt0110-43&amp;rft_id=info%3Apmid%2F20062041&amp;rft.aulast=Porreca&amp;rft.aufirst=GJ&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-109"><span class="mw-cite-backlink"><b><a href="#cite_ref-109">^</a></b></span> <span class="reference-text"><a rel="nofollow" class="external text" href="https://web.archive.org/web/20100825003320/http://www.completegenomics.com/news-events/press-releases/">Complete Genomics</a> Press release, 2010</span>
</li>
<li id="cite_note-110"><span class="mw-cite-backlink"><b><a href="#cite_ref-110">^</a></b></span> <span class="reference-text"><cite class="citation web"><a rel="nofollow" class="external text" href="https://web.archive.org/web/20091102041828/http://www.helicosbio.com/Products/HelicosregGeneticAnalysisSystem/HeliScopetradeSequencer/tabid/87/Default.aspx">"HeliScope Gene Sequencing / Genetic Analyzer System&#160;: Helicos BioSciences"</a>. 2 November 2009. Archived from <a rel="nofollow" class="external text" href="http://www.helicosbio.com/Products/HelicosregGeneticAnalysisSystem/HeliScopetradeSequencer/tabid/87/Default.aspx">the original</a> on 2 November 2009.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook&amp;rft.genre=unknown&amp;rft.btitle=HeliScope+Gene+Sequencing+%2F+Genetic+Analyzer+System+%3A+Helicos+BioSciences&amp;rft.date=2009-11-02&amp;rft_id=http%3A%2F%2Fwww.helicosbio.com%2FProducts%2FHelicosregGeneticAnalysisSystem%2FHeliScopetradeSequencer%2Ftabid%2F87%2FDefault.aspx&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-111"><span class="mw-cite-backlink"><b><a href="#cite_ref-111">^</a></b></span> <span class="reference-text"><cite class="citation book">Thompson JF, Steinmann KE (October 2010). <i>Single molecule sequencing with a HeliScope genetic analysis system</i>. <i>Current Protocols in Molecular Biology</i>. Chapter 7. pp.&#160;Unit7.10. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1002%2F0471142727.mb0710s92">10.1002/0471142727.mb0710s92</a>. <a href="/wiki/International_Standard_Book_Number" title="International Standard Book Number">ISBN</a>&#160;<a href="/wiki/Special:BookSources/978-0471142720" title="Special:BookSources/978-0471142720"><bdi>978-0471142720</bdi></a>. <a href="/wiki/PubMed_Central" title="PubMed Central">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC2954431">2954431</a></span>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/20890904">20890904</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook&amp;rft.genre=book&amp;rft.btitle=Single+molecule+sequencing+with+a+HeliScope+genetic+analysis+system&amp;rft.pages=Unit7.10&amp;rft.date=2010-10&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2954431&amp;rft_id=info%3Apmid%2F20890904&amp;rft_id=info%3Adoi%2F10.1002%2F0471142727.mb0710s92&amp;rft.isbn=978-0471142720&amp;rft.aulast=Thompson&amp;rft.aufirst=JF&amp;rft.au=Steinmann%2C+KE&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-112"><span class="mw-cite-backlink"><b><a href="#cite_ref-112">^</a></b></span> <span class="reference-text"><cite class="citation web"><a rel="nofollow" class="external text" href="https://web.archive.org/web/20140808055229/http://seqll.com/technical-description/">"tSMS SeqLL Technical Explanation"</a>. SeqLL. Archived from <a rel="nofollow" class="external text" href="http://seqll.com/technical-description/">the original</a> on 8 August 2014<span class="reference-accessdate">. Retrieved <span class="nowrap">9 August</span> 2015</span>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook&amp;rft.genre=unknown&amp;rft.btitle=tSMS+SeqLL+Technical+Explanation&amp;rft.pub=SeqLL&amp;rft_id=http%3A%2F%2Fseqll.com%2Ftechnical-description%2F&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-113"><span class="mw-cite-backlink"><b><a href="#cite_ref-113">^</a></b></span> <span class="reference-text"><cite class="citation journal">Heather, James M.; Chain, Benjamin (January 2016). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC4727787">"The sequence of sequencers: The history of sequencing DNA"</a>. <i>Genomics</i>. <b>107</b> (1): 1–8. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1016%2Fj.ygeno.2015.11.003">10.1016/j.ygeno.2015.11.003</a>. <a href="/wiki/International_Standard_Serial_Number" title="International Standard Serial Number">ISSN</a>&#160;<a rel="nofollow" class="external text" href="//www.worldcat.org/issn/1089-8646">1089-8646</a>. <a href="/wiki/PubMed_Central" title="PubMed Central">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC4727787">4727787</a></span>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/26554401">26554401</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Genomics&amp;rft.atitle=The+sequence+of+sequencers%3A+The+history+of+sequencing+DNA&amp;rft.volume=107&amp;rft.issue=1&amp;rft.pages=1-8&amp;rft.date=2016-01&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4727787&amp;rft.issn=1089-8646&amp;rft_id=info%3Apmid%2F26554401&amp;rft_id=info%3Adoi%2F10.1016%2Fj.ygeno.2015.11.003&amp;rft.aulast=Heather&amp;rft.aufirst=James+M.&amp;rft.au=Chain%2C+Benjamin&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4727787&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-114"><span class="mw-cite-backlink"><b><a href="#cite_ref-114">^</a></b></span> <span class="reference-text"><cite class="citation book">Sara El-Metwally; Osama M. Ouda; Mohamed Helmy (2014). "New Horizons in Next-Generation Sequencing". <i>Next Generation Sequencing Technologies and Challenges in Sequence Assembly</i>. <i>Next Generation Sequencing Technologies and Challenges in Sequence Assembly, Springer Briefs in Systems Biology Volume 7</i>. SpringerBriefs in Systems Biology. <b>7</b>. pp.&#160;51–59. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1007%2F978-1-4939-0715-1_6">10.1007/978-1-4939-0715-1_6</a>. <a href="/wiki/International_Standard_Book_Number" title="International Standard Book Number">ISBN</a>&#160;<a href="/wiki/Special:BookSources/978-1-4939-0714-4" title="Special:BookSources/978-1-4939-0714-4"><bdi>978-1-4939-0714-4</bdi></a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook&amp;rft.genre=bookitem&amp;rft.atitle=New+Horizons+in+Next-Generation+Sequencing&amp;rft.btitle=Next+Generation+Sequencing+Technologies+and+Challenges+in+Sequence+Assembly&amp;rft.series=SpringerBriefs+in+Systems+Biology&amp;rft.pages=51-59&amp;rft.date=2014&amp;rft_id=info%3Adoi%2F10.1007%2F978-1-4939-0715-1_6&amp;rft.isbn=978-1-4939-0714-4&amp;rft.au=Sara+El-Metwally&amp;rft.au=Osama+M.+Ouda&amp;rft.au=Mohamed+Helmy&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-:4-115"><span class="mw-cite-backlink">^ <a href="#cite_ref-:4_115-0"><sup><i><b>a</b></i></sup></a> <a href="#cite_ref-:4_115-1"><sup><i><b>b</b></i></sup></a></span> <span class="reference-text"><cite class="citation journal">Fair RB, Khlystov A, Tailor TD, Ivanov V, Evans RD, Srinivasan V, Pamula VK, Pollack MG, Griffin PB, Zhou J (January 2007). "Chemical and Biological Applications of Digital-Microfluidic Devices". <i>IEEE Design &amp; Test of Computers</i>. <b>24</b> (1): 10–24. <a href="/wiki/CiteSeerX" title="CiteSeerX">CiteSeerX</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//citeseerx.ist.psu.edu/viewdoc/summary?doi=10.1.1.559.1440">10.1.1.559.1440</a></span>. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1109%2FMDT.2007.8">10.1109/MDT.2007.8</a>. <a href="/wiki/Handle_System" title="Handle System">hdl</a>:<a rel="nofollow" class="external text" href="//hdl.handle.net/10161%2F6987">10161/6987</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=IEEE+Design+%26+Test+of+Computers&amp;rft.atitle=Chemical+and+Biological+Applications+of+Digital-Microfluidic+Devices&amp;rft.volume=24&amp;rft.issue=1&amp;rft.pages=10-24&amp;rft.date=2007-01&amp;rft_id=%2F%2Fciteseerx.ist.psu.edu%2Fviewdoc%2Fsummary%3Fdoi%3D10.1.1.559.1440&amp;rft_id=info%3Ahdl%2F10161%2F6987&amp;rft_id=info%3Adoi%2F10.1109%2FMDT.2007.8&amp;rft.aulast=Fair&amp;rft.aufirst=RB&amp;rft.au=Khlystov%2C+A&amp;rft.au=Tailor%2C+TD&amp;rft.au=Ivanov%2C+V&amp;rft.au=Evans%2C+RD&amp;rft.au=Srinivasan%2C+V&amp;rft.au=Pamula%2C+VK&amp;rft.au=Pollack%2C+MG&amp;rft.au=Griffin%2C+PB&amp;rft.au=Zhou%2C+J&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-:5-116"><span class="mw-cite-backlink">^ <a href="#cite_ref-:5_116-0"><sup><i><b>a</b></i></sup></a> <a href="#cite_ref-:5_116-1"><sup><i><b>b</b></i></sup></a></span> <span class="reference-text"><cite class="citation journal">Boles DJ, Benton JL, Siew GJ, Levy MH, Thwar PK, Sandahl MA,  et al. (November 2011). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3690483">"Droplet-based pyrosequencing using digital microfluidics"</a>. <i>Analytical Chemistry</i>. <b>83</b> (22): 8439–47. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1021%2Fac201416j">10.1021/ac201416j</a>. <a href="/wiki/PubMed_Central" title="PubMed Central">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3690483">3690483</a></span>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/21932784">21932784</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Analytical+Chemistry&amp;rft.atitle=Droplet-based+pyrosequencing+using+digital+microfluidics&amp;rft.volume=83&amp;rft.issue=22&amp;rft.pages=8439-47&amp;rft.date=2011-11&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3690483&amp;rft_id=info%3Apmid%2F21932784&amp;rft_id=info%3Adoi%2F10.1021%2Fac201416j&amp;rft.aulast=Boles&amp;rft.aufirst=DJ&amp;rft.au=Benton%2C+JL&amp;rft.au=Siew%2C+GJ&amp;rft.au=Levy%2C+MH&amp;rft.au=Thwar%2C+PK&amp;rft.au=Sandahl%2C+MA&amp;rft.au=Rouse%2C+JL&amp;rft.au=Perkins%2C+LC&amp;rft.au=Sudarsan%2C+AP&amp;rft.au=Jalili%2C+R&amp;rft.au=Pamula%2C+VK&amp;rft.au=Srinivasan%2C+V&amp;rft.au=Fair%2C+RB&amp;rft.au=Griffin%2C+PB&amp;rft.au=Eckhardt%2C+AE&amp;rft.au=Pollack%2C+MG&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3690483&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-117"><span class="mw-cite-backlink"><b><a href="#cite_ref-117">^</a></b></span> <span class="reference-text"><cite class="citation journal">Zilionis R, Nainys J, Veres A, Savova V, Zemmour D, Klein AM, Mazutis L (January 2017). "Single-cell barcoding and sequencing using droplet microfluidics". <i>Nature Protocols</i>. <b>12</b> (1): 44–73. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1038%2Fnprot.2016.154">10.1038/nprot.2016.154</a>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/27929523">27929523</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Nature+Protocols&amp;rft.atitle=Single-cell+barcoding+and+sequencing+using+droplet+microfluidics&amp;rft.volume=12&amp;rft.issue=1&amp;rft.pages=44-73&amp;rft.date=2017-01&amp;rft_id=info%3Adoi%2F10.1038%2Fnprot.2016.154&amp;rft_id=info%3Apmid%2F27929523&amp;rft.aulast=Zilionis&amp;rft.aufirst=R&amp;rft.au=Nainys%2C+J&amp;rft.au=Veres%2C+A&amp;rft.au=Savova%2C+V&amp;rft.au=Zemmour%2C+D&amp;rft.au=Klein%2C+AM&amp;rft.au=Mazutis%2C+L&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-118"><span class="mw-cite-backlink"><b><a href="#cite_ref-118">^</a></b></span> <span class="reference-text"><cite class="citation web"><a rel="nofollow" class="external text" href="https://web.archive.org/web/20020221002907/http://mcb.harvard.edu/branton/index.htm">"The Harvard Nanopore Group"</a>. Mcb.harvard.edu. Archived from <a rel="nofollow" class="external text" href="http://mcb.harvard.edu/branton/index.htm">the original</a> on 21 February 2002<span class="reference-accessdate">. Retrieved <span class="nowrap">15 November</span> 2009</span>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook&amp;rft.genre=unknown&amp;rft.btitle=The+Harvard+Nanopore+Group&amp;rft.pub=Mcb.harvard.edu&amp;rft_id=http%3A%2F%2Fmcb.harvard.edu%2Fbranton%2Findex.htm&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-Physorg-119"><span class="mw-cite-backlink"><b><a href="#cite_ref-Physorg_119-0">^</a></b></span> <span class="reference-text"><cite class="citation web"><a rel="nofollow" class="external text" href="http://www.physorg.com/news157378086.html">"Nanopore Sequencing Could Slash DNA Analysis Costs"</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook&amp;rft.genre=unknown&amp;rft.btitle=Nanopore+Sequencing+Could+Slash+DNA+Analysis+Costs&amp;rft_id=http%3A%2F%2Fwww.physorg.com%2Fnews157378086.html&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-120"><span class="mw-cite-backlink"><b><a href="#cite_ref-120">^</a></b></span> <span class="reference-text"><span class="citation patent" id="CITEREFZS_Genetics2005"><a rel="nofollow" class="external text" href="https://worldwide.espacenet.com/textdoc?DB=EPODOC&amp;IDX=US20060029957">US&#32;patent 20060029957</a>,&#32;ZS Genetics,&#32;"Systems and methods of analyzing nucleic acid polymers and related components",&#32;issued 2005-07-14</span><span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Apatent&amp;rft.number=20060029957&amp;rft.cc=US&amp;rft.title=Systems+and+methods+of+analyzing+nucleic+acid+polymers+and+related+components&amp;rft.inventor=ZS+Genetics&amp;rft.date=2005-07-14"><span style="display: none;">&#160;</span></span></span>
</li>
<li id="cite_note-121"><span class="mw-cite-backlink"><b><a href="#cite_ref-121">^</a></b></span> <span class="reference-text"><cite class="citation journal">Xu M, Fujita D, Hanagata N (December 2009). "Perspectives and challenges of emerging single-molecule DNA sequencing technologies". <i>Small</i>. <b>5</b> (23): 2638–49. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1002%2Fsmll.200900976">10.1002/smll.200900976</a>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/19904762">19904762</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Small&amp;rft.atitle=Perspectives+and+challenges+of+emerging+single-molecule+DNA+sequencing+technologies&amp;rft.volume=5&amp;rft.issue=23&amp;rft.pages=2638-49&amp;rft.date=2009-12&amp;rft_id=info%3Adoi%2F10.1002%2Fsmll.200900976&amp;rft_id=info%3Apmid%2F19904762&amp;rft.aulast=Xu&amp;rft.aufirst=M&amp;rft.au=Fujita%2C+D&amp;rft.au=Hanagata%2C+N&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-122"><span class="mw-cite-backlink"><b><a href="#cite_ref-122">^</a></b></span> <span class="reference-text"><cite class="citation journal">Schadt EE, Turner S, Kasarskis A (2010). "A window into third-generation sequencing". <i>Human Molecular Genetics</i>. <b>19</b> (R2): R227–40. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fhmg%2Fddq416">10.1093/hmg/ddq416</a>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/20858600">20858600</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Human+Molecular+Genetics&amp;rft.atitle=A+window+into+third-generation+sequencing&amp;rft.volume=19&amp;rft.issue=R2&amp;rft.pages=R227-40&amp;rft.date=2010&amp;rft_id=info%3Adoi%2F10.1093%2Fhmg%2Fddq416&amp;rft_id=info%3Apmid%2F20858600&amp;rft.aulast=Schadt&amp;rft.aufirst=EE&amp;rft.au=Turner%2C+S&amp;rft.au=Kasarskis%2C+A&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-123"><span class="mw-cite-backlink"><b><a href="#cite_ref-123">^</a></b></span> <span class="reference-text"><cite class="citation journal">Xu M, Endres RG, Arakawa Y (2007). "The electronic properties of DNA bases". <i>Small</i>. <b>3</b> (9): 1539–43. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1002%2Fsmll.200600732">10.1002/smll.200600732</a>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/17786897">17786897</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Small&amp;rft.atitle=The+electronic+properties+of+DNA+bases&amp;rft.volume=3&amp;rft.issue=9&amp;rft.pages=1539-43&amp;rft.date=2007&amp;rft_id=info%3Adoi%2F10.1002%2Fsmll.200600732&amp;rft_id=info%3Apmid%2F17786897&amp;rft.aulast=Xu&amp;rft.aufirst=M&amp;rft.au=Endres%2C+RG&amp;rft.au=Arakawa%2C+Y&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-124"><span class="mw-cite-backlink"><b><a href="#cite_ref-124">^</a></b></span> <span class="reference-text"><cite class="citation journal">Di Ventra M (2013). <a rel="nofollow" class="external text" href="https://zenodo.org/record/896777">"Fast DNA sequencing by electrical means inches closer"</a>. <i>Nanotechnology</i>. <b>24</b> (34): 342501. <a href="/wiki/Bibcode" title="Bibcode">Bibcode</a>:<a rel="nofollow" class="external text" href="https://ui.adsabs.harvard.edu/abs/2013Nanot..24H2501D">2013Nanot..24H2501D</a>. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1088%2F0957-4484%2F24%2F34%2F342501">10.1088/0957-4484/24/34/342501</a>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/23899780">23899780</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Nanotechnology&amp;rft.atitle=Fast+DNA+sequencing+by+electrical+means+inches+closer&amp;rft.volume=24&amp;rft.issue=34&amp;rft.pages=342501&amp;rft.date=2013&amp;rft_id=info%3Apmid%2F23899780&amp;rft_id=info%3Adoi%2F10.1088%2F0957-4484%2F24%2F34%2F342501&amp;rft_id=info%3Abibcode%2F2013Nanot..24H2501D&amp;rft.aulast=Di+Ventra&amp;rft.aufirst=M&amp;rft_id=https%3A%2F%2Fzenodo.org%2Frecord%2F896777&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-pmid22787559-125"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid22787559_125-0">^</a></b></span> <span class="reference-text"><cite class="citation journal">Ohshiro T, Matsubara K, Tsutsui M, Furuhashi M, Taniguchi M, Kawai T (2012). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3392642">"Single-molecule electrical random resequencing of DNA and RNA"</a>. <i>Sci Rep</i>. <b>2</b>: 501. <a href="/wiki/Bibcode" title="Bibcode">Bibcode</a>:<a rel="nofollow" class="external text" href="https://ui.adsabs.harvard.edu/abs/2012NatSR...2E.501O">2012NatSR...2E.501O</a>. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1038%2Fsrep00501">10.1038/srep00501</a>. <a href="/wiki/PubMed_Central" title="PubMed Central">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3392642">3392642</a></span>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/22787559">22787559</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Sci+Rep&amp;rft.atitle=Single-molecule+electrical+random+resequencing+of+DNA+and+RNA&amp;rft.volume=2&amp;rft.pages=501&amp;rft.date=2012&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3392642&amp;rft_id=info%3Apmid%2F22787559&amp;rft_id=info%3Adoi%2F10.1038%2Fsrep00501&amp;rft_id=info%3Abibcode%2F2012NatSR...2E.501O&amp;rft.aulast=Ohshiro&amp;rft.aufirst=T&amp;rft.au=Matsubara%2C+K&amp;rft.au=Tsutsui%2C+M&amp;rft.au=Furuhashi%2C+M&amp;rft.au=Taniguchi%2C+M&amp;rft.au=Kawai%2C+T&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3392642&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-126"><span class="mw-cite-backlink"><b><a href="#cite_ref-126">^</a></b></span> <span class="reference-text"><cite class="citation journal">Hanna GJ, Johnson VA, Kuritzkes DR, Richman DD, Martinez-Picado J, Sutton L, Hazelwood JD, D'Aquila RT (1 July 2000). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC87006">"Comparison of Sequencing by Hybridization and Cycle Sequencing for Genotyping of Human Immunodeficiency Virus Type 1 Reverse Transcriptase"</a>. <i>J. Clin. Microbiol</i>. <b>38</b> (7): 2715–21. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1128%2FJCM.38.7.2715-2721.2000">10.1128/JCM.38.7.2715-2721.2000</a>. <a href="/wiki/PubMed_Central" title="PubMed Central">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC87006">87006</a></span>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/10878069">10878069</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=J.+Clin.+Microbiol.&amp;rft.atitle=Comparison+of+Sequencing+by+Hybridization+and+Cycle+Sequencing+for+Genotyping+of+Human+Immunodeficiency+Virus+Type+1+Reverse+Transcriptase&amp;rft.volume=38&amp;rft.issue=7&amp;rft.pages=2715-21&amp;rft.date=2000-07-01&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC87006&amp;rft_id=info%3Apmid%2F10878069&amp;rft_id=info%3Adoi%2F10.1128%2FJCM.38.7.2715-2721.2000&amp;rft.aulast=Hanna&amp;rft.aufirst=GJ&amp;rft.au=Johnson%2C+VA&amp;rft.au=Kuritzkes%2C+DR&amp;rft.au=Richman%2C+DD&amp;rft.au=Martinez-Picado%2C+J&amp;rft.au=Sutton%2C+L&amp;rft.au=Hazelwood%2C+JD&amp;rft.au=D%27Aquila%2C+RT&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC87006&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-Morey-127"><span class="mw-cite-backlink">^ <a href="#cite_ref-Morey_127-0"><sup><i><b>a</b></i></sup></a> <a href="#cite_ref-Morey_127-1"><sup><i><b>b</b></i></sup></a></span> <span class="reference-text"><cite class="citation journal">Morey M, Fernández-Marmiesse A, Castiñeiras D, Fraga JM, Couce ML, Cocho JA (2013). "A glimpse into past, present, and future DNA sequencing". <i>Molecular Genetics and Metabolism</i>. <b>110</b> (1–2): 3–24. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1016%2Fj.ymgme.2013.04.024">10.1016/j.ymgme.2013.04.024</a>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/23742747">23742747</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Molecular+Genetics+and+Metabolism&amp;rft.atitle=A+glimpse+into+past%2C+present%2C+and+future+DNA+sequencing&amp;rft.volume=110&amp;rft.issue=1%E2%80%932&amp;rft.pages=3-24&amp;rft.date=2013&amp;rft_id=info%3Adoi%2F10.1016%2Fj.ymgme.2013.04.024&amp;rft_id=info%3Apmid%2F23742747&amp;rft.aulast=Morey&amp;rft.aufirst=M&amp;rft.au=Fern%C3%A1ndez-Marmiesse%2C+A&amp;rft.au=Casti%C3%B1eiras%2C+D&amp;rft.au=Fraga%2C+JM&amp;rft.au=Couce%2C+ML&amp;rft.au=Cocho%2C+JA&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-Qin-128"><span class="mw-cite-backlink"><b><a href="#cite_ref-Qin_128-0">^</a></b></span> <span class="reference-text"><cite class="citation journal">Qin Y, Schneider TM, Brenner MP (2012).  Gibas C (ed.). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3344849">"Sequencing by Hybridization of Long Targets"</a>. <i>PLoS ONE</i>. <b>7</b> (5): e35819. <a href="/wiki/Bibcode" title="Bibcode">Bibcode</a>:<a rel="nofollow" class="external text" href="https://ui.adsabs.harvard.edu/abs/2012PLoSO...735819Q">2012PLoSO...735819Q</a>. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1371%2Fjournal.pone.0035819">10.1371/journal.pone.0035819</a>. <a href="/wiki/PubMed_Central" title="PubMed Central">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3344849">3344849</a></span>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/22574124">22574124</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=PLoS+ONE&amp;rft.atitle=Sequencing+by+Hybridization+of+Long+Targets&amp;rft.volume=7&amp;rft.issue=5&amp;rft.pages=e35819&amp;rft.date=2012&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3344849&amp;rft_id=info%3Apmid%2F22574124&amp;rft_id=info%3Adoi%2F10.1371%2Fjournal.pone.0035819&amp;rft_id=info%3Abibcode%2F2012PLoSO...735819Q&amp;rft.aulast=Qin&amp;rft.aufirst=Y&amp;rft.au=Schneider%2C+TM&amp;rft.au=Brenner%2C+MP&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3344849&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-129"><span class="mw-cite-backlink"><b><a href="#cite_ref-129">^</a></b></span> <span class="reference-text"><cite class="citation journal">Edwards JR, Ruparel H, Ju J (2005). "Mass-spectrometry DNA sequencing". <i>Mutation Research</i>. <b>573</b> (1–2): 3–12. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1016%2Fj.mrfmmm.2004.07.021">10.1016/j.mrfmmm.2004.07.021</a>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/15829234">15829234</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Mutation+Research&amp;rft.atitle=Mass-spectrometry+DNA+sequencing&amp;rft.volume=573&amp;rft.issue=1%E2%80%932&amp;rft.pages=3-12&amp;rft.date=2005&amp;rft_id=info%3Adoi%2F10.1016%2Fj.mrfmmm.2004.07.021&amp;rft_id=info%3Apmid%2F15829234&amp;rft.aulast=Edwards&amp;rft.aufirst=JR&amp;rft.au=Ruparel%2C+H&amp;rft.au=Ju%2C+J&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-130"><span class="mw-cite-backlink"><b><a href="#cite_ref-130">^</a></b></span> <span class="reference-text"><cite class="citation journal">Hall TA, Budowle B, Jiang Y, Blyn L, Eshoo M, Sannes-Lowery KA, Sampath R, Drader JJ, Hannis JC, Harrell P, Samant V, White N, Ecker DJ, Hofstadler SA (2005). "Base composition analysis of human mitochondrial DNA using electrospray ionization mass spectrometry: A novel tool for the identification and differentiation of humans". <i>Analytical Biochemistry</i>. <b>344</b> (1): 53–69. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1016%2Fj.ab.2005.05.028">10.1016/j.ab.2005.05.028</a>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/16054106">16054106</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Analytical+Biochemistry&amp;rft.atitle=Base+composition+analysis+of+human+mitochondrial+DNA+using+electrospray+ionization+mass+spectrometry%3A+A+novel+tool+for+the+identification+and+differentiation+of+humans&amp;rft.volume=344&amp;rft.issue=1&amp;rft.pages=53-69&amp;rft.date=2005&amp;rft_id=info%3Adoi%2F10.1016%2Fj.ab.2005.05.028&amp;rft_id=info%3Apmid%2F16054106&amp;rft.aulast=Hall&amp;rft.aufirst=TA&amp;rft.au=Budowle%2C+B&amp;rft.au=Jiang%2C+Y&amp;rft.au=Blyn%2C+L&amp;rft.au=Eshoo%2C+M&amp;rft.au=Sannes-Lowery%2C+KA&amp;rft.au=Sampath%2C+R&amp;rft.au=Drader%2C+JJ&amp;rft.au=Hannis%2C+JC&amp;rft.au=Harrell%2C+P&amp;rft.au=Samant%2C+V&amp;rft.au=White%2C+N&amp;rft.au=Ecker%2C+DJ&amp;rft.au=Hofstadler%2C+SA&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-131"><span class="mw-cite-backlink"><b><a href="#cite_ref-131">^</a></b></span> <span class="reference-text"><cite class="citation journal">Howard R, Encheva V, Thomson J, Bache K, Chan YT, Cowen S, Debenham P, Dixon A, Krause JU, Krishan E, Moore D, Moore V, Ojo M, Rodrigues S, Stokes P, Walker J, Zimmermann W, Barallon R (15 June 2011). "Comparative analysis of human mitochondrial DNA from World War I bone samples by DNA sequencing and ESI-TOF mass spectrometry". <i>Forensic Science International: Genetics</i>. <b>7</b> (1): 1–9. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1016%2Fj.fsigen.2011.05.009">10.1016/j.fsigen.2011.05.009</a>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/21683667">21683667</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Forensic+Science+International%3A+Genetics&amp;rft.atitle=Comparative+analysis+of+human+mitochondrial+DNA+from+World+War+I+bone+samples+by+DNA+sequencing+and+ESI-TOF+mass+spectrometry&amp;rft.volume=7&amp;rft.issue=1&amp;rft.pages=1-9&amp;rft.date=2011-06-15&amp;rft_id=info%3Adoi%2F10.1016%2Fj.fsigen.2011.05.009&amp;rft_id=info%3Apmid%2F21683667&amp;rft.aulast=Howard&amp;rft.aufirst=R&amp;rft.au=Encheva%2C+V&amp;rft.au=Thomson%2C+J&amp;rft.au=Bache%2C+K&amp;rft.au=Chan%2C+YT&amp;rft.au=Cowen%2C+S&amp;rft.au=Debenham%2C+P&amp;rft.au=Dixon%2C+A&amp;rft.au=Krause%2C+JU&amp;rft.au=Krishan%2C+E&amp;rft.au=Moore%2C+D&amp;rft.au=Moore%2C+V&amp;rft.au=Ojo%2C+M&amp;rft.au=Rodrigues%2C+S&amp;rft.au=Stokes%2C+P&amp;rft.au=Walker%2C+J&amp;rft.au=Zimmermann%2C+W&amp;rft.au=Barallon%2C+R&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-132"><span class="mw-cite-backlink"><b><a href="#cite_ref-132">^</a></b></span> <span class="reference-text"><cite class="citation journal">Monforte JA, Becker CH (1 March 1997). "High-throughput DNA analysis by time-of-flight mass spectrometry". <i>Nature Medicine</i>. <b>3</b> (3): 360–62. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1038%2Fnm0397-360">10.1038/nm0397-360</a>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/9055869">9055869</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Nature+Medicine&amp;rft.atitle=High-throughput+DNA+analysis+by+time-of-flight+mass+spectrometry&amp;rft.volume=3&amp;rft.issue=3&amp;rft.pages=360-62&amp;rft.date=1997-03-01&amp;rft_id=info%3Adoi%2F10.1038%2Fnm0397-360&amp;rft_id=info%3Apmid%2F9055869&amp;rft.aulast=Monforte&amp;rft.aufirst=JA&amp;rft.au=Becker%2C+CH&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-133"><span class="mw-cite-backlink"><b><a href="#cite_ref-133">^</a></b></span> <span class="reference-text"><cite class="citation journal">Beres SB, Carroll RK, Shea PR, Sitkiewicz I, Martinez-Gutierrez JC, Low DE, McGeer A, Willey BM, Green K, Tyrrell GJ, Goldman TD, Feldgarden M, Birren BW, Fofanov Y, Boos J, Wheaton WD, Honisch C, Musser JM (8 February 2010). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC2840111">"Molecular complexity of successive bacterial epidemics deconvoluted by comparative pathogenomics"</a>. <i>Proceedings of the National Academy of Sciences</i>. <b>107</b> (9): 4371–76. <a href="/wiki/Bibcode" title="Bibcode">Bibcode</a>:<a rel="nofollow" class="external text" href="https://ui.adsabs.harvard.edu/abs/2010PNAS..107.4371B">2010PNAS..107.4371B</a>. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1073%2Fpnas.0911295107">10.1073/pnas.0911295107</a>. <a href="/wiki/PubMed_Central" title="PubMed Central">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC2840111">2840111</a></span>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/20142485">20142485</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Proceedings+of+the+National+Academy+of+Sciences&amp;rft.atitle=Molecular+complexity+of+successive+bacterial+epidemics+deconvoluted+by+comparative+pathogenomics&amp;rft.volume=107&amp;rft.issue=9&amp;rft.pages=4371-76&amp;rft.date=2010-02-08&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2840111&amp;rft_id=info%3Apmid%2F20142485&amp;rft_id=info%3Adoi%2F10.1073%2Fpnas.0911295107&amp;rft_id=info%3Abibcode%2F2010PNAS..107.4371B&amp;rft.aulast=Beres&amp;rft.aufirst=SB&amp;rft.au=Carroll%2C+RK&amp;rft.au=Shea%2C+PR&amp;rft.au=Sitkiewicz%2C+I&amp;rft.au=Martinez-Gutierrez%2C+JC&amp;rft.au=Low%2C+DE&amp;rft.au=McGeer%2C+A&amp;rft.au=Willey%2C+BM&amp;rft.au=Green%2C+K&amp;rft.au=Tyrrell%2C+GJ&amp;rft.au=Goldman%2C+TD&amp;rft.au=Feldgarden%2C+M&amp;rft.au=Birren%2C+BW&amp;rft.au=Fofanov%2C+Y&amp;rft.au=Boos%2C+J&amp;rft.au=Wheaton%2C+WD&amp;rft.au=Honisch%2C+C&amp;rft.au=Musser%2C+JM&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2840111&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-134"><span class="mw-cite-backlink"><b><a href="#cite_ref-134">^</a></b></span> <span class="reference-text"><cite class="citation journal">Kan CW, Fredlake CP, Doherty EA, Barron AE (1 November 2004). "DNA sequencing and genotyping in miniaturized electrophoresis systems". <i>Electrophoresis</i>. <b>25</b> (21–22): 3564–88. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1002%2Felps.200406161">10.1002/elps.200406161</a>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/15565709">15565709</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Electrophoresis&amp;rft.atitle=DNA+sequencing+and+genotyping+in+miniaturized+electrophoresis+systems&amp;rft.volume=25&amp;rft.issue=21%E2%80%9322&amp;rft.pages=3564-88&amp;rft.date=2004-11-01&amp;rft_id=info%3Adoi%2F10.1002%2Felps.200406161&amp;rft_id=info%3Apmid%2F15565709&amp;rft.aulast=Kan&amp;rft.aufirst=CW&amp;rft.au=Fredlake%2C+CP&amp;rft.au=Doherty%2C+EA&amp;rft.au=Barron%2C+AE&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-135"><span class="mw-cite-backlink"><b><a href="#cite_ref-135">^</a></b></span> <span class="reference-text"><cite class="citation journal">Chen YJ, Roller EE, Huang X (2010). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC2881221">"DNA sequencing by denaturation: experimental proof of concept with an integrated fluidic device"</a>. <i>Lab on a Chip</i>. <b>10</b> (9): 1153–59. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1039%2Fb921417h">10.1039/b921417h</a>. <a href="/wiki/PubMed_Central" title="PubMed Central">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC2881221">2881221</a></span>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/20390134">20390134</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Lab+on+a+Chip&amp;rft.atitle=DNA+sequencing+by+denaturation%3A+experimental+proof+of+concept+with+an+integrated+fluidic+device&amp;rft.volume=10&amp;rft.issue=9&amp;rft.pages=1153-59&amp;rft.date=2010&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2881221&amp;rft_id=info%3Apmid%2F20390134&amp;rft_id=info%3Adoi%2F10.1039%2Fb921417h&amp;rft.aulast=Chen&amp;rft.aufirst=YJ&amp;rft.au=Roller%2C+EE&amp;rft.au=Huang%2C+X&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2881221&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-136"><span class="mw-cite-backlink"><b><a href="#cite_ref-136">^</a></b></span> <span class="reference-text"><cite class="citation journal">Bell DC, Thomas WK, Murtagh KM, Dionne CA, Graham AC, Anderson JE, Glover WR (9 October 2012). "DNA Base Identification by Electron Microscopy". <i>Microscopy and Microanalysis</i>. <b>18</b> (5): 1049–53. <a href="/wiki/Bibcode" title="Bibcode">Bibcode</a>:<a rel="nofollow" class="external text" href="https://ui.adsabs.harvard.edu/abs/2012MiMic..18.1049B">2012MiMic..18.1049B</a>. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1017%2FS1431927612012615">10.1017/S1431927612012615</a>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/23046798">23046798</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Microscopy+and+Microanalysis&amp;rft.atitle=DNA+Base+Identification+by+Electron+Microscopy&amp;rft.volume=18&amp;rft.issue=5&amp;rft.pages=1049-53&amp;rft.date=2012-10-09&amp;rft_id=info%3Apmid%2F23046798&amp;rft_id=info%3Adoi%2F10.1017%2FS1431927612012615&amp;rft_id=info%3Abibcode%2F2012MiMic..18.1049B&amp;rft.aulast=Bell&amp;rft.aufirst=DC&amp;rft.au=Thomas%2C+WK&amp;rft.au=Murtagh%2C+KM&amp;rft.au=Dionne%2C+CA&amp;rft.au=Graham%2C+AC&amp;rft.au=Anderson%2C+JE&amp;rft.au=Glover%2C+WR&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-137"><span class="mw-cite-backlink"><b><a href="#cite_ref-137">^</a></b></span> <span class="reference-text"><cite class="citation journal">Pareek CS, Smoczynski R, Tretyn A (November 2011). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3189340">"Sequencing technologies and genome sequencing"</a>. <i>Journal of Applied Genetics</i>. <b>52</b> (4): 413–35. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1007%2Fs13353-011-0057-x">10.1007/s13353-011-0057-x</a>. <a href="/wiki/PubMed_Central" title="PubMed Central">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3189340">3189340</a></span>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/21698376">21698376</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Journal+of+Applied+Genetics&amp;rft.atitle=Sequencing+technologies+and+genome+sequencing&amp;rft.volume=52&amp;rft.issue=4&amp;rft.pages=413-35&amp;rft.date=2011-11&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3189340&amp;rft_id=info%3Apmid%2F21698376&amp;rft_id=info%3Adoi%2F10.1007%2Fs13353-011-0057-x&amp;rft.aulast=Pareek&amp;rft.aufirst=CS&amp;rft.au=Smoczynski%2C+R&amp;rft.au=Tretyn%2C+A&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3189340&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-Pareek_CS-138"><span class="mw-cite-backlink"><b><a href="#cite_ref-Pareek_CS_138-0">^</a></b></span> <span class="reference-text"><cite class="citation journal">Pareek CS, Smoczynski R, Tretyn A (2011). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3189340">"Sequencing technologies and genome sequencing"</a>. <i>Journal of Applied Genetics</i>. <b>52</b> (4): 413–35. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1007%2Fs13353-011-0057-x">10.1007/s13353-011-0057-x</a>. <a href="/wiki/PubMed_Central" title="PubMed Central">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3189340">3189340</a></span>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/21698376">21698376</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Journal+of+Applied+Genetics&amp;rft.atitle=Sequencing+technologies+and+genome+sequencing&amp;rft.volume=52&amp;rft.issue=4&amp;rft.pages=413-35&amp;rft.date=2011&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3189340&amp;rft_id=info%3Apmid%2F21698376&amp;rft_id=info%3Adoi%2F10.1007%2Fs13353-011-0057-x&amp;rft.aulast=Pareek&amp;rft.aufirst=CS&amp;rft.au=Smoczynski%2C+R&amp;rft.au=Tretyn%2C+A&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3189340&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-139"><span class="mw-cite-backlink"><b><a href="#cite_ref-139">^</a></b></span> <span class="reference-text"><cite class="citation journal">Fujimori S, Hirai N, Ohashi H, Masuoka K, Nishikimi A, Fukui Y, Washio T, Oshikubo T, Yamashita T, Miyamoto-Sato E (2012). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3466446">"Next-generation sequencing coupled with a cell-free display technology for high-throughput production of reliable interactome data"</a>. <i>Scientific Reports</i>. <b>2</b>: 691. <a href="/wiki/Bibcode" title="Bibcode">Bibcode</a>:<a rel="nofollow" class="external text" href="https://ui.adsabs.harvard.edu/abs/2012NatSR...2E.691F">2012NatSR...2E.691F</a>. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1038%2Fsrep00691">10.1038/srep00691</a>. <a href="/wiki/PubMed_Central" title="PubMed Central">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3466446">3466446</a></span>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/23056904">23056904</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Scientific+Reports&amp;rft.atitle=Next-generation+sequencing+coupled+with+a+cell-free+display+technology+for+high-throughput+production+of+reliable+interactome+data&amp;rft.volume=2&amp;rft.pages=691&amp;rft.date=2012&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3466446&amp;rft_id=info%3Apmid%2F23056904&amp;rft_id=info%3Adoi%2F10.1038%2Fsrep00691&amp;rft_id=info%3Abibcode%2F2012NatSR...2E.691F&amp;rft.aulast=Fujimori&amp;rft.aufirst=S&amp;rft.au=Hirai%2C+N&amp;rft.au=Ohashi%2C+H&amp;rft.au=Masuoka%2C+K&amp;rft.au=Nishikimi%2C+A&amp;rft.au=Fukui%2C+Y&amp;rft.au=Washio%2C+T&amp;rft.au=Oshikubo%2C+T&amp;rft.au=Yamashita%2C+T&amp;rft.au=Miyamoto-Sato%2C+E&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3466446&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-140"><span class="mw-cite-backlink"><b><a href="#cite_ref-140">^</a></b></span> <span class="reference-text"><cite class="citation journal">Harbers M (2008). "The Current Status of cDNA Cloning". <i>Genomics</i>. <b>91</b> (3): 232–42. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1016%2Fj.ygeno.2007.11.004">10.1016/j.ygeno.2007.11.004</a>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/18222633">18222633</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Genomics&amp;rft.atitle=The+Current+Status+of+cDNA+Cloning&amp;rft.volume=91&amp;rft.issue=3&amp;rft.pages=232-42&amp;rft.date=2008&amp;rft_id=info%3Adoi%2F10.1016%2Fj.ygeno.2007.11.004&amp;rft_id=info%3Apmid%2F18222633&amp;rft.aulast=Harbers&amp;rft.aufirst=M&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-141"><span class="mw-cite-backlink"><b><a href="#cite_ref-141">^</a></b></span> <span class="reference-text"><cite class="citation journal">Alberti A, Belser C, Engelen S, Bertrand L, Orvain C, Brinas L, Cruaud C,  et al. (2014). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC4213505">"Comparison of Library Preparation Methods Reveals Their Impact on Interpretation of Metatranscriptomic Data"</a>. <i>BMC Genomics</i>. <b>15</b>: 912–12. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1186%2F1471-2164-15-912">10.1186/1471-2164-15-912</a>. <a href="/wiki/PubMed_Central" title="PubMed Central">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC4213505">4213505</a></span>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/25331572">25331572</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=BMC+Genomics&amp;rft.atitle=Comparison+of+Library+Preparation+Methods+Reveals+Their+Impact+on+Interpretation+of+Metatranscriptomic+Data&amp;rft.volume=15&amp;rft.pages=912-12&amp;rft.date=2014&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4213505&amp;rft_id=info%3Apmid%2F25331572&amp;rft_id=info%3Adoi%2F10.1186%2F1471-2164-15-912&amp;rft.aulast=Alberti&amp;rft.aufirst=A&amp;rft.au=Belser%2C+C&amp;rft.au=Engelen%2C+S&amp;rft.au=Bertrand%2C+L&amp;rft.au=Orvain%2C+C&amp;rft.au=Brinas%2C+L&amp;rft.au=Cruaud%2C+C&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4213505&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-142"><span class="mw-cite-backlink"><b><a href="#cite_ref-142">^</a></b></span> <span class="reference-text"><cite class="citation web"><a rel="nofollow" class="external text" href="https://www.illumina.com/content/dam/illumina-marketing/documents/products/appnotes/library-qc-fragment-analyzer-application-note-770-2017-002.pdf">"Scalable Nucleic Acid Quality Assessments for Illumina Next-Generation Sequencing Library Prep"</a> <span class="cs1-format">(PDF)</span><span class="reference-accessdate">. Retrieved <span class="nowrap">27 December</span> 2017</span>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook&amp;rft.genre=unknown&amp;rft.btitle=Scalable+Nucleic+Acid+Quality+Assessments+for+Illumina+Next-Generation+Sequencing+Library+Prep&amp;rft_id=https%3A%2F%2Fwww.illumina.com%2Fcontent%2Fdam%2Fillumina-marketing%2Fdocuments%2Fproducts%2Fappnotes%2Flibrary-qc-fragment-analyzer-application-note-770-2017-002.pdf&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-143"><span class="mw-cite-backlink"><b><a href="#cite_ref-143">^</a></b></span> <span class="reference-text"><cite class="citation web"><a rel="nofollow" class="external text" href="http://genomics.xprize.org/">"Archon Genomics XPRIZE"</a>. <i>Archon Genomics XPRIZE</i>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=unknown&amp;rft.jtitle=Archon+Genomics+XPRIZE&amp;rft.atitle=Archon+Genomics+XPRIZE&amp;rft_id=http%3A%2F%2Fgenomics.xprize.org%2F&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-144"><span class="mw-cite-backlink"><b><a href="#cite_ref-144">^</a></b></span> <span class="reference-text"><cite class="citation web"><a rel="nofollow" class="external text" href="http://www.genome.gov/10000004">"Grant Information"</a>. <i>National Human Genome Research Institute (NHGRI)</i>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=unknown&amp;rft.jtitle=National+Human+Genome+Research+Institute+%28NHGRI%29&amp;rft.atitle=Grant+Information&amp;rft_id=http%3A%2F%2Fwww.genome.gov%2F10000004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-pmid24727769-145"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid24727769_145-0">^</a></b></span> <span class="reference-text"><cite class="citation journal">Severin J, Lizio M, Harshbarger J, Kawaji H, Daub CO, Hayashizaki Y, Bertin N, Forrest AR (2014). "Interactive visualization and analysis of large-scale sequencing datasets using ZENBU". <i>Nat. Biotechnol</i>. <b>32</b> (3): 217–19. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1038%2Fnbt.2840">10.1038/nbt.2840</a>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/24727769">24727769</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Nat.+Biotechnol.&amp;rft.atitle=Interactive+visualization+and+analysis+of+large-scale+sequencing+datasets+using+ZENBU&amp;rft.volume=32&amp;rft.issue=3&amp;rft.pages=217-19&amp;rft.date=2014&amp;rft_id=info%3Adoi%2F10.1038%2Fnbt.2840&amp;rft_id=info%3Apmid%2F24727769&amp;rft.aulast=Severin&amp;rft.aufirst=J&amp;rft.au=Lizio%2C+M&amp;rft.au=Harshbarger%2C+J&amp;rft.au=Kawaji%2C+H&amp;rft.au=Daub%2C+CO&amp;rft.au=Hayashizaki%2C+Y&amp;rft.au=Bertin%2C+N&amp;rft.au=Forrest%2C+AR&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-146"><span class="mw-cite-backlink"><b><a href="#cite_ref-146">^</a></b></span> <span class="reference-text"><cite class="citation journal">Shmilovici A, Ben-Gal I (2007). <a rel="nofollow" class="external text" href="http://www.eng.tau.ac.il/~bengal/VOM_EST.pdf">"Using a VOM model for reconstructing potential coding regions in EST sequences"</a> <span class="cs1-format">(PDF)</span>. <i>Computational Statistics</i>. <b>22</b> (1): 49–69. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1007%2Fs00180-007-0021-8">10.1007/s00180-007-0021-8</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Computational+Statistics&amp;rft.atitle=Using+a+VOM+model+for+reconstructing+potential+coding+regions+in+EST+sequences&amp;rft.volume=22&amp;rft.issue=1&amp;rft.pages=49-69&amp;rft.date=2007&amp;rft_id=info%3Adoi%2F10.1007%2Fs00180-007-0021-8&amp;rft.aulast=Shmilovici&amp;rft.aufirst=A&amp;rft.au=Ben-Gal%2C+I&amp;rft_id=http%3A%2F%2Fwww.eng.tau.ac.il%2F~bengal%2FVOM_EST.pdf&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-147"><span class="mw-cite-backlink"><b><a href="#cite_ref-147">^</a></b></span> <span class="reference-text"><cite class="citation journal">Del Fabbro C, Scalabrin S, Morgante M, Giorgi FM (2013). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3871669">"An Extensive Evaluation of Read Trimming Effects on Illumina NGS Data Analysis"</a>. <i>PLoS ONE</i>. <b>8</b> (12): e85024. <a href="/wiki/Bibcode" title="Bibcode">Bibcode</a>:<a rel="nofollow" class="external text" href="https://ui.adsabs.harvard.edu/abs/2013PLoSO...885024D">2013PLoSO...885024D</a>. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1371%2Fjournal.pone.0085024">10.1371/journal.pone.0085024</a>. <a href="/wiki/PubMed_Central" title="PubMed Central">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3871669">3871669</a></span>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/24376861">24376861</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=PLoS+ONE&amp;rft.atitle=An+Extensive+Evaluation+of+Read+Trimming+Effects+on+Illumina+NGS+Data+Analysis&amp;rft.volume=8&amp;rft.issue=12&amp;rft.pages=e85024&amp;rft.date=2013&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3871669&amp;rft_id=info%3Apmid%2F24376861&amp;rft_id=info%3Adoi%2F10.1371%2Fjournal.pone.0085024&amp;rft_id=info%3Abibcode%2F2013PLoSO...885024D&amp;rft.aulast=Del+Fabbro&amp;rft.aufirst=C&amp;rft.au=Scalabrin%2C+S&amp;rft.au=Morgante%2C+M&amp;rft.au=Giorgi%2C+FM&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3871669&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-cutadapt-148"><span class="mw-cite-backlink"><b><a href="#cite_ref-cutadapt_148-0">^</a></b></span> <span class="reference-text"><cite class="citation journal">Martin, Marcel (2 May 2011). "Cutadapt removes adapter sequences from high-throughput sequencing reads". <i>EMBnet.journal</i>. <b>17</b> (1): 10. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.14806%2Fej.17.1.200">10.14806/ej.17.1.200</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=EMBnet.journal&amp;rft.atitle=Cutadapt+removes+adapter+sequences+from+high-throughput+sequencing+reads&amp;rft.volume=17&amp;rft.issue=1&amp;rft.pages=10&amp;rft.date=2011-05-02&amp;rft_id=info%3Adoi%2F10.14806%2Fej.17.1.200&amp;rft.aulast=Martin&amp;rft.aufirst=Marcel&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-condetri-149"><span class="mw-cite-backlink"><b><a href="#cite_ref-condetri_149-0">^</a></b></span> <span class="reference-text"><cite class="citation journal">Smeds L, Künstner A (19 October 2011). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3198461">"ConDeTri--a content dependent read trimmer for Illumina data"</a>. <i>PLOS ONE</i>. <b>6</b> (10): e26314. <a href="/wiki/Bibcode" title="Bibcode">Bibcode</a>:<a rel="nofollow" class="external text" href="https://ui.adsabs.harvard.edu/abs/2011PLoSO...626314S">2011PLoSO...626314S</a>. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1371%2Fjournal.pone.0026314">10.1371/journal.pone.0026314</a>. <a href="/wiki/PubMed_Central" title="PubMed Central">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3198461">3198461</a></span>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/22039460">22039460</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=PLOS+ONE&amp;rft.atitle=ConDeTri--a+content+dependent+read+trimmer+for+Illumina+data&amp;rft.volume=6&amp;rft.issue=10&amp;rft.pages=e26314&amp;rft.date=2011-10-19&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3198461&amp;rft_id=info%3Apmid%2F22039460&amp;rft_id=info%3Adoi%2F10.1371%2Fjournal.pone.0026314&amp;rft_id=info%3Abibcode%2F2011PLoSO...626314S&amp;rft.aulast=Smeds&amp;rft.aufirst=L&amp;rft.au=K%C3%BCnstner%2C+A&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3198461&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-erne-bs5-150"><span class="mw-cite-backlink"><b><a href="#cite_ref-erne-bs5_150-0">^</a></b></span> <span class="reference-text"><cite class="citation book">Prezza N, Del Fabbro C, Vezzi F, De Paoli E, Policriti A (2012). <i>ERNE-BS5: Aligning BS-treated Sequences by Multiple Hits on a 5-letters Alphabet</i>. <i>Proceedings of the ACM Conference on Bioinformatics, Computational Biology and Biomedicine</i>. <b>12</b>. pp.&#160;12–19. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1145%2F2382936.2382938">10.1145/2382936.2382938</a>. <a href="/wiki/International_Standard_Book_Number" title="International Standard Book Number">ISBN</a>&#160;<a href="/wiki/Special:BookSources/9781450316705" title="Special:BookSources/9781450316705"><bdi>9781450316705</bdi></a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook&amp;rft.genre=book&amp;rft.btitle=ERNE-BS5%3A+Aligning+BS-treated+Sequences+by+Multiple+Hits+on+a+5-letters+Alphabet&amp;rft.pages=12-19&amp;rft.date=2012&amp;rft_id=info%3Adoi%2F10.1145%2F2382936.2382938&amp;rft.isbn=9781450316705&amp;rft.aulast=Prezza&amp;rft.aufirst=N&amp;rft.au=Del+Fabbro%2C+C&amp;rft.au=Vezzi%2C+F&amp;rft.au=De+Paoli%2C+E&amp;rft.au=Policriti%2C+A&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-prinseq-151"><span class="mw-cite-backlink"><b><a href="#cite_ref-prinseq_151-0">^</a></b></span> <span class="reference-text"><cite class="citation journal">Schmieder R, Edwards R (March 2011). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3051327">"Quality control and preprocessing of metagenomic datasets"</a>. <i>Bioinformatics</i>. <b>27</b> (6): 863–4. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fbioinformatics%2Fbtr026">10.1093/bioinformatics/btr026</a>. <a href="/wiki/PubMed_Central" title="PubMed Central">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3051327">3051327</a></span>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/21278185">21278185</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Bioinformatics&amp;rft.atitle=Quality+control+and+preprocessing+of+metagenomic+datasets&amp;rft.volume=27&amp;rft.issue=6&amp;rft.pages=863-4&amp;rft.date=2011-03&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3051327&amp;rft_id=info%3Apmid%2F21278185&amp;rft_id=info%3Adoi%2F10.1093%2Fbioinformatics%2Fbtr026&amp;rft.aulast=Schmieder&amp;rft.aufirst=R&amp;rft.au=Edwards%2C+R&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3051327&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-trimmomatic-152"><span class="mw-cite-backlink"><b><a href="#cite_ref-trimmomatic_152-0">^</a></b></span> <span class="reference-text"><cite class="citation journal">Bolger AM, Lohse M, Usadel B (August 2014). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC4103590">"Trimmomatic: a flexible trimmer for Illumina sequence data"</a>. <i>Bioinformatics</i>. <b>30</b> (15): 2114–20. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fbioinformatics%2Fbtu170">10.1093/bioinformatics/btu170</a>. <a href="/wiki/PubMed_Central" title="PubMed Central">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC4103590">4103590</a></span>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/24695404">24695404</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Bioinformatics&amp;rft.atitle=Trimmomatic%3A+a+flexible+trimmer+for+Illumina+sequence+data&amp;rft.volume=30&amp;rft.issue=15&amp;rft.pages=2114-20&amp;rft.date=2014-08&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4103590&amp;rft_id=info%3Apmid%2F24695404&amp;rft_id=info%3Adoi%2F10.1093%2Fbioinformatics%2Fbtu170&amp;rft.aulast=Bolger&amp;rft.aufirst=AM&amp;rft.au=Lohse%2C+M&amp;rft.au=Usadel%2C+B&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4103590&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-solexaqa-153"><span class="mw-cite-backlink"><b><a href="#cite_ref-solexaqa_153-0">^</a></b></span> <span class="reference-text"><cite class="citation journal">Cox MP, Peterson DA, Biggs PJ (September 2010). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC2956736">"SolexaQA: At-a-glance quality assessment of Illumina second-generation sequencing data"</a>. <i>BMC Bioinformatics</i>. <b>11</b> (1): 485. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1186%2F1471-2105-11-485">10.1186/1471-2105-11-485</a>. <a href="/wiki/PubMed_Central" title="PubMed Central">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC2956736">2956736</a></span>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/20875133">20875133</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=BMC+Bioinformatics&amp;rft.atitle=SolexaQA%3A+At-a-glance+quality+assessment+of+Illumina+second-generation+sequencing+data&amp;rft.volume=11&amp;rft.issue=1&amp;rft.pages=485&amp;rft.date=2010-09&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2956736&amp;rft_id=info%3Apmid%2F20875133&amp;rft_id=info%3Adoi%2F10.1186%2F1471-2105-11-485&amp;rft.aulast=Cox&amp;rft.aufirst=MP&amp;rft.au=Peterson%2C+DA&amp;rft.au=Biggs%2C+PJ&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2956736&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-ethics-murray-154"><span class="mw-cite-backlink"><b><a href="#cite_ref-ethics-murray_154-0">^</a></b></span> <span class="reference-text"><cite class="citation journal">Murray TH (January 1991). "Ethical issues in human genome research". <i>FASEB Journal</i>. <b>5</b> (1): 55–60. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1096%2Ffasebj.5.1.1825074">10.1096/fasebj.5.1.1825074</a>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/1825074">1825074</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=FASEB+Journal&amp;rft.atitle=Ethical+issues+in+human+genome+research&amp;rft.volume=5&amp;rft.issue=1&amp;rft.pages=55-60&amp;rft.date=1991-01&amp;rft_id=info%3Adoi%2F10.1096%2Ffasebj.5.1.1825074&amp;rft_id=info%3Apmid%2F1825074&amp;rft.aulast=Murray&amp;rft.aufirst=TH&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-usd1000-genome-ethics-155"><span class="mw-cite-backlink">^ <a href="#cite_ref-usd1000-genome-ethics_155-0"><sup><i><b>a</b></i></sup></a> <a href="#cite_ref-usd1000-genome-ethics_155-1"><sup><i><b>b</b></i></sup></a> <a href="#cite_ref-usd1000-genome-ethics_155-2"><sup><i><b>c</b></i></sup></a></span> <span class="reference-text"><cite class="citation journal">Robertson JA (August 2003). "The $1000 genome: ethical and legal issues in whole genome sequencing of individuals". <i>The American Journal of Bioethics</i>. <b>3</b> (3): W–IF1. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1162%2F152651603322874762">10.1162/152651603322874762</a>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/14735880">14735880</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=The+American+Journal+of+Bioethics&amp;rft.atitle=The+%241000+genome%3A+ethical+and+legal+issues+in+whole+genome+sequencing+of+individuals&amp;rft.volume=3&amp;rft.issue=3&amp;rft.pages=W-IF1&amp;rft.date=2003-08&amp;rft_id=info%3Adoi%2F10.1162%2F152651603322874762&amp;rft_id=info%3Apmid%2F14735880&amp;rft.aulast=Robertson&amp;rft.aufirst=JA&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-guardian-ethics-156"><span class="mw-cite-backlink">^ <a href="#cite_ref-guardian-ethics_156-0"><sup><i><b>a</b></i></sup></a> <a href="#cite_ref-guardian-ethics_156-1"><sup><i><b>b</b></i></sup></a></span> <span class="reference-text"><cite class="citation news">Henderson, Mark (9 September 2013). <a rel="nofollow" class="external text" href="https://www.theguardian.com/science/2013/sep/09/genetics-ethics-human-gene-sequencing">"Human genome sequencing: the real ethical dilemmas"</a>. <i>The Guardian</i><span class="reference-accessdate">. Retrieved <span class="nowrap">20 May</span> 2015</span>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=The+Guardian&amp;rft.atitle=Human+genome+sequencing%3A+the+real+ethical+dilemmas&amp;rft.date=2013-09-09&amp;rft.aulast=Henderson&amp;rft.aufirst=Mark&amp;rft_id=https%3A%2F%2Fwww.theguardian.com%2Fscience%2F2013%2Fsep%2F09%2Fgenetics-ethics-human-gene-sequencing&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-ny-times-insurance-ethics-157"><span class="mw-cite-backlink"><b><a href="#cite_ref-ny-times-insurance-ethics_157-0">^</a></b></span> <span class="reference-text"><cite class="citation news">Harmon, Amy (24 February 2008). <a rel="nofollow" class="external text" href="https://www.nytimes.com/2008/02/24/health/24dna.html?pagewanted=all&amp;_r=0">"Insurance Fears Lead Many to Shun DNA Tests"</a>. <i>The New York Times</i><span class="reference-accessdate">. Retrieved <span class="nowrap">20 May</span> 2015</span>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=The+New+York+Times&amp;rft.atitle=Insurance+Fears+Lead+Many+to+Shun+DNA+Tests&amp;rft.date=2008-02-24&amp;rft.aulast=Harmon&amp;rft.aufirst=Amy&amp;rft_id=https%3A%2F%2Fwww.nytimes.com%2F2008%2F02%2F24%2Fhealth%2F24dna.html%3Fpagewanted%3Dall%26_r%3D0&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-OMB_support-158"><span class="mw-cite-backlink"><b><a href="#cite_ref-OMB_support_158-0">^</a></b></span> <span class="reference-text"><a rel="nofollow" class="external text" href="http://www.genome.gov/Pages/PolicyEthics/GeneticDiscrimination/SAPonHR493.pdf">Statement of Administration policy</a>, Executive Office of the President, Office of Management and Budget, 27 April 2007</span>
</li>
<li id="cite_note-Signing-159"><span class="mw-cite-backlink"><b><a href="#cite_ref-Signing_159-0">^</a></b></span> <span class="reference-text"><cite class="citation news">National Human Genome Research Institute (21 May 2008). <a rel="nofollow" class="external text" href="http://www.genome.gov/27026050">"President Bush Signs the Genetic Information Nondiscrimination Act of 2008"</a><span class="reference-accessdate">. Retrieved <span class="nowrap">17 February</span> 2014</span>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.atitle=President+Bush+Signs+the+Genetic+Information+Nondiscrimination+Act+of+2008&amp;rft.date=2008-05-21&amp;rft.au=National+Human+Genome+Research+Institute&amp;rft_id=http%3A%2F%2Fwww.genome.gov%2F27026050&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-nature-ethics-160"><span class="mw-cite-backlink"><b><a href="#cite_ref-nature-ethics_160-0">^</a></b></span> <span class="reference-text"><cite class="citation web">Baker, Monya. <a rel="nofollow" class="external text" href="http://blogs.nature.com/news/2012/10/us-ethics-panel-reports-on-dna-sequencing-and-privacy.html">"US ethics panel reports on DNA sequencing and privacy"</a>. <i>Nature New Blog</i><span class="reference-accessdate">. Retrieved <span class="nowrap">20 May</span> 2015</span>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=unknown&amp;rft.jtitle=Nature+New+Blog&amp;rft.atitle=US+ethics+panel+reports+on+DNA+sequencing+and+privacy&amp;rft.aulast=Baker&amp;rft.aufirst=Monya&amp;rft_id=http%3A%2F%2Fblogs.nature.com%2Fnews%2F2012%2F10%2Fus-ethics-panel-reports-on-dna-sequencing-and-privacy.html&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-privacy-progress-report-161"><span class="mw-cite-backlink"><b><a href="#cite_ref-privacy-progress-report_161-0">^</a></b></span> <span class="reference-text"><cite class="citation web"><a rel="nofollow" class="external text" href="https://web.archive.org/web/20150612001900/http://bioethics.gov/sites/default/files/PrivacyProgress508_1.pdf">"Privacy and Progress in Whole Genome Sequencing"</a> <span class="cs1-format">(PDF)</span>. Presidential Commission for the Study of Bioethical Issues. Archived from <a rel="nofollow" class="external text" href="http://bioethics.gov/sites/default/files/PrivacyProgress508_1.pdf">the original</a> <span class="cs1-format">(PDF)</span> on 12 June 2015<span class="reference-accessdate">. Retrieved <span class="nowrap">20 May</span> 2015</span>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook&amp;rft.genre=unknown&amp;rft.btitle=Privacy+and+Progress+in+Whole+Genome+Sequencing&amp;rft.pub=Presidential+Commission+for+the+Study+of+Bioethical+Issues&amp;rft_id=http%3A%2F%2Fbioethics.gov%2Fsites%2Fdefault%2Ffiles%2FPrivacyProgress508_1.pdf&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-ethics-newborns-162"><span class="mw-cite-backlink"><b><a href="#cite_ref-ethics-newborns_162-0">^</a></b></span> <span class="reference-text"><cite class="citation journal">Goldenberg AJ, Sharp RR (February 2012). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3868436">"The ethical hazards and programmatic challenges of genomic newborn screening"</a>. <i>JAMA</i>. <b>307</b> (5): 461–2. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1001%2Fjama.2012.68">10.1001/jama.2012.68</a>. <a href="/wiki/PubMed_Central" title="PubMed Central">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3868436">3868436</a></span>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/22298675">22298675</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=JAMA&amp;rft.atitle=The+ethical+hazards+and+programmatic+challenges+of+genomic+newborn+screening&amp;rft.volume=307&amp;rft.issue=5&amp;rft.pages=461-2&amp;rft.date=2012-02&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3868436&amp;rft_id=info%3Apmid%2F22298675&amp;rft_id=info%3Adoi%2F10.1001%2Fjama.2012.68&amp;rft.aulast=Goldenberg&amp;rft.aufirst=AJ&amp;rft.au=Sharp%2C+RR&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3868436&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-ethics-hughes-163"><span class="mw-cite-backlink"><b><a href="#cite_ref-ethics-hughes_163-0">^</a></b></span> <span class="reference-text"><cite class="citation web">Hughes, Virginia (7 January 2013). <a rel="nofollow" class="external text" href="http://www.slate.com/articles/health_and_science/medical_examiner/2013/01/ethics_of_genetic_information_whole_genome_sequencing_is_here_and_we_need.html">"It's Time To Stop Obsessing About the Dangers of Genetic Information"</a>. <i>Slate Magazine</i><span class="reference-accessdate">. Retrieved <span class="nowrap">22 May</span> 2015</span>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=unknown&amp;rft.jtitle=Slate+Magazine&amp;rft.atitle=It%27s+Time+To+Stop+Obsessing+About+the+Dangers+of+Genetic+Information&amp;rft.date=2013-01-07&amp;rft.aulast=Hughes&amp;rft.aufirst=Virginia&amp;rft_id=http%3A%2F%2Fwww.slate.com%2Farticles%2Fhealth_and_science%2Fmedical_examiner%2F2013%2F01%2Fethics_of_genetic_information_whole_genome_sequencing_is_here_and_we_need.html&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-ethics-bloss-164"><span class="mw-cite-backlink">^ <a href="#cite_ref-ethics-bloss_164-0"><sup><i><b>a</b></i></sup></a> <a href="#cite_ref-ethics-bloss_164-1"><sup><i><b>b</b></i></sup></a></span> <span class="reference-text"><cite class="citation journal">Bloss CS, Schork NJ, Topol EJ (February 2011). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3786730">"Effect of direct-to-consumer genomewide profiling to assess disease risk"</a>. <i>The New England Journal of Medicine</i>. <b>364</b> (6): 524–34. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1056%2FNEJMoa1011893">10.1056/NEJMoa1011893</a>. <a href="/wiki/PubMed_Central" title="PubMed Central">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3786730">3786730</a></span>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//pubmed.ncbi.nlm.nih.gov/21226570">21226570</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=The+New+England+Journal+of+Medicine&amp;rft.atitle=Effect+of+direct-to-consumer+genomewide+profiling+to+assess+disease+risk&amp;rft.volume=364&amp;rft.issue=6&amp;rft.pages=524-34&amp;rft.date=2011-02&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3786730&amp;rft_id=info%3Apmid%2F21226570&amp;rft_id=info%3Adoi%2F10.1056%2FNEJMoa1011893&amp;rft.aulast=Bloss&amp;rft.aufirst=CS&amp;rft.au=Schork%2C+NJ&amp;rft.au=Topol%2C+EJ&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3786730&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
<li id="cite_note-ethics-time-165"><span class="mw-cite-backlink"><b><a href="#cite_ref-ethics-time_165-0">^</a></b></span> <span class="reference-text"><cite class="citation news">Rochman, Bonnie (25 October 2012). <a rel="nofollow" class="external text" href="http://healthland.time.com/2012/10/25/what-your-doctor-isnt-telling-you-about-your-dna/">"What Your Doctor Isn't Telling You About Your DNA"</a>. <i>Time.com</i><span class="reference-accessdate">. Retrieved <span class="nowrap">22 May</span> 2015</span>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Time.com&amp;rft.atitle=What+Your+Doctor+Isn%27t+Telling+You+About+Your+DNA&amp;rft.date=2012-10-25&amp;rft.aulast=Rochman&amp;rft.aufirst=Bonnie&amp;rft_id=http%3A%2F%2Fhealthland.time.com%2F2012%2F10%2F25%2Fwhat-your-doctor-isnt-telling-you-about-your-dna%2F&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ADNA+sequencing" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r935243608"/></span>
</li>
</ol></div></div>
<h2><span class="mw-headline" id="External_links">External links</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=DNA_sequencing&amp;action=edit&amp;section=51" title="Edit section: External links">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<table class="vertical-navbox nowraplinks" style="float:right;clear:right;width:auto;margin:0 0 1.0em 1.0em;background:#f9f9f9;border:1px solid #aaa;padding:0.2em;border-spacing:0.4em 0;text-align:center;line-height:1.4em;font-size:88%;text-align:left;"><tbody><tr><td style="padding-top:0.4em;line-height:1.2em"><a href="/wiki/Wikipedia:The_Wikipedia_Library" title="Wikipedia:The Wikipedia Library">Library resources</a> about <br /> <b>DNA sequencing</b> <hr /></td></tr><tr><td class="plainlist" style="padding:0 0.1em 0.4em">
<ul><li><a class="external text" href="//tools.wmflabs.org/ftl/cgi-bin/ftl?st=wp&amp;su=DNA+sequencing">Resources in your library</a></li>
<li><a class="external text" href="//tools.wmflabs.org/ftl/cgi-bin/ftl?st=wp&amp;su=DNA+sequencing&amp;library=0CHOOSE0">Resources in other libraries</a></li></ul></td>
</tr></tbody></table>
<table role="presentation" class="mbox-small plainlinks sistersitebox" style="background-color:#f9f9f9;border:1px solid #aaa;color:#000">
<tbody><tr>
<td class="mbox-image"><img alt="" src="//upload.wikimedia.org/wikipedia/commons/thumb/d/df/Wikibooks-logo-en-noslogan.svg/40px-Wikibooks-logo-en-noslogan.svg.png" decoding="async" width="40" height="40" class="noviewer" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/d/df/Wikibooks-logo-en-noslogan.svg/60px-Wikibooks-logo-en-noslogan.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/d/df/Wikibooks-logo-en-noslogan.svg/80px-Wikibooks-logo-en-noslogan.svg.png 2x" data-file-width="400" data-file-height="400" /></td>
<td class="mbox-text plainlist">Wikibooks has a book on the topic of: <i><b><a href="https://en.wikibooks.org/wiki/Next_Generation_Sequencing_(NGS)" class="extiw" title="wikibooks:Next Generation Sequencing (NGS)">Next Generation Sequencing (NGS)</a></b></i></td></tr>
</tbody></table>
<ul><li>A <a href="https://en.wikibooks.org/wiki/Next_Generation_Sequencing_(NGS)" class="extiw" title="wikibooks:Next Generation Sequencing (NGS)">wikibook on next generation sequencing</a></li>
<li>A <a rel="nofollow" class="external text" href="http://omicx.com/">free didactic directory for DNA sequencing analysis.</a></li>
<li>A <a rel="nofollow" class="external text" href="http://www.whatisbiotechnology.org/exhibitions/sanger">The path to DNA sequencing: The life and work of Fred Sanger</a></li>
<li>A <a rel="nofollow" class="external text" href="https://map.meenta.io/">US Map of New Generation Sequencers</a></li></ul>
<div class="noprint metadata navbox" role="navigation" aria-label="Portals" style="font-weight:bold;padding:0.4em 2em"><ul style="margin:0.1em 0 0"><li style="display:inline"><span style="display:inline-block;white-space:nowrap"><span style="margin:0 0.5em"><a href="/wiki/File:Issoria_lathonia.jpg" class="image"><img alt="Issoria lathonia.jpg" src="//upload.wikimedia.org/wikipedia/commons/thumb/2/2d/Issoria_lathonia.jpg/24px-Issoria_lathonia.jpg" decoding="async" width="24" height="17" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/2/2d/Issoria_lathonia.jpg/36px-Issoria_lathonia.jpg 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/2/2d/Issoria_lathonia.jpg/48px-Issoria_lathonia.jpg 2x" data-file-width="629" data-file-height="445" /></a></span><a href="/wiki/Portal:Biology" title="Portal:Biology">Biology portal</a></span></li></ul></div>
<!-- 
NewPP limit report
Parsed by mw1262
Cached time: 20200326171837
Cache expiry: 2592000
Dynamic content: false
Complications: [vary‐revision‐sha1]
CPU time usage: 2.060 seconds
Real time usage: 2.385 seconds
Preprocessor visited node count: 8747/1000000
Post‐expand include size: 372178/2097152 bytes
Template argument size: 5055/2097152 bytes
Highest expansion depth: 14/40
Expensive parser function count: 14/500
Unstrip recursion depth: 1/20
Unstrip post‐expand size: 558627/5000000 bytes
Number of Wikibase entities loaded: 7/400
Lua time usage: 1.293/10.000 seconds
Lua memory usage: 8.66 MB/50 MB
Lua Profile:
    Scribunto_LuaSandboxCallback::find                               260 ms       19.1%
    Scribunto_LuaSandboxCallback::match                              160 ms       11.8%
    recursiveClone <mwInit.lua:41>                                   140 ms       10.3%
    ?                                                                140 ms       10.3%
    Scribunto_LuaSandboxCallback::getEntity                          120 ms        8.8%
    Scribunto_LuaSandboxCallback::getAllExpandedArguments            100 ms        7.4%
    Scribunto_LuaSandboxCallback::callParserFunction                 100 ms        7.4%
    type                                                              60 ms        4.4%
    <mw.lua:683>                                                      60 ms        4.4%
    maskEntityTable <mw.wikibase.entity.lua:54>                       40 ms        2.9%
    [others]                                                         180 ms       13.2%
-->
<!--
Transclusion expansion time report (%,ms,calls,template)
100.00% 2099.731      1 -total
 78.13% 1640.580      2 Template:Reflist
 53.81% 1129.874    113 Template:Cite_journal
  5.75%  120.797     28 Template:Cite_web
  3.27%   68.603      5 Template:Fix
  3.10%   64.992      4 Template:Citation_needed
  2.19%   46.064      1 Template:Genetics_sidebar
  2.02%   42.426      1 Template:Short_description
  1.95%   41.033      1 Template:Sidebar
  1.78%   37.314     10 Template:Category_handler
-->

<!-- Saved in parser cache with key enwiki:pcache:idhash:1158125-0!canonical and timestamp 20200326171834 and revision id 945603030
 -->
</div><noscript><img src="//en.wikipedia.org/wiki/Special:CentralAutoLogin/start?type=1x1" alt="" title="" width="1" height="1" style="border: none; position: absolute;" /></noscript></div>
		<div class="printfooter">Retrieved from "<a dir="ltr" href="https://en.wikipedia.org/w/index.php?title=DNA_sequencing&amp;oldid=945603030">https://en.wikipedia.org/w/index.php?title=DNA_sequencing&amp;oldid=945603030</a>"</div>
		<div id="catlinks" class="catlinks" data-mw="interface"><div id="mw-normal-catlinks" class="mw-normal-catlinks"><a href="/wiki/Help:Category" title="Help:Category">Categories</a>: <ul><li><a href="/wiki/Category:DNA_sequencing" title="Category:DNA sequencing">DNA sequencing</a></li><li><a href="/wiki/Category:Biotechnology" title="Category:Biotechnology">Biotechnology</a></li><li><a href="/wiki/Category:DNA" title="Category:DNA">DNA</a></li><li><a href="/wiki/Category:Genetic_mapping" title="Category:Genetic mapping">Genetic mapping</a></li><li><a href="/wiki/Category:Molecular_biology" title="Category:Molecular biology">Molecular biology</a></li><li><a href="/wiki/Category:Molecular_biology_techniques" title="Category:Molecular biology techniques">Molecular biology techniques</a></li><li><a href="/wiki/Category:1970_introductions" title="Category:1970 introductions">1970 introductions</a></li><li><a href="/wiki/Category:1970_in_biology" title="Category:1970 in biology">1970 in biology</a></li><li><a href="/wiki/Category:1970_in_biotechnology" title="Category:1970 in biotechnology">1970 in biotechnology</a></li><li><a href="/wiki/Category:1970_in_science" title="Category:1970 in science">1970 in science</a></li><li><a href="/wiki/Category:1998_in_technology" title="Category:1998 in technology">1998 in technology</a></li></ul></div><div id="mw-hidden-catlinks" class="mw-hidden-catlinks mw-hidden-cats-hidden">Hidden categories: <ul><li><a href="/wiki/Category:CS1_errors:_missing_periodical" title="Category:CS1 errors: missing periodical">CS1 errors: missing periodical</a></li><li><a href="/wiki/Category:Pages_containing_links_to_subscription-only_content" title="Category:Pages containing links to subscription-only content">Pages containing links to subscription-only content</a></li><li><a href="/wiki/Category:Wikipedia_articles_needing_page_number_citations_from_June_2013" title="Category:Wikipedia articles needing page number citations from June 2013">Wikipedia articles needing page number citations from June 2013</a></li><li><a href="/wiki/Category:Pages_with_login_required_references_or_sources" title="Category:Pages with login required references or sources">Pages with login required references or sources</a></li><li><a href="/wiki/Category:CS1_maint:_archived_copy_as_title" title="Category:CS1 maint: archived copy as title">CS1 maint: archived copy as title</a></li><li><a href="/wiki/Category:CS1:_long_volume_value" title="Category:CS1: long volume value">CS1: long volume value</a></li><li><a href="/wiki/Category:Wikipedia_indefinitely_move-protected_pages" title="Category:Wikipedia indefinitely move-protected pages">Wikipedia indefinitely move-protected pages</a></li><li><a href="/wiki/Category:Articles_with_short_description" title="Category:Articles with short description">Articles with short description</a></li><li><a href="/wiki/Category:Use_dmy_dates_from_April_2011" title="Category:Use dmy dates from April 2011">Use dmy dates from April 2011</a></li><li><a href="/wiki/Category:Articles_containing_potentially_dated_statements_from_2019" title="Category:Articles containing potentially dated statements from 2019">Articles containing potentially dated statements from 2019</a></li><li><a href="/wiki/Category:All_articles_containing_potentially_dated_statements" title="Category:All articles containing potentially dated statements">All articles containing potentially dated statements</a></li><li><a href="/wiki/Category:All_articles_with_unsourced_statements" title="Category:All articles with unsourced statements">All articles with unsourced statements</a></li><li><a href="/wiki/Category:Articles_with_unsourced_statements_from_July_2018" title="Category:Articles with unsourced statements from July 2018">Articles with unsourced statements from July 2018</a></li><li><a href="/wiki/Category:Articles_to_be_expanded_from_May_2015" title="Category:Articles to be expanded from May 2015">Articles to be expanded from May 2015</a></li><li><a href="/wiki/Category:All_articles_to_be_expanded" title="Category:All articles to be expanded">All articles to be expanded</a></li><li><a href="/wiki/Category:Articles_using_small_message_boxes" title="Category:Articles using small message boxes">Articles using small message boxes</a></li></ul></div></div>
		<div class="visualClear"></div>
		
	</div>
</div>
<div id='mw-data-after-content'>
	<div class="read-more-container"></div>
</div>

<div id="mw-navigation">
    <h2>Navigation menu</h2>
    <div id="mw-head">
        
        <div id="p-personal" role="navigation" class="" aria-labelledby="p-personal-label">
        	<h3 id="p-personal-label">Personal tools</h3>
        	<ul >
        		
        		<li id="pt-anonuserpage">Not logged in</li>
        		<li id="pt-anontalk"><a href="/wiki/Special:MyTalk" title="Discussion about edits from this IP address [n]" accesskey="n">Talk</a></li><li id="pt-anoncontribs"><a href="/wiki/Special:MyContributions" title="A list of edits made from this IP address [y]" accesskey="y">Contributions</a></li><li id="pt-createaccount"><a href="/w/index.php?title=Special:CreateAccount&amp;returnto=DNA+sequencing" title="You are encouraged to create an account and log in; however, it is not mandatory">Create account</a></li><li id="pt-login"><a href="/w/index.php?title=Special:UserLogin&amp;returnto=DNA+sequencing" title="You&#039;re encouraged to log in; however, it&#039;s not mandatory. [o]" accesskey="o">Log in</a></li>
        	</ul>
        </div>
        <div id="left-navigation">
            <div id="p-namespaces" role="navigation" class="vectorTabs " aria-labelledby="p-namespaces-label">
            	<h3 id="p-namespaces-label">Namespaces</h3>
            	<ul >
            		<li id="ca-nstab-main" class="selected"><a href="/wiki/DNA_sequencing" title="View the content page [c]" accesskey="c">Article</a></li><li id="ca-talk"><a href="/wiki/Talk:DNA_sequencing" rel="discussion" title="Discussion about the content page [t]" accesskey="t">Talk</a></li>
            	</ul>
            </div>
            <div id="p-variants" role="navigation" class="vectorMenu emptyPortlet" aria-labelledby="p-variants-label">
            	<input type="checkbox" class="vectorMenuCheckbox" aria-labelledby="p-variants-label" />
            	<h3 id="p-variants-label">
            		<span>Variants</span>
            	</h3>
            	<ul class="menu" >
            		
            	</ul>
            </div>
        </div>
        <div id="right-navigation">
            <div id="p-views" role="navigation" class="vectorTabs " aria-labelledby="p-views-label">
            	<h3 id="p-views-label">Views</h3>
            	<ul >
            		<li id="ca-view" class="collapsible selected"><a href="/wiki/DNA_sequencing">Read</a></li><li id="ca-edit" class="collapsible"><a href="/w/index.php?title=DNA_sequencing&amp;action=edit" title="Edit this page [e]" accesskey="e">Edit</a></li><li id="ca-history" class="collapsible"><a href="/w/index.php?title=DNA_sequencing&amp;action=history" title="Past revisions of this page [h]" accesskey="h">View history</a></li>
            	</ul>
            </div>
            <div id="p-cactions" role="navigation" class="vectorMenu emptyPortlet" aria-labelledby="p-cactions-label">
            	<input type="checkbox" class="vectorMenuCheckbox" aria-labelledby="p-cactions-label" />
            	<h3 id="p-cactions-label">
            		<span>More</span>
            	</h3>
            	<ul class="menu" >
            		
            	</ul>
            </div>
            <div id="p-search" role="search">
            	<h3 >
            		<label for="searchInput">Search</label>
            	</h3>
            	<form action="/w/index.php" id="searchform">
            		<div id="simpleSearch">
            			<input type="search" name="search" placeholder="Search Wikipedia" title="Search Wikipedia [f]" accesskey="f" id="searchInput"/>
            			<input type="hidden" value="Special:Search" name="title"/>
            			<input type="submit" name="fulltext" value="Search" title="Search Wikipedia for this text" id="mw-searchButton" class="searchButton mw-fallbackSearchButton"/>
            			<input type="submit" name="go" value="Go" title="Go to a page with this exact name if it exists" id="searchButton" class="searchButton"/>
            		</div>
            	</form>
            </div>
        </div>
    </div>
    
    <div id="mw-panel">
    	<div id="p-logo" role="banner">
    		<a  title="Visit the main page" class="mw-wiki-logo" href="/wiki/Main_Page"></a>
    	</div>
    	<div class="portal" role="navigation" id="p-navigation"  aria-labelledby="p-navigation-label">
    		<h3  id="p-navigation-label">
    			Navigation
    		</h3>
    		<div class="body">
    			<ul><li id="n-mainpage-description"><a href="/wiki/Main_Page" title="Visit the main page [z]" accesskey="z">Main page</a></li><li id="n-contents"><a href="/wiki/Wikipedia:Contents" title="Guides to browsing Wikipedia">Contents</a></li><li id="n-featuredcontent"><a href="/wiki/Wikipedia:Featured_content" title="Featured content – the best of Wikipedia">Featured content</a></li><li id="n-currentevents"><a href="/wiki/Portal:Current_events" title="Find background information on current events">Current events</a></li><li id="n-randompage"><a href="/wiki/Special:Random" title="Load a random article [x]" accesskey="x">Random article</a></li><li id="n-sitesupport"><a href="https://donate.wikimedia.org/wiki/Special:FundraiserRedirector?utm_source=donate&amp;utm_medium=sidebar&amp;utm_campaign=C13_en.wikipedia.org&amp;uselang=en" title="Support us">Donate to Wikipedia</a></li><li id="n-shoplink"><a href="//shop.wikimedia.org" title="Visit the Wikipedia store">Wikipedia store</a></li></ul>
    			
    		</div>
    	</div>
    	
    	<div class="portal" role="navigation" id="p-interaction"  aria-labelledby="p-interaction-label">
    		<h3  id="p-interaction-label">
    			Interaction
    		</h3>
    		<div class="body">
    			<ul><li id="n-help"><a href="/wiki/Help:Contents" title="Guidance on how to use and edit Wikipedia">Help</a></li><li id="n-aboutsite"><a href="/wiki/Wikipedia:About" title="Find out about Wikipedia">About Wikipedia</a></li><li id="n-portal"><a href="/wiki/Wikipedia:Community_portal" title="About the project, what you can do, where to find things">Community portal</a></li><li id="n-recentchanges"><a href="/wiki/Special:RecentChanges" title="A list of recent changes in the wiki [r]" accesskey="r">Recent changes</a></li><li id="n-contactpage"><a href="//en.wikipedia.org/wiki/Wikipedia:Contact_us" title="How to contact Wikipedia">Contact page</a></li></ul>
    			
    		</div>
    	</div>
    	
    	<div class="portal" role="navigation" id="p-tb"  aria-labelledby="p-tb-label">
    		<h3  id="p-tb-label">
    			Tools
    		</h3>
    		<div class="body">
    			<ul><li id="t-whatlinkshere"><a href="/wiki/Special:WhatLinksHere/DNA_sequencing" title="List of all English Wikipedia pages containing links to this page [j]" accesskey="j">What links here</a></li><li id="t-recentchangeslinked"><a href="/wiki/Special:RecentChangesLinked/DNA_sequencing" rel="nofollow" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</a></li><li id="t-upload"><a href="/wiki/Wikipedia:File_Upload_Wizard" title="Upload files [u]" accesskey="u">Upload file</a></li><li id="t-specialpages"><a href="/wiki/Special:SpecialPages" title="A list of all special pages [q]" accesskey="q">Special pages</a></li><li id="t-permalink"><a href="/w/index.php?title=DNA_sequencing&amp;oldid=945603030" title="Permanent link to this revision of the page">Permanent link</a></li><li id="t-info"><a href="/w/index.php?title=DNA_sequencing&amp;action=info" title="More information about this page">Page information</a></li><li id="t-wikibase"><a href="https://www.wikidata.org/wiki/Special:EntityPage/Q380546" title="Link to connected data repository item [g]" accesskey="g">Wikidata item</a></li><li id="t-cite"><a href="/w/index.php?title=Special:CiteThisPage&amp;page=DNA_sequencing&amp;id=945603030&amp;wpFormIdentifier=titleform" title="Information on how to cite this page">Cite this page</a></li></ul>
    			
    		</div>
    	</div>
    	
    	<div class="portal" role="navigation" id="p-wikibase-otherprojects"  aria-labelledby="p-wikibase-otherprojects-label">
    		<h3  id="p-wikibase-otherprojects-label">
    			In other projects
    		</h3>
    		<div class="body">
    			<ul><li class="wb-otherproject-link wb-otherproject-commons"><a href="https://commons.wikimedia.org/wiki/Category:DNA_sequencing" hreflang="en">Wikimedia Commons</a></li></ul>
    			
    		</div>
    	</div>
    	
    	<div class="portal" role="navigation" id="p-coll-print_export"  aria-labelledby="p-coll-print_export-label">
    		<h3  id="p-coll-print_export-label">
    			Print/export
    		</h3>
    		<div class="body">
    			<ul><li id="coll-create_a_book"><a href="/w/index.php?title=Special:Book&amp;bookcmd=book_creator&amp;referer=DNA+sequencing">Create a book</a></li><li id="coll-download-as-rl"><a href="/w/index.php?title=Special:ElectronPdf&amp;page=DNA+sequencing&amp;action=show-download-screen">Download as PDF</a></li><li id="t-print"><a href="/w/index.php?title=DNA_sequencing&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</a></li></ul>
    			
    		</div>
    	</div>
    	
    	<div class="portal" role="navigation" id="p-lang"  aria-labelledby="p-lang-label">
    		<h3  id="p-lang-label">
    			Languages
    		</h3>
    		<div class="body">
    			<ul><li class="interlanguage-link interwiki-ar"><a href="https://ar.wikipedia.org/wiki/%D8%AA%D8%B3%D9%84%D8%B3%D9%84_%D8%A7%D9%84%D8%AD%D9%85%D8%B6_%D8%A7%D9%84%D9%86%D9%88%D9%88%D9%8A_%D8%A7%D9%84%D8%B1%D9%8A%D8%A8%D9%88%D8%B2%D9%8A_%D9%85%D9%86%D9%82%D9%88%D8%B5_%D8%A7%D9%84%D8%A3%D9%83%D8%B3%D8%AC%D9%8A%D9%86" title="تسلسل الحمض النووي الريبوزي منقوص الأكسجين – Arabic" lang="ar" hreflang="ar" class="interlanguage-link-target">العربية</a></li><li class="interlanguage-link interwiki-ca"><a href="https://ca.wikipedia.org/wiki/Seq%C3%BCenciaci%C3%B3_d%27ADN" title="Seqüenciació d&#039;ADN – Catalan" lang="ca" hreflang="ca" class="interlanguage-link-target">Català</a></li><li class="interlanguage-link interwiki-cs"><a href="https://cs.wikipedia.org/wiki/Sekvenov%C3%A1n%C3%AD_DNA" title="Sekvenování DNA – Czech" lang="cs" hreflang="cs" class="interlanguage-link-target">Čeština</a></li><li class="interlanguage-link interwiki-da"><a href="https://da.wikipedia.org/wiki/DNA-sekventering" title="DNA-sekventering – Danish" lang="da" hreflang="da" class="interlanguage-link-target">Dansk</a></li><li class="interlanguage-link interwiki-de"><a href="https://de.wikipedia.org/wiki/DNA-Sequenzierung" title="DNA-Sequenzierung – German" lang="de" hreflang="de" class="interlanguage-link-target">Deutsch</a></li><li class="interlanguage-link interwiki-et"><a href="https://et.wikipedia.org/wiki/DNA_sekveneerimine" title="DNA sekveneerimine – Estonian" lang="et" hreflang="et" class="interlanguage-link-target">Eesti</a></li><li class="interlanguage-link interwiki-es"><a href="https://es.wikipedia.org/wiki/Secuenciaci%C3%B3n_del_ADN" title="Secuenciación del ADN – Spanish" lang="es" hreflang="es" class="interlanguage-link-target">Español</a></li><li class="interlanguage-link interwiki-eo"><a href="https://eo.wikipedia.org/wiki/DNA-vicrivelado" title="DNA-vicrivelado – Esperanto" lang="eo" hreflang="eo" class="interlanguage-link-target">Esperanto</a></li><li class="interlanguage-link interwiki-fa"><a href="https://fa.wikipedia.org/wiki/%D8%AA%D9%88%D8%A7%D9%84%DB%8C%E2%80%8C%DB%8C%D8%A7%D8%A8%DB%8C_%D8%AF%DB%8C%E2%80%8C%D8%A7%D9%86%E2%80%8C%D8%A7%DB%8C" title="توالی‌یابی دی‌ان‌ای – Persian" lang="fa" hreflang="fa" class="interlanguage-link-target">فارسی</a></li><li class="interlanguage-link interwiki-fr"><a href="https://fr.wikipedia.org/wiki/S%C3%A9quen%C3%A7age_de_l%27ADN" title="Séquençage de l&#039;ADN – French" lang="fr" hreflang="fr" class="interlanguage-link-target">Français</a></li><li class="interlanguage-link interwiki-gl"><a href="https://gl.wikipedia.org/wiki/Secuenciaci%C3%B3n_do_ADN" title="Secuenciación do ADN – Galician" lang="gl" hreflang="gl" class="interlanguage-link-target">Galego</a></li><li class="interlanguage-link interwiki-hy"><a href="https://hy.wikipedia.org/wiki/%D4%B4%D5%86%D4%B9-%D5%AB_%D5%B0%D5%A1%D5%BB%D5%B8%D6%80%D5%A4%D5%A1%D5%AF%D5%A1%D5%B6%D5%A1%D6%81%D5%B8%D6%82%D5%B4" title="ԴՆԹ-ի հաջորդականացում – Armenian" lang="hy" hreflang="hy" class="interlanguage-link-target">Հայերեն</a></li><li class="interlanguage-link interwiki-hr"><a href="https://hr.wikipedia.org/wiki/Sekvenciranje_DNK" title="Sekvenciranje DNK – Croatian" lang="hr" hreflang="hr" class="interlanguage-link-target">Hrvatski</a></li><li class="interlanguage-link interwiki-id"><a href="https://id.wikipedia.org/wiki/Pengurutan_DNA" title="Pengurutan DNA – Indonesian" lang="id" hreflang="id" class="interlanguage-link-target">Bahasa Indonesia</a></li><li class="interlanguage-link interwiki-is"><a href="https://is.wikipedia.org/wiki/DNA-ra%C3%B0greining" title="DNA-raðgreining – Icelandic" lang="is" hreflang="is" class="interlanguage-link-target">Íslenska</a></li><li class="interlanguage-link interwiki-it"><a href="https://it.wikipedia.org/wiki/Sequenziamento_del_DNA" title="Sequenziamento del DNA – Italian" lang="it" hreflang="it" class="interlanguage-link-target">Italiano</a></li><li class="interlanguage-link interwiki-he"><a href="https://he.wikipedia.org/wiki/%D7%A8%D7%99%D7%A6%D7%95%D7%A3_DNA" title="ריצוף DNA – Hebrew" lang="he" hreflang="he" class="interlanguage-link-target">עברית</a></li><li class="interlanguage-link interwiki-jv"><a href="https://jv.wikipedia.org/wiki/Sekuensing_DNA" title="Sekuensing DNA – Javanese" lang="jv" hreflang="jv" class="interlanguage-link-target">Jawa</a></li><li class="interlanguage-link interwiki-lt"><a href="https://lt.wikipedia.org/wiki/DNR_sekoskaita" title="DNR sekoskaita – Lithuanian" lang="lt" hreflang="lt" class="interlanguage-link-target">Lietuvių</a></li><li class="interlanguage-link interwiki-hu"><a href="https://hu.wikipedia.org/wiki/DNS-szekven%C3%A1l%C3%A1s" title="DNS-szekvenálás – Hungarian" lang="hu" hreflang="hu" class="interlanguage-link-target">Magyar</a></li><li class="interlanguage-link interwiki-ms"><a href="https://ms.wikipedia.org/wiki/Penjujukan_DNA" title="Penjujukan DNA – Malay" lang="ms" hreflang="ms" class="interlanguage-link-target">Bahasa Melayu</a></li><li class="interlanguage-link interwiki-nl"><a href="https://nl.wikipedia.org/wiki/DNA-sequencing" title="DNA-sequencing – Dutch" lang="nl" hreflang="nl" class="interlanguage-link-target">Nederlands</a></li><li class="interlanguage-link interwiki-ja"><a href="https://ja.wikipedia.org/wiki/DNA%E3%82%B7%E3%83%BC%E3%82%AF%E3%82%A8%E3%83%B3%E3%82%B7%E3%83%B3%E3%82%B0" title="DNAシークエンシング – Japanese" lang="ja" hreflang="ja" class="interlanguage-link-target">日本語</a></li><li class="interlanguage-link interwiki-no"><a href="https://no.wikipedia.org/wiki/DNA-sekvensering" title="DNA-sekvensering – Norwegian Bokmål" lang="nb" hreflang="nb" class="interlanguage-link-target">Norsk bokmål</a></li><li class="interlanguage-link interwiki-pl"><a href="https://pl.wikipedia.org/wiki/Sekwencjonowanie_DNA" title="Sekwencjonowanie DNA – Polish" lang="pl" hreflang="pl" class="interlanguage-link-target">Polski</a></li><li class="interlanguage-link interwiki-pt"><a href="https://pt.wikipedia.org/wiki/Sequenciamento_de_DNA" title="Sequenciamento de DNA – Portuguese" lang="pt" hreflang="pt" class="interlanguage-link-target">Português</a></li><li class="interlanguage-link interwiki-ro"><a href="https://ro.wikipedia.org/wiki/Secven%C8%9Bierea_ADN" title="Secvențierea ADN – Romanian" lang="ro" hreflang="ro" class="interlanguage-link-target">Română</a></li><li class="interlanguage-link interwiki-sr"><a href="https://sr.wikipedia.org/wiki/DNK_sekvenciranje" title="DNK sekvenciranje – Serbian" lang="sr" hreflang="sr" class="interlanguage-link-target">Српски / srpski</a></li><li class="interlanguage-link interwiki-sh"><a href="https://sh.wikipedia.org/wiki/DNK_sekvenciranje" title="DNK sekvenciranje – Serbo-Croatian" lang="sh" hreflang="sh" class="interlanguage-link-target">Srpskohrvatski / српскохрватски</a></li><li class="interlanguage-link interwiki-fi"><a href="https://fi.wikipedia.org/wiki/DNA:n_sekvensointi" title="DNA:n sekvensointi – Finnish" lang="fi" hreflang="fi" class="interlanguage-link-target">Suomi</a></li><li class="interlanguage-link interwiki-sv"><a href="https://sv.wikipedia.org/wiki/DNA-sekvensering" title="DNA-sekvensering – Swedish" lang="sv" hreflang="sv" class="interlanguage-link-target">Svenska</a></li><li class="interlanguage-link interwiki-ta"><a href="https://ta.wikipedia.org/wiki/%E0%AE%9F%E0%AE%BF.%E0%AE%8E%E0%AE%A9%E0%AF%8D.%E0%AE%8F_%E0%AE%B5%E0%AE%B0%E0%AE%A9%E0%AF%8D%E0%AE%AE%E0%AF%81%E0%AE%B1%E0%AF%88%E0%AE%AF%E0%AE%BF%E0%AE%9F%E0%AE%B2%E0%AF%8D" title="டி.என்.ஏ வரன்முறையிடல் – Tamil" lang="ta" hreflang="ta" class="interlanguage-link-target">தமிழ்</a></li><li class="interlanguage-link interwiki-th"><a href="https://th.wikipedia.org/wiki/%E0%B8%81%E0%B8%B2%E0%B8%A3%E0%B8%AB%E0%B8%B2%E0%B8%A5%E0%B8%B3%E0%B8%94%E0%B8%B1%E0%B8%9A%E0%B8%94%E0%B8%B5%E0%B9%80%E0%B8%AD%E0%B9%87%E0%B8%99%E0%B9%80%E0%B8%AD" title="การหาลำดับดีเอ็นเอ – Thai" lang="th" hreflang="th" class="interlanguage-link-target">ไทย</a></li><li class="interlanguage-link interwiki-tr"><a href="https://tr.wikipedia.org/wiki/DNA_dizileme" title="DNA dizileme – Turkish" lang="tr" hreflang="tr" class="interlanguage-link-target">Türkçe</a></li><li class="interlanguage-link interwiki-uk"><a href="https://uk.wikipedia.org/wiki/%D0%A1%D0%B5%D0%BA%D0%B2%D0%B5%D0%BD%D1%83%D0%B2%D0%B0%D0%BD%D0%BD%D1%8F_%D0%94%D0%9D%D0%9A" title="Секвенування ДНК – Ukrainian" lang="uk" hreflang="uk" class="interlanguage-link-target">Українська</a></li><li class="interlanguage-link interwiki-vi"><a href="https://vi.wikipedia.org/wiki/Ph%C6%B0%C6%A1ng_ph%C3%A1p_Dideoxy" title="Phương pháp Dideoxy – Vietnamese" lang="vi" hreflang="vi" class="interlanguage-link-target">Tiếng Việt</a></li><li class="interlanguage-link interwiki-zh"><a href="https://zh.wikipedia.org/wiki/DNA%E6%B8%AC%E5%BA%8F" title="DNA測序 – Chinese" lang="zh" hreflang="zh" class="interlanguage-link-target">中文</a></li></ul>
    			<div class="after-portlet after-portlet-lang"><span class="wb-langlinks-edit wb-langlinks-link"><a href="https://www.wikidata.org/wiki/Special:EntityPage/Q380546#sitelinks-wikipedia" title="Edit interlanguage links" class="wbc-editpage">Edit links</a></span></div>
    		</div>
    	</div>
    	
    </div>
</div>

<div id="footer" role="contentinfo" >
	<ul id="footer-info" class="">
		<li id="footer-info-lastmod"> This page was last edited on 15 March 2020, at 00:44<span class="anonymous-show">&#160;(UTC)</span>.</li>
		<li id="footer-info-copyright">Text is available under the <a rel="license" href="//en.wikipedia.org/wiki/Wikipedia:Text_of_Creative_Commons_Attribution-ShareAlike_3.0_Unported_License">Creative Commons Attribution-ShareAlike License</a><a rel="license" href="//creativecommons.org/licenses/by-sa/3.0/" style="display:none;"></a>;
additional terms may apply.  By using this site, you agree to the <a href="//foundation.wikimedia.org/wiki/Terms_of_Use">Terms of Use</a> and <a href="//foundation.wikimedia.org/wiki/Privacy_policy">Privacy Policy</a>. Wikipedia® is a registered trademark of the <a href="//www.wikimediafoundation.org/">Wikimedia Foundation, Inc.</a>, a non-profit organization.</li>
	</ul>
	<ul id="footer-places" class="">
		<li id="footer-places-privacy"><a href="https://foundation.wikimedia.org/wiki/Privacy_policy" class="extiw" title="wmf:Privacy policy">Privacy policy</a></li>
		<li id="footer-places-about"><a href="/wiki/Wikipedia:About" title="Wikipedia:About">About Wikipedia</a></li>
		<li id="footer-places-disclaimer"><a href="/wiki/Wikipedia:General_disclaimer" title="Wikipedia:General disclaimer">Disclaimers</a></li>
		<li id="footer-places-contact"><a href="//en.wikipedia.org/wiki/Wikipedia:Contact_us">Contact Wikipedia</a></li>
		<li id="footer-places-developers"><a href="https://www.mediawiki.org/wiki/Special:MyLanguage/How_to_contribute">Developers</a></li>
		<li id="footer-places-statslink"><a href="https://stats.wikimedia.org/#/en.wikipedia.org">Statistics</a></li>
		<li id="footer-places-cookiestatement"><a href="https://foundation.wikimedia.org/wiki/Cookie_statement">Cookie statement</a></li>
		<li id="footer-places-mobileview"><a href="//en.m.wikipedia.org/w/index.php?title=DNA_sequencing&amp;mobileaction=toggle_view_mobile" class="noprint stopMobileRedirectToggle">Mobile view</a></li>
	</ul>
	<ul id="footer-icons" class="noprint">
		<li id="footer-copyrightico"><a href="https://wikimediafoundation.org/"><img src="/static/images/wikimedia-button.png" srcset="/static/images/wikimedia-button-1.5x.png 1.5x, /static/images/wikimedia-button-2x.png 2x" width="88" height="31" alt="Wikimedia Foundation"/></a></li>
		<li id="footer-poweredbyico"><a href="https://www.mediawiki.org/"><img src="/static/images/poweredby_mediawiki_88x31.png" alt="Powered by MediaWiki" srcset="/static/images/poweredby_mediawiki_132x47.png 1.5x, /static/images/poweredby_mediawiki_176x62.png 2x" width="88" height="31"/></a></li>
	</ul>
	<div style="clear: both;"></div>
</div>

<script>(RLQ=window.RLQ||[]).push(function(){mw.config.set({"wgPageParseReport":{"limitreport":{"cputime":"2.060","walltime":"2.385","ppvisitednodes":{"value":8747,"limit":1000000},"postexpandincludesize":{"value":372178,"limit":2097152},"templateargumentsize":{"value":5055,"limit":2097152},"expansiondepth":{"value":14,"limit":40},"expensivefunctioncount":{"value":14,"limit":500},"unstrip-depth":{"value":1,"limit":20},"unstrip-size":{"value":558627,"limit":5000000},"entityaccesscount":{"value":7,"limit":400},"timingprofile":["100.00% 2099.731      1 -total"," 78.13% 1640.580      2 Template:Reflist"," 53.81% 1129.874    113 Template:Cite_journal","  5.75%  120.797     28 Template:Cite_web","  3.27%   68.603      5 Template:Fix","  3.10%   64.992      4 Template:Citation_needed","  2.19%   46.064      1 Template:Genetics_sidebar","  2.02%   42.426      1 Template:Short_description","  1.95%   41.033      1 Template:Sidebar","  1.78%   37.314     10 Template:Category_handler"]},"scribunto":{"limitreport-timeusage":{"value":"1.293","limit":"10.000"},"limitreport-memusage":{"value":9080688,"limit":52428800},"limitreport-profile":[["Scribunto_LuaSandboxCallback::find","260","19.1"],["Scribunto_LuaSandboxCallback::match","160","11.8"],["recursiveClone \u003CmwInit.lua:41\u003E","140","10.3"],["?","140","10.3"],["Scribunto_LuaSandboxCallback::getEntity","120","8.8"],["Scribunto_LuaSandboxCallback::getAllExpandedArguments","100","7.4"],["Scribunto_LuaSandboxCallback::callParserFunction","100","7.4"],["type","60","4.4"],["\u003Cmw.lua:683\u003E","60","4.4"],["maskEntityTable \u003Cmw.wikibase.entity.lua:54\u003E","40","2.9"],["[others]","180","13.2"]]},"cachereport":{"origin":"mw1262","timestamp":"20200326171837","ttl":2592000,"transientcontent":false}}});});</script>
<script type="application/ld+json">{"@context":"https:\/\/schema.org","@type":"Article","name":"DNA sequencing","url":"https:\/\/en.wikipedia.org\/wiki\/DNA_sequencing","sameAs":"http:\/\/www.wikidata.org\/entity\/Q380546","mainEntity":"http:\/\/www.wikidata.org\/entity\/Q380546","author":{"@type":"Organization","name":"Contributors to Wikimedia projects"},"publisher":{"@type":"Organization","name":"Wikimedia Foundation, Inc.","logo":{"@type":"ImageObject","url":"https:\/\/www.wikimedia.org\/static\/images\/wmf-hor-googpub.png"}},"datePublished":"2004-11-11T17:08:08Z","dateModified":"2020-03-15T00:44:54Z","image":"https:\/\/upload.wikimedia.org\/wikipedia\/commons\/f\/f0\/Hybridogenesis_in_water_frogs_gametes.svg","headline":"biological research program"}</script>
<script>(RLQ=window.RLQ||[]).push(function(){mw.config.set({"wgBackendResponseTime":118,"wgHostname":"mw1393"});});</script></body></html>
